% Template for PLoS
% Version 3.4 January 2017
%
% % % % % % % % % % % % % % % % % % % % % %
%
% -- IMPORTANT NOTE
%
% This template contains comments intended
% to minimize problems and delays during our production
% process. Please follow the template instructions
% whenever possible.
%
% % % % % % % % % % % % % % % % % % % % % % %
%
% Once your paper is accepted for publication,
% PLEASE REMOVE ALL TRACKED CHANGES in this file
% and leave only the final text of your manuscript.
% PLOS recommends the use of latexdiff to track changes during review, as this will help to maintain a clean tex file.
% Visit https://www.ctan.org/pkg/latexdiff?lang=en for info or contact us at latex@plos.org.
%
%
% There are no restrictions on package use within the LaTeX files except that
% no packages listed in the template may be deleted.
%
% Please do not include colors or graphics in the text.
%
% The manuscript LaTeX source should be contained within a single file (do not use \input, \externaldocument, or similar commands).
%
% % % % % % % % % % % % % % % % % % % % % % %
%
% -- FIGURES AND TABLES
%
% Please include tables/figure captions directly after the paragraph where they are first cited in the text.
%
% DO NOT INCLUDE GRAPHICS IN YOUR MANUSCRIPT
% - Figures should be uploaded separately from your manuscript file.
% - Figures generated using LaTeX should be extracted and removed from the PDF before submission.
% - Figures containing multiple panels/subfigures must be combined into one image file before submission.
% For figure citations, please use "Fig" instead of "Figure".
% See http://journals.plos.org/plosone/s/figures for PLOS figure guidelines.
%
% Tables should be cell-based and may not contain:
% - spacing/line breaks within cells to alter layout or alignment
% - do not nest tabular environments (no tabular environments within tabular environments)
% - no graphics or colored text (cell background color/shading OK)
% See http://journals.plos.org/plosone/s/tables for table guidelines.
%
% For tables that exceed the width of the text column, use the adjustwidth environment as illustrated in the example table in text below.
%
% % % % % % % % % % % % % % % % % % % % % % % %
%
% -- EQUATIONS, MATH SYMBOLS, SUBSCRIPTS, AND SUPERSCRIPTS
%
% IMPORTANT
% Below are a few tips to help format your equations and other special characters according to our specifications. For more tips to help reduce the possibility of formatting errors during conversion, please see our LaTeX guidelines at http://journals.plos.org/plosone/s/latex
%
% For inline equations, please be sure to include all portions of an equation in the math environment.  For example, x$^2$ is incorrect; this should be formatted as $x^2$ (or $\mathrm{x}^2$ if the romanized font is desired).
%
% Do not include text that is not math in the math environment. For example, CO2 should be written as CO\textsubscript{2} instead of CO$_2$.
%
% Please add line breaks to long display equations when possible in order to fit size of the column.
%
% For inline equations, please do not include punctuation (commas, etc) within the math environment unless this is part of the equation.
%
% When adding superscript or subscripts outside of brackets/braces, please group using {}.  For example, change "[U(D,E,\gamma)]^2" to "{[U(D,E,\gamma)]}^2".
%
% Do not use \cal for caligraphic font.  Instead, use \mathcal{}
%
% % % % % % % % % % % % % % % % % % % % % % % %
%
% Please contact latex@plos.org with any questions.
%
% % % % % % % % % % % % % % % % % % % % % % % %

\documentclass[10pt,letterpaper]{article}
\usepackage[top=0.85in,left=2.75in,footskip=0.75in]{geometry}

% amsmath and amssymb packages, useful for mathematical formulas and symbols
\usepackage{amsmath,amssymb}

% Use adjustwidth environment to exceed column width (see example table in text)
\usepackage{changepage}

% Use Unicode characters when possible
\usepackage[utf8x]{inputenc}

% textcomp package and marvosym package for additional characters
\usepackage{textcomp,marvosym}

% cite package, to clean up citations in the main text. Do not remove.
\usepackage{cite}

% % Added natbib for citet commands:
% \usepackage[numbers]{natbib}

% Added url for urls:
\usepackage{url}

% Use nameref to cite supporting information files (see Supporting Information section for more info)
\usepackage{nameref,hyperref}

% line numbers
\usepackage[right]{lineno}

% ligatures disabled
\usepackage{microtype}
\DisableLigatures[f]{encoding = *, family = * }

% color can be used to apply background shading to table cells only
\usepackage[table]{xcolor}

% array package and thick rules for tables
\usepackage{array}

% create "+" rule type for thick vertical lines
\newcolumntype{+}{!{\vrule width 2pt}}

% create \thickcline for thick horizontal lines of variable length
\newlength\savedwidth
\newcommand\thickcline[1]{%
  \noalign{\global\savedwidth\arrayrulewidth\global\arrayrulewidth 2pt}%
  \cline{#1}%
  \noalign{\vskip\arrayrulewidth}%
  \noalign{\global\arrayrulewidth\savedwidth}%
}

% \thickhline command for thick horizontal lines that span the table
\newcommand\thickhline{\noalign{\global\savedwidth\arrayrulewidth\global\arrayrulewidth 2pt}%
\hline
\noalign{\global\arrayrulewidth\savedwidth}}


% Remove comment for double spacing
%\usepackage{setspace}
%\doublespacing

% Text layout
\raggedright
\setlength{\parindent}{0.5cm}
\textwidth 5.25in
\textheight 8.75in

% Bold the 'Figure #' in the caption and separate it from the title/caption with a period
% Captions will be left justified
\usepackage[aboveskip=1pt,labelfont=bf,labelsep=period,justification=raggedright,singlelinecheck=off]{caption}
\renewcommand{\figurename}{Fig}

% Use the PLoS provided BiBTeX style
\bibliographystyle{plos2015}
% \bibliographystyle{plainnat}

% Remove brackets from numbering in List of References
\makeatletter
\renewcommand{\@biblabel}[1]{\quad#1.}
\makeatother

% Leave date blank
\date{}

% Header and Footer with logo
\usepackage{lastpage,fancyhdr,graphicx}
\usepackage{epstopdf}
\usepackage{hyperref}% http://ctan.org/pkg/hyperref
\pagestyle{myheadings}
\pagestyle{fancy}
\fancyhf{}
\setlength{\headheight}{27.023pt}
\lhead{\includegraphics[width=2.0in]{PLOS-submission.eps}}
\rfoot{\thepage/\pageref{LastPage}}
\renewcommand{\footrule}{\hrule height 2pt \vspace{2mm}}
\fancyheadoffset[L]{2.25in}
\fancyfootoffset[L]{2.25in}
\lfoot{\sf PLOS}


% Make it so figures are more likely to share a page with text:
\renewcommand{\topfraction}{0.85} % 85% rather than default 70% of the top of a text page can contain figures.
\renewcommand{\textfraction}{0.1} % as little as 10% of a page can be text when shared with a figure.
\renewcommand{\floatpagefraction}{0.80} % only makes a figure-only page if figure takes up more than 75% of a page

%% Include all macros below

\newcommand{\lorem}{{\bf LOREM}}
\newcommand{\ipsum}{{\bf IPSUM}}

%% END MACROS SECTION

\graphicspath{{./figs/}}

\begin{document}
\vspace*{0.2in}

% Title must be 250 characters or less.
\begin{flushleft}
{\Large
\textbf\newline{Hub connectivity and gene expression in the worm connectome} % Please use "sentence case" for title and headings (capitalize only the first word in a title (or heading), the first word in a subtitle (or subheading), and any proper nouns).
}
\newline
% Insert author names, affiliations and corresponding author email (do not include titles, positions, or degrees).
\\
Aurina Arnatkevi\u{c}i\={u}t\.{e}\textsuperscript{1\Yinyang},
Ben D. Fulcher\textsuperscript{1\Yinyang},
Alex Fornito\textsuperscript{1}\\
\bigskip
\textbf{1} Brain and Mental Health Laboratory, Monash Institute of Cognitive and Clinical Neurosciences, School of Psychological Sciences, Monash University, 770 Blackburn Rd, Clayton, 3168, VIC, Australia.
\\
\bigskip

% Insert additional author notes using the symbols described below. Insert symbol callouts after author names as necessary.
%
% Remove or comment out the author notes below if they aren't used.
%
% Primary Equal Contribution Note
\Yinyang These authors contributed equally to this work.

% Additional Equal Contribution Note
% Also use this double-dagger symbol for special authorship notes, such as senior authorship.
% \ddag These authors also contributed equally to this work.

% Current address notes
% \textcurrency Current Address: Dept/Program/Center, Institution Name, City, State, Country % change symbol to "\textcurrency a" if more than one current address note
% \textcurrency b Insert second current address
% \textcurrency c Insert third current address

% Deceased author note
% \dag Deceased

% Group/Consortium Author Note
% \textpilcrow Membership list can be found in the Acknowledgments section.

% Use the asterisk to denote corresponding authorship and provide email address in note below.
* correspondingauthor@institute.edu

\end{flushleft}
% Please keep the abstract below 300 words
\section*{Abstract}
Text


% Please keep the Author Summary between 150 and 200 words
% Use first person. PLOS ONE authors please skip this step.
% Author Summary not valid for PLOS ONE submissions.
\section*{Author summary}
Text

\linenumbers

% Use "Eq" instead of "Equation" for equation citations.
\section*{Introduction}

ALEX:

\begin{itemize}
    \item Nervous systems are complex
    \item Complexity commonly studied using graphs
    \item Generality of graphs allows nervous systems to be modeled using a common framework across scales and species
    \item This work has shown that nervous systems are modular, has hubs, and hubs are connected forming an RC
    \item RC important for integration, but also costly
    % \item In particular, conservation of hub connectivity across scales and species implies genetic influences
    % \item this has been confirmed in mouse by Fulcher, and human by Vertes (cf. Fornito heritability work)
    % \item C elegans is attractive model to extend these results as connectome mapped completely at synapse and neuron level. We also have detailed information on neuron birth times, cell lineage, neuron type... etc that can help determine whether genomic similarity is driven by one of these factors
    % \item In this work, we collated all these diverse data to test hypothesis that elevated expression coupling of hubs would be apparent at microscale, and to identify what specifically drives this result.
\end{itemize}


\begin{enumerate}
    \item{Structural network properties}
    \begin{itemize}
    \item{rich-club, hubs, conservation cross-scale/species/methods}
    \item{Prior work in worm: different types of connectomes - wired/unwired, hubs}
    \end{itemize}

    \item{Structure-expression}
    \begin{itemize}
    \item{Datasets in different scales: mouse/rat, human (also heritability), developmental, large-scale datasets}
    \end{itemize}
 \end{enumerate}


% Introduce the structural connectome, characterization of its properties, focusing on worm, but also linking to other species to make the point of consistency
The complex pattern of axonal connections between neural elements of nervous systems provides key insights into the physical constraints underlying a functionally optimized system.
Mapping the network of structural connections between neural elements, from the scale of neurons in model organisms like \emph{C. elegans}, to mesoscale brain regions using tract tracing in rodents, to non-invasive MRI imaging in human subjects, has revealed a surprising consistency of organizational properties, suggesting a common set of selection pressures under which efficient neural systems have evolved.
In this work, the brain network, or `connectome', is abstracted to a graph representation, in which neural elements are represented as nodes (e.g., neurons in \emph{C. elegans}, or macroscopic brain regions in human), with edges connecting pairs of elements that have an axonal connection; in this way allowing very different species, measurements, and scales of neuronal networks to be represented in a consistent and unified way.
As well as showing a modular network structure, that can often be interpreted in terms of distinct functional brain networks, connectomes have been consistently shown to exhibit a non-uniform distribution of connectivity across the network, leading to the existence of highly connected `hub' nodes.
Hubs have also been shown to be strongly interconnected, forming an integrated `rich club' that may be crucial for facilitating efficient integration of information between more specialized processing modules.
Prior work has characterized the rich club organization of the neuronal connectome of the \emph{C. elegans} nervous system \cite{Towlson:2013gf}, \emph{Drosophila} \cite{Shih:2015cu}, mouse \cite{Fulcher:2016ck}, rat \cite{vandenHeuvel:2015ie}, 53 \cite{ZamoraLopez:2010hy} or 65 areas \cite{deReus:2013cy} of the cat cortex, 242 macaque cortical regions \cite{Harriger:2012bb}, and in 82 brain regions \cite{vandenHeuvel:2011he} and 1\,170 human cortical areas \cite{vandenHeuvel:2012kh}.
Although hub connectivity is costly (with strong, long-range projections), it is thought to provide a functional benefit in facilitating efficient integration of diverse neural systems \cite{VandenHeuvel2012}.
The consistency of this structural feature across the connectomes of so many species, and scales suggests that it may be subject to genetic control.
% I DON'T BELIEVE IN THIS ONE SO LEAVING IT OUT: pigeon \cite{Shanahan:2013ej},

% Introduce gene expression work and prior work relating them
Although much effort has characterized structural network topology across species and scales, the recent availability of gene expression data for neural systems has allowed the structural network features described above to be related to molecular information, allowing the field of macroscale connectomics to provide microscale understanding and vice versa \cite{vandenHeuvel:2017ex}.
Initial work was done in \emph{C. elegans}, showing that the expression of a small number of genes can be used to predict connectivity between pairs of neurons using statistical approaches \cite{Varadan:2006ek, Kaufman2006, Baruch2008b}.
Later work, utilizing detailed gene expression data in mouse from the Allen Institute \cite{Lein:2007jn} revealed similar patterns in mouse \cite{Fakhry:2015kl, Ji:2014jw}, and that pairs of brain regions with similar gene expression patterns tend show more similar connectivity profiles \cite{French2011}.
% Fakhry:2015dg
% that connected pairs of brain regions display more similar gene expression profiles \cite{French2011},
Looked further into types of connections, we showed in mouse that pairs of connected hubs have the most similar gene expression profiles, and that hub connectivity is associated with the correlated expression of genes related to oxidative energy metabolism \cite{Fulcher:2016ck} (a result mirrored at the regional level in human \cite{Vertes2016a}).
% These findings highlight a close relationship between metabolic expenditure and the high signaling load of hub regions in the brain, as has been previously proposed \cite{Bullmore2012}.
Like the cross-species consistency of hub connectivity, these results hint that structural connectivity and gene expression may also be consistently related.
\emph{C. elegans} is attractive model to extend these results to the microscale, being the only connectome fully mapped at the level of individual synapses and neurons, avoiding statistical estimation of connectomes from tract-tracing experiments \cite{Ypma:2016em}, or the choice of a discrete parcellation for a continuous brain.
In \emph{C. elegans}, we also have detailed information on neuron birth times, cell lineage, neurotransmitters, neuron type, etc. that can help delineate the effect of cell type properties on our findings in greater depth.
In this work, we collated all these diverse data to test the hypothesis that elevated expression coupling of hubs would be apparent at microscale, and to identify what genes, or other factors, drive the result.

% Introduce hub connectivity-expression hypothesis
% Connectivity in brain networks is not uniformly distributed. Network elements with high node degree – i.e., a large number of connections to other areas – are called `hubs'.
% When hubs are more densely interconnected than expected by chance they form a `rich-club', the idea being that the richest members of the network (in terms of connections) are tightly connected to each other, thus forming a club.
% These densely interconnected hubs are thought to promote efficient integration between anatomically distinct areas and play an important role in brain functioning.
% It has been shown that hubs exhibit distinct transcriptional signatures in both humans \cite{Vertes2016a} and mice \cite{Fulcher2016}.
% According to \citet{Fulcher2016}, connections involving rich club hubs carry a distinctive genetic signature, which is driven by genes regulating the synthesis and breakdown of adenosine triphosphate (ATP) – the primary energetic substrate of neuronal signaling \cite{Fulcher2016}.
% These findings highlight a close relationship between metabolic expenditure and the high signaling load of hub regions in the brain, as has been previously proposed \cite{Bullmore2012}.
% We therefore have preliminary indications that the transcriptional signature of hubs may be a consistent feature of mammalian brain networks, but it is not known how distinctive this expression signature is; and in particular, whether it holds true for networks resolved at the scale of individual neurons and synapses.
% To test this possibility, we aimed to replicate findings presented in \cite{Fulcher2016} using microscale connectivity data in \emph{C. elegans} and gene expression data from \emph{WormBase}.
% We sought to determine whether hubs in the \emph{C. elegans} connectome exhibit distinctive gene expression patterns.


% METHODS SECTION
\section*{Materials and methods}

In this work, we couple synaptic connectivity to gene expression signatures for the somatic nervous system of the adult \textit{C. elegans} hermaphrodite.
We first describe the neural connectivity data used to construct a connectome and the methods we use to quantify connectome structure.
We then go on to describe the binary gene expression data, how we measure expression similarity between pairs of neurons (coexpression), and our method for scoring genes for functional enrichment analysis.

% Neurons making up the nervous system of the nematode worm \emph{C. elegans} can be divided into groups, commonly as: sensory neurons (support receptive function), motor neurons (cells containing neuromuscular junctions), interneurons (all other neurons), and polymodal neurons (performing more than one type of circuit function) \cite{White:1986tx}.

% METHODS: Outline (add what it's about)
% To investigate the relationship between hub connectivity and gene expression in a micro-scale network we coupled two publicly available datasets containing synaptic-level connectivity network and gene expression signatures for the somatic nervous system of the \textit{C. elegans} hermaphrodite.

% METHODS: connectivity data
\subsection*{Neuronal connectivity data}

Neuronal connectivity data for the adult hermaphrodite nematode \emph{C. elegans} was first mapped by White et al. \cite{White:1986tx}, and then revised by Chen et al. \cite{Chen:2006ie} and by Varshney et al. \cite{Varshney2011} through detailed electron microscopy.
Here we analyze data from Varshney et al. \cite{Varshney2011} for 279 somatic neurons (282 nonpharyngeal neurons, excluding CANL, CANR, and VC6, for which connectivity data is unavailable), downloaded from WormAtlas (\url{http://www.wormatlas.org/neuronalwiring.html#NeuronalconnectivityII}).
Connectivity information is available both for electrical gap junctions and chemical synapses, which are further distinguished by presynaptic and postsynaptic neurons (providing connection direction information), as well as the number of synapses (providing connection weight information).
% The result can be represented as a weighted, directed connectivity matrix, with weights given by the number of synapses.
% By contrast, the directionality of electrical gap junctions remains unknown, and have thus been represented as bidirectional connections.
To avoid any possible differences in gene expression between chemical synapses and gap junctions, our analysis focuses just on the synaptic connectivity network, which contains a total of 1\,961 connections (6\,394 synapses).

Note the previous work has used different processed versions of this data, including just synaptic connections as here (e.g., after thresholding them \cite{Kashtan:2004ev}), a combination of synaptic connections and gap junctions together \cite{Azulay:2016cg, Kim:2016gl}, or combining synaptic and gap junctions and then symmetrizing unidirectional synaptic connections to be bidirectional \cite{Towlson:2013gf, Kim:2014bu, Pavlovic:2014gx}.

Spatial information about neurons was obtained for 277 neurons (including VC6 [[TODO: Aurina, why mention this?]]) as two dimensional spatial coordinates from \url{www.biological-networks.org/?page_id=25}.
Coordinates for three neurons (AIBL, AIYL, SMDVL) were missing in this dataset, and were reconstructed based on the positions of the contralateral neurons (AIBR, AIYR, SMDVR) by assigning identical coordinates for the contralateral neurons in these three pairs according to Varier \cite{Varier2011} [[TODO: Explain the precedent for doing this]]. [[TODO: Aurina, provide details on how these were `reconstructed']].

% The C. elegans nervous system consists of two distinct parts: a small pharyngeal nervous system (20 neurons), responsible for  the feeding behavior of the animal and a large somatic nervous system (282 neurons).

In order to relate our topological and gene expression results to other information about individual neurons or pairs of neurons, we assembled diverse publicly available data for \emph{C. elegans}.
Neurons were first labeled by their type, as:
(i) `sensory' neuron (support receptive function),
(ii) `motor' neuron (cells containing neuromuscular junctions),
(iii) `interneuron' (all other neurons), or
(iv) polymodal neuron (obtaining multiple assignments of the above categories) \cite{White:1986tx}.
Neurons were also assigned labels based on their anatomical location, as:
(i) `head', (ii) `tail', or (iii) `body'.
These assignments were obtained from release WS256 of WormBase, \url{ftp://ftp.wormbase.org/pub/wormbase/releases/WS256/ONTOLOGY/anatomy_association.WS256.wb}, with annotations to individual neurons achieved from the hierarchy defined in WormBase \cite{Harris:2009kd}, which we retrieved using the WormBase API (wormmine) and propagated each term down the hierarchy to label individual neurons.
% (and other child terms of `neuron', WB:0003679)
% RetreiveHierarchy.py, NeuronRegionFinder.m
Note that some head neurons (12: ALA, AVFL, AVFR, AVG, RIFL, RIFR, RIGL, RIGR, SABD, SABVL, SABVR, SMDVL) were not labeled as head in this file, and were thus manually corrected by matching the assignment against the annotations provided at WormBase (http://www.wormatlas.org/).
% [[TODO: how and where from hierarchy file was downloaded?]]% RetreiveHierarchy.py, NeuronRegionFinder.m
The neurotransmitter systems used by each neuron was labeled by matching to data in Table 2 of Pereira et al. \cite{Pereira:2015er}.
% to label the neurotransmitter type of all neurons.
% in the hermaphrodite C. elegans nervous system has been mapped, revealing the dominance of cholinergic signaling \cite{Pereira:2015er}.
Lineage similarity for all pairs of neurons was obtained from previously published embryonic and  post-embryonic lineage trees \cite{Sulston1977, Sulston1983}; data were downloaded from WormAtlas (\url{http://www.wormatlas.org/neuronalwiring.html#Lineageanalysis}).
In this dataset for each pair of neurons a common ancestor cell was identified using and a total number of cell divisions from the common progenitor was calculated.
% Downloaded data were arranged into a matrix format where lineage distance for each pair of neurons was defined. % part of Print_neuroconnect.m
[[TODO: Aurina -- can we go through all of this to clarify all of the data sources -- wormMine, wormBase, WormAtlas, etc. I'm a bit confused]]

% METHODS: rich club analysis
\subsection*{Network analysis}

In this section, we describe the methods used to characterize the \emph{C. elegans} structural connectome, represented as a graph with neurons as nodes and synaptic connections as edges \cite{Schroter:2017eo}.

% \paragraph{Degree}
The number of regions that a neuron projects to is its out-degree, $k_\mathrm{out}$, and the number of regions that project to a given neuron is its in-degree, $k_\mathrm{in}$.
The total number of connections involving a given neuron is defined by its degree, $k = k_\mathrm{in} + k_\mathrm{out}$.
% Weighted analogues of these degree measures involve counting the number of synaptic connections that a neuron makes to other regions (weighted out-degree, $k^w_\mathrm{out}$), that are afferent to it (weighted in-degree, $k^w_\mathrm{in}$), or the combination, weighted degree, $k^w = k^w_\mathrm{in} + k^w_\mathrm{out}$.

At a given degree threshold, $k$, neurons were classified as either `hub' (degree $>k$) or `non-hub' (degree $\leq k$).
All edges could subsequently be classified as either `rich' (hub $\rightarrow$ hub), `feed-in' (non-hub $\rightarrow$ hub), `feed-out' (hub $\rightarrow$ non-hub), or `peripheral' (non-hub $\rightarrow$ non-hub).
Note that, in the absence of directed connectivity information, previous analyses involving rich clubs has mostly grouped `feed-in' and `feed-out' as a single `feeder' category, but here we distinguish the two connection types.
% For simplicity, hub $\rightarrow$ non-hub and non-hub $\rightarrow$ hub connections were grouped as a single `feeder' category for simplicity.

% \paragraph{Rich-club organization}
To measure whether hubs are densely interconnected, we used the rich-club coefficient, $\phi(k)$ \cite{Colizza:2006kz}:
\begin{equation}
    \label{eqn:rich_club}
    \phi(k) = \frac{2E_{>k}}{N_{>k}(N_{>k}-1)},
\end{equation}
where $N_{>k}$ is the number of nodes with degree $>k$ and $E_{>k}$ counts the number of edges between them.
The rich-club coefficient, $\phi(k)$, thus measures the link density in the subgraph containing nodes with degree $>k$.
However, $\phi(k)$ is expected to increase with $k$ for general networks, since retaining nodes with higher degree will yield a higher expected link density in the subgraph containing nodes with degree $>k$.
Consequently, we compared $\phi(k)$ measured from the \emph{C. elegans} connectome, to that obtained from an ensemble of randomized null networks, $\phi_\mathrm{rand}(k)$ \cite{Colizza:2006kz}.
An ensemble of 1\,000 null networks were generated, constructed by shuffling the links in the empirical network while retaining the degree distribution of nodes in the network \cite{Maslov:2002hi} (rewiring each edge an average of 50 times per null network) using the \texttt{randmio\_dir} function from the {\it Brain Connectivity Toolbox} \cite{Rubinov:2010jd}.
We then computed the normalized rich-club coefficient, $\Phi_\mathrm{norm}(k)$, as the ratio of the rich-club coefficient of the empirical network to the mean rich-club coefficient of the ensemble of randomized networks:
\begin{equation}
    \Phi_\mathrm{norm}(k) = \phi(k)/\langle \phi_\mathrm{rand}(k) \rangle.
\end{equation}
$\Phi_\mathrm{norm} > 1$ indicates the rich-club organization of the network, with statistical significant deviations assessed by computing a $P$-value directly from the empirical null distribution, $\phi_\mathrm{rand}(k)$, as a permutation test under the null hypothesis $\phi(k) \leq \phi_\mathrm{rand}(k)$ \cite{vandenHeuvel:2011he}.

% NOTE THAT TOWLSON USED UNDIRECTED (CHEMICAL+ELECTRICAL) network. WE USE DIRECTED CHEMICAL. (should probably put this into results - not in methids).
%The phenomenon of the rich-club organisation in \textit{C. elegans} connectome was originally demonstrated by Towlson et al. \cite{Towlson2013} in an undirected binary version of the connectome where data for both chemical and electrical synapses were combined.
%As a result, a small number of high degree nodes forming a rich club was identified.

% \paragraph{Modularity}
The modular structure of the connectome was determined applying the Louvain community detection algorithm using a standard practice, implemented in the Brain Connectivity Toolbox \cite{Rubinov2010}.
To identify the optimal modular assignment, the algorithm was repeated 10\,000 times [[TODO: is this correct? Not 1\,000?]] resulting in a thousand community affiliation vectors [[TODO: provide some context wrt algorithmic degeneracy]].
The results of these 1\,000 partitions were aggregated into a single agreement matrix, in which each element encoded the frequency with which a given pair of nodes was classified in the same module across runs.
These frequencies were weighted by the quality of each individual partition, as quantified by the Q-statistic (Newman), such that partitions with higher Q contributed more strongly to the final values in the agreement matrix.
We then applied a consensus clustering approach to this agreement matrix, as described by L + F ().
Specifically, we first applied a threshold tau = 0.1 to the matrix, ran the Louvain algorithm 1000 times, computed a new agreement matrix and thresholded the data again.
We repeated this procedure until all elements in the agreement matrix had a value of either 1 (i.e., the pair of nodes was always classified in the same module) or 0 (i.e., the pair of nodes was never classified in the same module).
The module assignments of this final agreement matrix represent a consensus estimate of the community structure of the network.
[[TODO: check these details but its about right from memory. My only concern is that if you weighted the values in the agreement matrix by the quality of the partition, the interpretation of tau becomes tricky - in the unweighted case, it is simply the proportion of runs in which two nodes are in the same module. in the weighted case, its less clear]]


% METHODS: gene expression
\subsection*{Gene expression}
Gene expression in C. elegans neurons is measured as a binary indicator from individual experiments, and compiled into a unified data resource on WormBase \cite{Harris:2009kd}.
The database is regularly updated; here we used release WS256 (\url{ftp://ftp.wormbase.org/pub/wormbase/releases/WS256/ONTOLOGY/anatomy_association.WS256.wb}, in which 948 genes were expressed in at least one neuron.
% downloaded on 6th Feb 2017
Expression annotations are made directly to individual neurons, and indirectly to groups or broader classes of neurons (e.g., `head', `interneuron', etc.) [[TODO: add some detail about this? We investigated and compared annotation types in some detail.]], with different levels of certainty: `certain' (means what?), `enriched' (means what?), `partial' (means what?), and `blank' (means what?) [[TODO: Aurina, what do all of these mean? Are there more possible annotations? We must describe here how these labels are ascribed, and our justifying for including these data -- e.g., why include `blank' genes. Must be clear how the decision to label `on' versus `off' is made in all cases, and our justification for our choices...]]
The main analyses presented here are of expression data that is annotated directly to individual neurons, and with one of the following qualifiers: `certain', `enriched', `partial', or `blank' [[TODO: Aurina, check? We need to justify why we picked these categories to include]].

The resulting binary expression data for 279 neurons and 948 genes is shown as a matrix in Fig.~\ref{fig:SchematicRepresentation}A.
An average of 30 genes are expressed in each neuron (range: 3 to 138 genes); each gene is expressed in an average of 9 neurons (range: 1 to 148 neurons).

As well as being unable to distinguish relative expression levels between neurons, we also cannot distinguish between the following two cases: (i) ``gene is not expressed'' and (ii) ``there is no information on whether gene is expressed'', which are both represented as a ``0'' in the expression data used here.

% Could add information about indirect annotations

% METHODS: gene coexpression
\subsection*{Gene coexpression}
As shown in Fig.~\ref{fig:SchematicRepresentation}A, we have a gene expression profile for each neuron that indicates which of 948 genes have been observed to be expressed in that neuron.
Our primary aim in this work is to understanding how gene expression related to pairwise patterns of connectivity.
We therefore analyzed patterns of coupled gene expression between pairs of neurons.
% rather than the presence or absence of gene expression in any single neuron on its own.
To this end, we computed the similarity in gene expression profiles between a pair of neurons, $(i,j)$, using a binary analogue of the linear Pearson correlation coefficient, the mean square contingency coefficient, $r_\phi$ \cite{Warrens2008}, [[AF: replace labeling using $\phi$ because of RC coefficient... But this is standard notation... What instead? Could use $\rho$?]]
\begin{equation} \label{eq:rphi}
    r_\phi = \frac{n_{11}n_{00} - n_{10}n_{01}}{\sqrt{n_{1\bullet}n_{0\bullet}n_{\bullet 0}n_{\bullet 1}}},
\end{equation}
for two vectors $x$, $y$, of length $L$, where $n_{xy}$ count the number of observations of each of the four outcomes (e.g., $n_{10} = \sum_i \delta_{x_i,1}\delta_{y_i,0}$ counts the number of times $x=1$ and $y=0$ across the length of $x$ and $y$), while the symbol $\bullet$ sums across a given variable (e.g., $n_{\bullet 0} = \sum_i \delta_{y_i,0}$ counts the number of times $y = 0$).
A maximum value $r_\phi = 1$ when $x$ and $y$ are identical such that $n_{11} + n_{00} = L$, and a minimum value $r_\phi = -1$ when $x$ and $y$ are always mismatched such that $n_{10} + n_{01} = L$.

Measures of correlation between binary vectors can be biased the relative proportion of ones between two vectors.
The gene expression data from WormBase is sparse (in part due to incomplete annotation data), with neurons ranging from between 3 to 138 expressed genes (from a total of 948 genes).
To ensure that our measure of gene coexpression between neurons was not biased by differences in the relative proportions of expression annotations, we conducted a sensitivity analysis in which we compared $r_\phi$ metric (Eq.~\ref{eq:rphi}), with two other binary correlation metrics, Yule's Q coefficient, and the Jaccard index, as well as a new metric developed here, based on positive matching probabilities (described in Supplementary Information).
The only metric that did not show a systematic bias on random binary vectors as a function of the proportion of ones was $r_\phi$ (see Fig.~\ref{fig:S_propOnes}), which was therefore used as a coexpression metric in this work, ensuring that increases in coexpression are driven by relative similarity in expression patterns, and not simply by differences in the number of gene annotations for a given pair of neurons.

A total of 92 anatomically distinct neuron classes are bilateral, that is, they contain both left [[TODO: left what? Is there a correct terminology?]] and right counterparts (e.g., AVAL/AVAR).
These neurons exhibit highly correlated gene expression patterns, with a mean coexpression value $\langle r_\phi^{\mathrm{intraclass}} \rangle = 0.98 (SD\pm0.0.25)$.
% [[CODE used - testLeftRightCoexpression.m. Distribution for these links is not normal. Median is 1;]]
To ensure that our analyses are not influenced by the high coexpression between bilateral pairs of neurons of the same class, we excluded coexpression values within neuron classes from all coexpression analyses.

\paragraph{Spatial effects}
Previous work has emphasized the importance of spatial effects in driving patterns of gene expression, whereby proximal brain areas typically exhibit more similar gene expression patterns \cite{Pantazatos:2016ir, Krienen2016, Fulcher:2016ck}.
Connection probability depends on spatial separation in mammalian brains \cite{Henderson:2014fg, Horvat:2016ia, Wang:2016gg, Markov:2013jo}, and in \emph{C. elegans} \cite{Azulay:2016cg}.
We verified that the probability of a pair of neurons being connected decreases as a function of their separation distance for connections within the head, connections within the tail, connections within the body, and connections between the head and tail (cf. Fig.~\ref{fig:S_connProbability}A,B).
It is thus important to correct for spatial effects to ensure that relationships between gene expression and connectivity are not simply the result of both properties showing spatial dependencies.

Gene coexpression, $r_\phi$, decreased with the separation distance between pairs of neurons for the spatially close networks of neurons within the head and within the tail of \emph{C. elegans} (Fig.~\ref{fig:S_distCoexp}A,B), but not for pairs of neurons involving the body (Fig.~\ref{fig:S_distCoexp}C) [[add some interpretation in terms of the function of body neurons as potentially distinct from those in the head and tail]].
This result, using sparsely-annotated binary expression data in a highly specialized neuronal nervous system, mirrors that found in the bulk regions of macroscopic mammalian brains of mouse \cite{Fulcher:2016ck} and human \cite{Krienen:2016eq}.
The result was not isolated to just a single class of connections within the head, with an approximately exponential relationship exhibited by pairs of interneurons, pairs of sensory neurons, pairs of motor neurons, and motor-interneuron pairs (Fig.~\ref{fig:S_distCoexp}).
Although an exponential form fits the mean coexpression as a function of separation distance well, the trend is driven by spatially close pairs of neurons with high coexpression, such that taking residuals after subtracting the trend line (as \cite{Fulcher:2016ck} for mouse data) leaves artifacts.
Future data that allow more accurate and continuous measurement of gene coexpression, may allow more accurate spatial correction to be applied more accurately.
[[TODO: add some comment verifying that, despite issues, if we do correct within the head and tail for the spatial relationship we get similar results, cf. \emph{CoExpressionDistanceCorrect}]]
% However, because we found no correlation between Euclidean distance between a pair of neurons (based on their two-dimensional coordinates) and their gene coexpression in the C. elegans nervous system (Fig.~\ref{fig:S_distDep}), no correction was necessary here.
% In \textit{C. elegans}, we found a semi-exponential relationship between connection probability and Euclidean separation distance (based on their two dimensional coordinates) between a pair of neurons, but no correlation between the Euclidean separation distance between a pair of neurons and their gene coexpression, $r_\phi$.

\subsection*{Gene enrichment analysis}
To determine whether certain sets of genes drive pattern of coupled expression between pairs of neurons, we used enrichment analysis to identify functionally annotated groups of genes from the Gene Ontology (GO) \cite{Ashburner2000}.
%  that contribute to the increased coexpression between:
% (i) connected pairs of neurons as compared to unconnected pairs as well as
% (ii) connections involving hubs (rich and feeder) compared to connections that do not involve hubs (peripheral).
% To this end, we first assigned a gene coexpression contribution (GCC) score to each inter-region pair, $(i,j)$, for each gene, $a$, as $\mathrm{GCC}_{ij}^{(a)}$, that quantifies the contribution of each gene to the coexpression score for a given pair of brain regions ($i$ and $j$).
In previous work involving continuous expression data, we developed a score for each gene that quantified its contribution, at each edge, $(i,j)$, to the overall Pearson correlation across all genes \cite{Fulcher:2016ck}.
With the current binary expression data, expression of a given gene, $a$, across brain regions, $i$, can take three possible combinations of coexpression at each edge, $(i,j)$:
(i) \emph{both expressed}: $g^{(a)}_i = 1$ and $g^{(a)}_j = 1$;
(ii) \emph{mismatched}: $g^{(a)}_i = 1$ and $g^{(a)}_j = 0$ or $g^{(a)}_i = 0$ and $g^{(a)}_j = 1$; or
(iii) \emph{neither expressed}: $g^{(a)}_i = 0$ and $g^{(a)}_j = 0$.
Our measure attempts to score individual genes by their patterns of expression in different types of inter-region pairs (e.g., connected versus unconnected pairs, or rich/feeder/peripheral connections).
In particular, for a given connection class, such as the set of all pairs of connected neurons, we quantified whether a given gene, $a$, is more likely to be expressed in both neurons of that class than expected by chance.
This was computed as a cumulative binomial probability of obtaining at least $m$ matches ($g^{(a)}_i = 1$ and $g^{(a)}_j = 1$) observed empirically:
\begin{eqnarray}
	\label{eq:CBinomialProbability}
     \nu^{(a)} = P(X > m) = 1 - \sum_{i=0}^{m}\binom{n}{i} p^{i}(1-p^{n-i}),
\end{eqnarray}
where $p$ is the probability of the given class of inter-region pairs ($p = n_\mathrm{class}/M$ for $n_\mathrm{class}$ connections out of $M$ possible pairs), and $n$ is the maximum number of possible matches.
This provides us with a score, $\nu^{(a)}$, for each gene, $a$, that is higher for genes that shows a greater number of matches in the inter-region class of interest than expected by chance.

To take a concrete example, when comparing differences in binary coexpression between pairs of neurons that are structurally connected and pairs that are unconnected, $p$ is computed as the proportion of all possible connections that exist, whereas when comparing links involving hubs compared to those between nonhubs, $p$ is the proportion of connections involving hubs.
% , and captures the probability of genes being expressed together in connected pairs of neurons at least as frequently as observed

% For each gene, we first determined the pattern of matching expression profiles (a match meaning that a gene is expressed by both neurons) between each pair of neurons resulting in a binary 279 x 279 matrix.
% Then, for each gene we calculated a cumulative binomial probability of getting more matches $(X)$ than in the empirical data $(m)$ on a particular connection type (see eq. \ref{eq:CBinomialProbability}).

To ensure that each gene contributed a meaningful score, we required more than one pair of regions $m > 1$ to exhibit matching expression ($g^{(a)}_i = 1$ and $g^{(a)}_j = 1$) in the inter-region class of interest.
A total of 577 genes out of 948 satisfied this criterion for connected/unconnected analysis; 390 out of 948 genes satisfied this criterion for hub connectivity analysis.
% When calculating GCC for connected links more then one match for a gene overall was required (as a result 577 genes out of 948 left); when calculating GCC for the links involving hubs (rich and feeder links) more than one match was required on connected links (as a result 390 out of 948 genes left).
% Otherwise, no information about the contribution of a specific gene to the overall coexpression pattern could be extracted.
% \textbf{(NOTE: not literally true as both on and off matches equally contribute to an increase in coexpression, - need to find a way how to explain, why non-expression matches are not taken into account here).}

For a given enrichment analysis, after computing a probability score for each genes using Eq.~\eqref{eq:CBinomialProbability}, we then took negative logarithms of the probability scores, $\log_{10}\nu^{(a)}$, and used them to perform gene score resampling (GSR) analysis using version 3.0.2 of \emph{ErmineJ} software \cite{Gillis2010} for biological process, molecular function and cellular components separately using Gene Ontology (GO) annotations \cite{Ashburner2000} obtained from GEMMA \cite{Zoubarev2012} (\texttt{Generic\_worm\_noParents.an.txt.gz} downloaded on March 31, 2017).
% downloaded using a build-in option within ErmineJ software
Rather than setting an arbitrary threshold on the score, this analysis allows us to use the continuous log-transformed probability scores, and compute the probability that the mean score in each GO group is larger than expected from a random assignment of scores to the genes in the list.
Gene Ontology terms and definitions were obtained in RDF XML file format downloaded from \url{archive.geneontology.org/latest-termdb/go_daily-termdb.rdf-xml.gz} on March 31 2017.
We included ontology terms containing between 5 and 100 annotations, using full resampling with $10^{5}$ iterations [[can do $10^{7}$ runs if needed]].
% {archive.geneontology.org/latest-termdb/go \textunderscore daily-termdb.rdf-xml.gz}

% Place figure captions after the first paragraph in which they are cited.
% \begin{figure}[!h]
% \caption{{\bf Bold the figure title.}
% Figure caption text here, please use this space for the figure panel descriptions instead of using subfigure commands. A: Lorem ipsum dolor sit amet. B: Consectetur adipiscing elit.}
% \label{fig1}
% \end{figure}

% Results and Discussion can be combined.
\section*{Results}

We aim to quantify patterns in the pairwise synaptic connectivity of \emph{C. elegans} that relate to patterns in the pairwise similarity in gene expression (gene coexpression).
A schematic overview of our data is shown in Fig.~\ref{fig:SchematicRepresentation}, which includes the directed binary synaptic connectome (Fig.~\ref{fig:SchematicRepresentation}A), binary gene expression across 948 genes (Fig.~\ref{fig:SchematicRepresentation}B), and resulting gene coexpression between all pairs of neurons (Fig.~\ref{fig:SchematicRepresentation}C).
Edges in the connectome are colored according to how they connect hubs ($k > 42$) and non-hubs ($k \leq 42$), as `rich' (hub $\rightarrow$ hub), `feed-in' (nonhub $\rightarrow$ hub), `feed-out' (nonhub $\rightarrow$ hub), and `peripheral' (nonhub $\rightarrow$ nonhub).

We present our analysis in three parts:
(i) we analyze the synaptic connectivity properties, demonstrating a rich-club organization of hubs,
(ii) we show that gene coexpression is increased in connected pairs of neurons relative to unconnected pairs, and varies also as a function of edge type, with the most similar gene expression patterns found in pairs of connected hub neurons (mirroring previous results in mouse \cite{Fulcher:2016ck}),
(iii) despite relatively incomplete gene annotations, we show that these transcriptional signatures may be driven by glutamate receptor signaling, and that the results are not driven simply by stereotypical interneuron expression or modular organization.
% Consistent with previous results in the mouse \cite{Fulcher:2016ck}, we show that synaptically connected neurons display more similar gene expression than those that are unconnected, and that, amongst connected regions, those involving hub neurons show more similar gene expression.

% ------------------------------------------------------------------------------
% <<>>
% ------------------------------------------------------------------------------
\begin{figure}[t]
  \centering
    \includegraphics[width=1\textwidth]{schematic.pdf}
 \caption{\textbf{Schematic data representation}
 \textbf{A} Binary synaptic connectivity between all pairs of neurons, colored according to type: rich, feeder, and peripheral.
  \textbf{B} Binary gene expression across the 948 genes that are expressed in at least one neuron.
 \textbf{C} Gene coexpression matrix, computed for all pairs of neurons as the mean square contingency coefficient, $r_\phi$.
Neurons have been ordered by their position, from head (upper) to tail (lower).
[[TODO: I think it would be nice to put neuronal metadata on the edge (if clear enough to read)]]
[[TODO: check that we have copyright to use this proprietary worm picture in this figure]]
}
\label{fig:SchematicRepresentation}
\end{figure}
% ------------------------------------------------------------------------------


%%
\subsection*{Hub connectivity in the C. elegans connectome}
%% INTRODUCE RICH CLUB  hene and focus on the difference between rich/feeder/peripheral links below
[[TODO: Shorten section, focusing on reproducing Towlson]]
% \paragraph{Rich club organization of the connectome}

% <<hubs>>
First, we analyze the topological properties of the \emph{C. elegans} connectome, represented here as a directed, binary connectivity matrix of the 1961 chemical synapses between 279 non-pharingeal neurons \cite{Varshney2011}, focusing particularly on hub connectivity.
The degree distribution of the \emph{C. elegans} synaptic connectome is shown in Fig.~\ref{fig:topology_rich}A, where neurons have been distinguish according to type: 68 sensory neurons, 85 interneurons, 108 motor neurons, and 18 multimodal neurons.
Consistent with previous work \cite{Towlson:2013gf}, we see a positively-skewed degree distribution containing an extended tail of high-degree hubs, the majority of which are interneurons [[characterized in previous work as control INs]].

% Here we confirm the high connectivity, centrality and wiring cost of hub connections in the C elegans connectome, as previoulsy shown by Towlson et al () and as also found in the macaue (Harriger) and human (van den Huevel-PNAS) brains.
% We show RC organization... and show hubs are interneurions...
% We show cost and centrality (may be worth including a curve for betweenness)


% ------------------------------------------------------------------------------
% <<TopologyFigures.m>>
% ------------------------------------------------------------------------------
\begin{figure}[h]
   \centering
    \includegraphics[width=0.9\textwidth]{topology_rich.pdf}
    % \includegraphics[width=0.7\textwidth]{RCcurve.eps}
 \caption{\textbf{Rich-club organization of the connectome}.
% A: Topological rich-club. B: Mixed rich-club. C: Weighted rich-club.
\textbf{A} Degree distribution of the binary chemical connectome, where neurons are labeled according to four categories:
(i) interneuron (85 neurons, orange),
(ii) sensory (68 neurons, blue),
(iii) motor (108 neurons, green), or
(iv) multiple assignments (18 neurons, yellow).
An extended tail containing neurons with relatively high degree is evident in the distribution.
Most of these neurons are interneurons.
\textbf{B}
Normalized rich club coefficient, $\Phi_\mathrm{norm}$ (red), as a function of the degree, $k$, at which hubs are defined (as neurons with degree $>k$).
Also shown is the mean Euclidean separation distance (yellow) between connected hub regions (across degree thresholds, $k$).
$\Phi_\mathrm{norm} > 1$ indicates that hubs are more densely interconnected among each other than expected by chance, with red circles indicate values of $\Phi_\mathrm{norm}$ that are significantly higher than an ensemble of 1\,000 degree-matched null networks ($P < 0.05$).
Yellow circles indicate where the Euclidean distance between connected pairs of hubs is significantly greater than the Euclidean distance for all other pairs of connected regions (Welch's $t$-test, $P < 0.05$).
[[TODO: Avoid yellow to mean two different things within the same plot (either change distance or the multiple assignment color) AF: yeah use light blue, cyan or magenta/violet instead of yellow. Also suggest replacing red with one of these, since red is used in coexpression plot to represent something different.]]
[[TODO: Add right-side vertical-axis to give units for distance.]]
}
 \label{fig:topology_rich}
 \end{figure}
% ------------------------------------------------------------------------------


% ------------------------------------------------------------------------------
% <<rich-club organization>>
% ------------------------------------------------------------------------------
We quantified the extend to which hubs are densely interconnected by computing the normalized rich-club coefficient, $\Phi_\mathrm{norm}$, with $\Phi_\mathrm{norm} > 1$ indicating rich-club organization of the network.
The variation in $\Phi_\mathrm{norm}$ across a range of thresholds, $k$ (at which hubs are defined, as neurons with degree $>k$), is shown in Fig.~\ref{fig:topology_rich}B, with red circles indicating a significant increase in link density among hubs relatively to 1\,000 degree-preserving nulls (permutation test, $P < 0.05$) [[check whether randmio\_dir only preserves degree]].
The plot reveals rich-club organization at the upper tail of the degree distribution, particularly for thresholds $41 < k < 58$, referred to here as the `topological rich-club regime', shaded gray in Fig.~\ref{fig:topology_rich}B.
% This indicates that when nodes are defined as hubs (degree $>\textit{k}$) in this range, they are more densely interconnected than expected by chance.
Most of the analyses presented below examine how different topological and genomic properties vary as a function of the hub threshold, $k$, but for analyses requiring a fixed hub definition, we define hubs as the 13 neurons with $k > 41$.
% Rich club in the chemical synapse network consists of 13 high degree neurons with degree $42 \leq k \leq 98$.


[[TODO-AA: Add a couple sentences comparing to Towlson. Maybe include a list with degrees and definitions in SUPP (like in Towlson's paper?). Also note whether RC curve same or different to Towlson.]]

% ------------------------------------------------------------------------------
% <<distance, geometric effects>>
% ------------------------------------------------------------------------------
Recent work has shown that many aspects of connectome organization can be partially accounted for by geometric effects, by treating the connectome as a spatially-embedded object \cite{Henderson:2014fg, Roberts2016, Horvat:2016ia}.
Unlike network analyses of brains, where all neurons are confined to a single, spatially contiguous organ, neurons of the \emph{C. elegans} nervous system are distributed throughout the entire organism, forming a dense cluster of 147 neurons in the head (all within 130\,$\mu m$), 105 sparser, mostly motor neurons (75\%) in the body (spanning 1.02\,mm), and a relatively dense cluster of 27 neurons in the tail (all within 90\,$\mu$m of each other), see Fig.~\ref{fig:S_neuronsSpace} for a plot of all neurons in space.
% <<cf. NeuronTypeStats.m to run these numbers>>
In Fig.~\ref{fig:topology_rich}B, we plot the mean distance between all pairs of connected hubs at each degree threshold, $k$, finding an increase in mean hub-hub connection distance with $k$, through to the extent of the topological rich-club regime.
This can be attributed to a relative increase in long-distance hub-hub connections between the head and tail (Fig.~\ref{fig:S_connProbability}C).
Despite an increase in hub-hub connection density across longer distances, connection probability decreases with distance in \emph{C. elegans} for connections within the head, body, and tail, and between the head and tail (see Figs.~\ref{fig:S_connProbability}A,B), mirroring recent results of mesoscale mammalian brains, in mouse \cite{Goulas:2016hr, Fulcher:2016ck}, and other rodents and primates \cite{Horvat:2016ia}.
Taken together, our results demonstrate that hub-hub connections in \emph{C. elegans} are dense, and extend over significantly longer distances than other types of connections in the synaptic connectome, consistent with the idea of the rich club as a costly but central backbone for neuronal communication \cite{vandenHeuvel:2012kh}.

% In connectomes across species and scales, degree is distributed unequally, with a small number of highly connected brain regions, known as hubs \cite{Sporns:2007ea}, that are themselves more densely connected between themselves, forming a so-called `rich club' \cite{deReus:2013cy, ZamoraLopez:2010hy, Shih:2015cu, vandenHeuvel:2012kh, vandenHeuvel:2011he}.
% The same is true in the C. elegans neuronal nervous system, analyzed in the past as an undirected binary connectome including both chemical and electrical synapses \cite{Towlson:2013gf}, which exhibits highly connected, and densely interconnected hub neurons.

%%
\subsection*{Coexpression and connectivity}

% ---connectivity/coexpression---
Having characterized the topological and geometric properties of connectivity, in particular the rich-club organization of hub neurons, we next investigate how connectivity properties relate to patterns of neuronal gene expression.
Being a pairwise analysis, we compare pairwise synaptic connectivity to pairwise similarity in gene expression using the mean square contingency coefficient, $r_\phi$ (see \emph{Methods}).
% , to define gene coexpression, and excluding coexpression values between homologous left/right neuron pairs, see Methods).

We first investigate whether gene coexpression relates to synaptic connectivity by comparing the distribution of $r_\phi$ computed for all connected pairs of neurons, and for all unconnected pairs of neurons, shown in Fig.~\ref{fig:coExp}A.
We find that connected pairs of neurons have more similar expression profiles than unconnected pairs (Wilcoxon rank-sum test, $P < 10^{-50}$), with no difference between unidirectional ($X \rightarrow Y$) and reciprocal ($X \leftrightarrow Y$) connections (see Fig.~\ref{fig:S3_Fig}).

% ------------------------------------------------------------------------------
% <<coexpConnUncon.m>>
% <<RichClubMedian.m>>
% ------------------------------------------------------------------------------
 \begin{figure}[h]
 \centering
    \includegraphics[width=1\textwidth]{MeanCoexpression.pdf}
 \caption{{\bf Gene coexpression varies as a function of connectivity, and as a function of connection type, with connections involving hubs being most similar.}
\textbf{A} Distribution of transcriptional similarity between connected (1940 pairs) and unconnected (36\,749 pairs) pairs of neurons as a violin plot, with the median of each distribution shown as a horizontal line. Connections between symmetrical (left/right) pairs of neurons are excluded from the analysis.
Coexpression, $r_\phi$ is increased in connected pairs of neurons ($P < 10^{-50}$, Wilcoxon rank sum test).
\textbf{B}
\emph{Top}: Degree distribution.
\emph{Middle}: proportion of connections that are `rich' (hub$\rightarrow$hub, red), `feed-in' (nonhub$\rightarrow$hub, yellow), `feed-out' (hub$\rightarrow$nonhub, orange), and `peripheral' (nonhub$\rightarrow$nonhub, blue) as a function of the degree threshold, $k$, used to define hubs.
Note that at high $k$, most neurons are labeled as nonhubs, and hence the vast majority of connections are `peripheral'.
\emph{Lower}: Mean gene coexpression, $r_\phi$, for each connection type as a function of $k$.
The mean coexpression across all network links is shown as a dotted black line; the topological rich-club regime (determined from the network topology, cf. Fig.~\ref{fig:topology_rich}) is shaded gray.
Circles indicate a statistically significant increase in gene coexpression in a given link type relative to the rest of the network (one-sided Welch’s t test; $P < 0.05$).
[[TODO: A: add $r_\phi$ to ylabel.
B: Remove box from degree distribution, Change y-label to `Mean gene coexpression', fonts are small, and inconsistent sizes between subplots.
I think we should crop the curves when there are fewer than X connections remaining -- in this case it's a bit ridiculous to show that constant feed-out/feed-in/peripheral coexpression when there are only two neurons
left as hubs]]
[[TODO: AF: Use colors in A. Use black for distribution in B.]]
}
 \label{fig:coExp}
\end{figure}
% ------------------------------------------------------------------------------

% ---HUB coexpression---
Having established an increase in gene coexpression in connected pairs of brain regions (relative to unconnected pairs), we next investigated whether coexpression varied between different types of connections, focusing particularly on the role of densely interconnected hub nodes characterized above.
Given the importance of hub neurons across species [[Ref]], and previous results in the mesoscale mouse connectome \cite{Fulcher:2016ck}, we expect increased coexpression in connections involving hubs.
To investigate this effect, we first labeled each neuron as either a hub (nodes with degree $> k$) or a nonhub (degree $\leq k$), for a given hub threshold, $k$, and then labeled each connection as either `rich' (hub $\rightarrow$ hub), `feed-in' (nonhub $\rightarrow$ hub), `feed-out' (hub $\rightarrow$ nonhub), or `peripheral' (nonhub $\rightarrow$ nonhub).
[[TODO: Establish whether we're computing pairs of hubs (e.g., for `rich'), or for every connection between any pair of hubs -- i.e., do reciprocal connections contribute twice to the distribution at each $k$?]]
The mean coexpression for each of these four connection types is plotted in Fig.~\ref{fig:coExp}B, with circles indicating statistically significant increases of a given connection type (relative to all other connections). 
We see that gene coexpression in rich connections increases with degree, reaching a plateau through the topological rich-club regime where hubs are densely interconnected (shaded in Fig.~\ref{fig:coExp}B).
Interestingly, we see a split between feed-in and feed-out connections, with feed-out connections showing increased gene coexpression through the topological rich-club regime, while feed-in and peripheral connections show the lowest levels of coexpression.
% We observed a difference in gene coexpression between feed-in and feed-out links, where feed-out links manifested increased coexpression for a broad range of degrees while feed-in links showed only a slight increase for very high hub thresholds.

These results, using partial binary expression estimates in 948 genes in a neuronal connectome, are broadly consistent with results in 213 regions of the mesoscale mouse connectome with over 17\,000 genes, in which we see an increase in coexpression in connected pairs of neurons, and the greatest coexpression amongst hub-hub connections and the lowest coexpression amongst nonhub-nonhub connections.
In contrast to the mouse connectome, however, in which hubs were distributed across a broad range of anatomical divisions, hubs in C. elegans are [[mostly. There is one motor neuron in there, right?AA: with threshold k>41, all interneurons]] interneurons \cite{Towlson:2013gf} (cf. Fig.~\ref{fig:topology_rich}A).
To determine the specificity of the effect between different neuron types, we split our coexpression analysis into connections involving: (i) interneurons, (ii) sensory neurons, and (iii) motor neurons, computing the mean coexpression within each class of connections as a function of the degree of the given neuron type
[[TODO: is this true? It's kind of complicated to explain -- is it right that for a given $k$ threshold we include, say, sensory neurons of degree greater than $k$ and their connections to all other neurons in the `sensory' category? Or do we include all neurons with degree $>k$ and compute the mean of those connections that involve both the high-degree neurons AND neurons of the given class? This would be another choice. Perhaps it's the latter that's plotted?YES: now we include connections for `sensory' deg$>k$ to any other other neurons with degree >k - my code is is really complicated - probably there is much easier way to do the same]], as shown in Fig.~\ref{fig:interneuron_dep}A.
Note that the three categories are not mutually exclusive; for example, the `interneuron' class includes connections between an interneuron and any other neuron.
We find that connections involving interneurons are unique in their increasing average coexpression with increasing degree, $k$.
Connections involving motor neurons, and connections involving sensory neurons, do not show any clear increase with degree.

\begin{figure}[h]
\centering
   \includegraphics[width=1\textwidth]{TypeDegree.pdf}
 \caption{
\textbf{The increase in coexpression with degree is specific to high-degree interneurons, and hub interneurons have higher coexpression than nonhub interneurons.}
\textbf{A} \emph{Upper}: The number of connections involving interneurons (orange), sensory neurons (green), and motor neurons (blue) across degree threshold, $k$ [[TODO: Aurina, is it correct that sensory is green and motor is blue?AA: NO - motor are green, sensory blue (I changed RC degree distribution colors in the PlotRichClub.m script accordingly)]].
\emph{Lower}: Average gene coexpression as a function of degree for connections involving different types of neurons.
\textbf{B} Coexpression distributions for all pairs of hub interneurons (red) and all pairs of non-hub interneurons (gray).
[[TODO: Did we exclude multimodal neurons from this analysis?? Can we do a statistical comparison between these distributions and add something here on that?]]
[[TODO: A: the vertical axis in lower subplot should state `mean gene coexpression']]
[[TODO: Both coexpression axes should be the same `Gene coexpression, $r_\phi$']]
[[TODO: AF: Plot makes it seem like there are no edges for $k>35$ -- Aurina can you try the log plotting I suggested -- will have to be three separate lines in semilogy]]
 \label{fig:interneuron_dep}
}
\end{figure}

Having found that interneurons drive the gene coexpression relationship with degree, $k$, we next verified whether hubs showed a unique transcriptional signature among this neuronal subtype.
We compared gene coexpression between all pairs of hub interneurons [[TODO: what $k$ threshold?]] and between all pairs of nonhub interneurons (AVFL, AVFR, AVHL, AVHR, AVKL, AVKR, AVJL, AVJR) [[TODO: what does this list represent? Not clear from the sentence structure]], as shown in Fig.~\ref{fig:interneuron_dep}B.
% that share similar anatomical properties in order to ensure that increase in coexpression for rich links is not determined by the fact that the majority of hub neurons belong to the same anatomical neuron type (???HOW TO CALL COMMAND INTERNEURONS???).
% In order to compare transcriptional similarity between all pairs of neurons we ignore the existence (or absence) of connections between these neurons,
Pairs of hub interneurons display significantly higher gene coexpression than pairs of nonhub interneurons ([[TODO: statistical test, p-value, etc.]]).
Thus, the increase in coexpression amongst hub interneurons is not simply due to most hubs being interneurons;  even amongst interneurons, [[TODO: isn't this obvious from plot A? It's very similar -- at high degree, we're only looking at hub interneuron pairs, which are more similar than others. Not sure B adds much -- we could just quantify the difference at a given $k$ threshold to put a statistic to it...?AA: it adds inf, because we're comparing hubs to interneurons with the same anatomical properties, not just random low degree interneurons. PLOT A says - effect driven by interneurons (other groups of neurons do not show this increase with degree), plot B says - among interneurons that are anatomically the same (hub interneurons and 8 interneurons that are not hubs), hubs have higher coexpression. Aurina -- can you rephrase to make this all clear in the main text. AF was confused.]].

\subsection*{Other influences}
Having demonstrated that gene coexpression is elevated for connected regions, and particularly for connected pairs of hubs, we next investigate whether other anatomical, neurotransmitter, or connectomic organizational influences might contribute to our results.
% , namely modular organization, lineage similarity, or neurotransmitter type, could explain our results.

% \paragraph{Distance}).
% Implementing the same methods as previously used in gene coexpression analysis at each degree threshold for each link type (rich, feeder, peripheral) we calculated the mean connection distance.
% Connection distance increased with increasing degree for both rich and feeder links while peripheral connections demonstrated no increase through the whole range of thresholds (Fig.~\ref{Distance}).
% Given this result, it is safe to say that increased coexpression for hub-related links is not determined by lower connection distance between them as the contrary is shown to be true.

%\begin{figure}[!h]
%\centering
 %   \includegraphics[width=0.7\textwidth]{distance_k}
 %\caption{{\bf Connection distance as a function of degree}}
 %\label{Distance}
 %\end{figure}

\paragraph{Modular organization}
We first consider the modular organization of the C. elegans connectome.
Like many other connectomes, the C. elegans nervous system contains modules, groups of neurons that are densely interconnected \cite{Kim2014, Bassett2010}, and modules can also be found of similarly connected neurons that correspond to specific functional circuits \cite{Achacoso:1992ay, Pavlovic2014}.
Given the more uniform functional role assumed of neurons within a module (and hence characteristic gene expression), we test the possibility that hubs belonging to the same module (which carries a characteristic gene expression profile) may drive the increased gene coexpression of hub-hub connections characterized above.
 and thus might drive the increased gene coexpression of hubs, we used the Louvain community detection algorithm to extract modules from the binary \emph{C. elegans} synaptic connectome (using consensus clustering, see \textit{Methods}).
The four resulting modules are shown in Fig.~\ref{fig:modules}A, in which hubs are predominantly placed in a single module (labeled green in the module indicator column), that contained sensory, motor, and interneurons spanning the head, tail, and body (see other indicator columns in Fig.~\ref{fig:modules}A).
% In this analysis, hubs are mostly concentrated in a single module (purple in Fig.~\ref{fig:modules}), but are also distributed across three other modules.
% The algorithm detects three comparably sized modules (M1-M3), each containing diverse neuron types, as well as a smaller fourth module (M4), which contains 13 motor neurons.
% Of the 13 hubs with $k \geq 42$, the majority (11) were assigned to M3, which spanned the length of the worm body, as pictured in Fig.~\ref{fig:modules}.
% As evident from the column labels to the left of the matrix, the hub-concentrated module is diverse, containing neurons across the head, tail, and body, as well as sensory, motor, and interneurons (see labels on Fig.~\ref{fig:modules}A).
% This module is comprised primarily of motor neurons, consistent with evidence that control interneurons are directly responsible for forward and backward locomotion (command interneurons).
% 10 out of 11 hub neurons in this module are directly responsible for
% The main goal of this analysis was to compare coexpression within and between modules in order to examine if neurons assigned to the same module are more genetically similar than neurons in other modules.
The expectation that gene coexpression, $r_\phi$, was increased within modules was not corroborated by the data.
In fact, gene coexpression was slightly lower for connections within modules than for connections between modules, as shown in Fig.~\ref{fig:modules}B ($p = 0.045$, Welch's $t$-test).
This result is not due to the deterministic Louvain algorithm; indeed we found an even larger decrease in intramodular coexpression using nine module partition of neurons derived from an Erd\"os-R\'enyi Mixture Model ($p = 2\times10^{-6}$) \cite{Pavlovic2014}.
Accordingly, looking within modules, we still found a significant increase in coexpression for hub-hub connections, with intra-modular connections between hubs were increased relative to other intra-modular connections (Welch's $t$-test, $p = 1.3\times 10^{-10}$).
We also computed an empirical null distribution for the mean coexpression for the 34 intramodular and 32 intramodular hub-hub links, by taking 1\,000 random samples from these two link types, yielding a null $r_\phi$ of $0.15 \pm 0.02$, compared to the observed value of $0.51$ ($p < 10^{-16}$).
We thus find that the increase in coexpression for hub-hub connections is not driven by the similar modular membership.
% Coexpression within modules did not exceed coexpression between modules, demonstrating that neurons that belong to the same module do not share any particular genetic similarity.
% The increased coexpression in connected pairs of hubs in \emph{C. elegans} can therefore not be attributed to the modular organization of the connectome.
% despite most hubs being assigned to the same module.
% [[NOTE: maybe not worth characterizing modules, just say: coexpression within modules is not higher than coexpression between modules on one sentence and distributions? When hub neurons are excluded, M2 and M3 coexpression is higher than M1, but no difference between M2 and M3]])

% [[TODO: this section required more justification. Why would we expect modules to drive hub effect? why is looking at modules in this context interesting? Unclear what the nulls are and how they were generated.]]

% ------------------------------------------------------------------------------
% <<plot_modules.m>>
% ------------------------------------------------------------------------------
\begin{figure}[!h]
\centering
    \includegraphics[width=1.2\textwidth]{Modules.eps}
 \caption{
 \textbf{Increased coexpression of hub neurons is not driven by their module membership}
%\textbf{A: Connection distance}
%Top: Degree distribution.
%Bottom: Median lineage distance for rich, feeder, and peripheral connections as a function of \textit{k}, with the %median across all network links shown as a dashed black line and the topological rich club regime shaded grey.
%Circles indicate a statistically significant change in lineage distance in a given link type relative to the rest %of the network (Wilcoxon rank sum test; $P < 0.05$). \\
\textbf{A} Connectome organized by module (white squares), with all connections labeled by type: rich (red), feed-in (yellow), feed-out (orange), and peripheral (blue).
Neuron indicators are shown to the left:
(i) module assignment,
(ii) neuron type (interneuron, orange; motor, green; sensory, blue; multimodal, yellow),
(iii) hub indicator (red for hubs), and
(iv) spatial position: head (gray), body (purple), tail (red).
\textbf{B} Distributions of gene coexpression, $r_\phi$, for intra-modular connections (black) and inter-modular connections (gray), shown as violin plots with the mean shown as a horizontal bar.
The number of connections of each type is shown in parentheses.
% \textbf{C} Violin plots of the distributions of $r_\phi$ for intra-modular hub-hub connections (black) compared to other intra-modular connections (gray).
%Composition of each module according to neuron type.
%Modular organisation of the connectome.
%Coexpression distributions within and between modules.
%\textbf{C: Lineage distance}
%Top: Degree distribution. Bottom: Mean lineage distance for rich, feeder, and peripheral connections as a function %of \textit{k}, with the median across all network links shown as a dashed black line and the topological rich club regime shaded grey.
%Circles indicate a statistically significant change in lineage distance in a given link type relative to the rest of the network (one-sided Welch’s t-test; $P < 0.05$) \\
%\textbf{D: Coexpression for different types of neurons. }
%Top: Median gene coexpression as a function of degree for connections involving different types of neurons.
%Bottom: The number of connections in each category for a range of degrees.
%Coexpression distributions for hub and non-hub interneurons. \\
[[TODO: Add labels for colors in matrix, bigger text]]
}
 \label{fig:modules}
\end{figure}


\paragraph{Birth time}
[[TODO: Add rationale for this analysis]]
[[TODO: Aurina, can you check this? Alex added: ``As first demonstrated by Towlson et al. \cite{Towlson:2013gf}'']], hubs are born early in the \textit{C. elegans} development \cite{Varier2011} (see Fig \ref{fig:S6_Fig}), therefore we next verified that the increase in coexpression between them can not be explained by the similarity in birth time. 
For this purpose, we separated neurons that are formed early (during the first 1000 min) in the development into two types, namely hubs and non-hubs, and compared coexpression within each of those groups (Fig \ref{fig:BirthTimeHubs}). 
Links between hubs show significantly increased coexpression and consequently confirm that even between the neurons that were born roughly at the same time, hubs exhibit a unique transcriptional pattern.

\begin{figure}[t]
  \centering
    \includegraphics[width=0.5\textwidth]{BirthTimeHubs.pdf}
 \caption{\textbf{Coexpression between early born hubs and early born non-hubs. COULD DO BORN EARLY OR BORN TOGETHER} ($P < 10^{-28}$, Wilcoxon rank sum test) or we could chow birth time diff curves and this one side by side (like with coexpression?))
}
\label{fig:BirthTimeHubs}
\end{figure}

\paragraph{Lineage similarity}
Lineage distance between pairs of neurons presents another source of information about the genetic makeup of the \textit{C. elegans} nervous system.
It has been shown that cells with identical fates can be formed by different gene regulatory pathways \cite{Liu2009}.
% however, to the best of knowledge, no lineage distance analysis was done in relation to the network properties of the \textit{C. elegans} connectome.
Implementing an analogous strategy as in our previous analysis of gene coexpression, we show that lineage distance between pairs of neurons increases with degree for feeder links while no significant change is observed for rich links (Fig.~\ref{Lineage}).
This finding suggests that the difference in lineage distance between hub and nonhub neurons increases with degree, [[therefore in a way confirming the relevance of this classification]][[TODO: Clarify???]].
Contrary, peripheral links manifest stably lower lineage distance through the range of degree thresholds wit (Fig.~\ref{Lineage}).
In addition, this shows that increase in gene coexpression with degree for rich links can not be attributed to their similarity in lineage.

Is it possible to say that connections within ACh are more similar when they involve hubs?

\subsection*{Functional enrichment}

%% comment on enrichment analysis. No write up can be done before deciding on options.

Having determined that pairs of neurons with different connectivity exhibit different levels of coexpression, we investigated which functional groups of genes contributed to these differences.
Despite lacking full genome coverage, and incomplete binary expression data, we developed a method to assign a gene coexpression contribution (GCC) score to each gene that quantifies its contribution to a given coexpression pattern (related to the probability of a gene being expressed together in a given set of neuron pairs compared to an alternative set).
We then performing an enrichment analysis using gene ontology categories for biological processes, molecular function and cell components, separately (see Methods).
Previous results in mouse \cite{Fulcher:2016ck}, using over 17\,000 genes, indicated that genes driving an increase in coexpression between connected pairs of neurons are enriched in genes related to basic neuronal function, whereas those driving an increase in connections involving hub neurons are enriched in oxidative energy metabolism (a result corroborated at a regional level in human \cite{Vertes2016a}).
However, [[TODO-AA: outline limitations in our data: we don't have gene coverage to check these things. What are results, and put name of script commented]].
[[Tweak the following to provide some hypothesis in light of data limitations:]]
Given that hub neurons are thought to be highly integrative, and disproportionately involved in mediating signal traffic \cite{Towlson:2013gf}, increased number communication and signal propagation related gene groups for hub-related links is in line with our expectation.

First, we estimated which functional groups of genes drive increased coexpression in connected pairs of neurons compared to unconnected pairs.
As listed in Table~\ref{tab:connected}, the majority of statistically significant GO categories are related to glutamate signaling and neuronal connectivity ($p_\mathrm{corrected} < 0.05$).
Next, we examined differences between connections in terms of their relationship to hub neurons, investigating which functional groups of genes contributed to the increased coexpression for links involving hubs (combining `rich' and `feeder' links together, for statistical power).
As well as glutamate signaling, we also found a broader range of functional categories, involving  ion channel activity, transmembrane transporters and synaptic cell components (see Table~\ref{tab:hub_enrichment}).

Although 11 out of the 13 hubs ($k > 41$) are cholinergic, and the remaining two use glutamate (Fig.~\ref{fig:S_feedInOutNodes}C), we found enrichment in glutamate signaling in comparing connections involving hubs to those between nonhubs.
% with ACh being the most broadly used neurotransmitter in general \cite{Pereira:2015er}
% It is worth noting that although both analyses above highlighted enrichment of glutamate, even though acetylcholine (ACh) is the most broadly used neurotransmitter in \textit{C. elegans} \cite{Pereira:2015er}, and most hubs ($k > 41$) are cholinergic (Fig.~\ref{fig:S_feedInOutNodes}C).
Looking closer at the neurotransmitters used by each neuron, we found that nonhub neurons that project to hubs (but do not receive projections back) are mostly sensory or interneurons (82\%), with the most common neurotransmitter being glutamate (41\%), whereas neurons that receive inputs from hubs (but do not project to hubs) are mostly motor neurons (84\%) and are cholinergic (93\%), as shown in Fig.~\ref{fig:S_feedInOutNodes}A,C.
Thus, although hubs are themselves mostly cholinergic, they receive inputs from glutamatergic neurons, which may explain their over-representation in two biological processes related to glutamate receptors: ionotropic glutamate receptor signaling pathway (GO:0035235) and glutamate receptor signaling pathway (GO:0007215).
[[TODO-AA: Still kind of surprising we don't get cholinergic, since ACh is still the dominant neurotransmitter for connections involving hubs. Can you look at cholinergic categories -- do we even have gene coverage to get power to find them??]]
% This may explain the overrepresentation of glutamate receptor related categories that we find in the enrichment analysis.
Thus, despite minimal gene coverage, we developed a new method that was able to find interpretable patterns in the existing binary expression data for \emph{C. elegans}, providing clues as to function of genes that may drive molecular differences between topologically different pairs of neurons.

%\begin{figure}[h]
%\centering
 %   \includegraphics[width=0.7\textwidth]{LineageRFP}
 %\caption{\textbf{}
 %\label{Lineage}
 %\end{figure}

% Place tables after the first paragraph in which they are cited.
% \begin{table}[!ht]
% \begin{adjustwidth}{-2.25in}{0in} % Comment out/remove adjustwidth environment if table fits in text column.
% \centering
% \caption{
% {\bf Table caption Nulla mi mi, venenatis sed ipsum varius, volutpat euismod diam.}}
% \begin{tabular}{|l+l|l|l|l|l|l|l|}
% \hline
% \multicolumn{4}{|l|}{\bf Heading1} & \multicolumn{4}{|l|}{\bf Heading2}\\ \thickhline
% $cell1 row1$ & cell2 row 1 & cell3 row 1 & cell4 row 1 & cell5 row 1 & cell6 row 1 & cell7 row 1 & cell8 row 1\\ \hline
% $cell1 row2$ & cell2 row 2 & cell3 row 2 & cell4 row 2 & cell5 row 2 & cell6 row 2 & cell7 row 2 & cell8 row 2\\ \hline
% $cell1 row3$ & cell2 row 3 & cell3 row 3 & cell4 row 3 & cell5 row 3 & cell6 row 3 & cell7 row 3 & cell8 row 3\\ \hline
% \end{tabular}
% \begin{flushleft} Table notes Phasellus venenatis, tortor nec vestibulum mattis, massa tortor interdum felis, nec pellentesque metus tortor nec nisl. Ut ornare mauris tellus, vel dapibus arcu suscipit sed.
% \end{flushleft}
% \label{table1}
% \end{adjustwidth}
% \end{table}


%PLOS does not support heading levels beyond the 3rd (no 4th level headings).

% \subsubsection*{3rd level heading}
% Text.
%
% \begin{enumerate}
%     \item{react}
%     \item{diffuse free particles}
%     \item{increment time by dt and go to 1}
% \end{enumerate}
%
%
% \subsection*{Another subsection}
%
%
% \begin{itemize}
%     \item First bulleted item.
%     \item Second bulleted item.
%     \item Third bulleted item.
% \end{itemize}

\section*{Discussion}
\begin{itemize}
    \item{First coexpression analysis in worm. Despite noisy gene expression data we get some insights into the genetic basis of connectivity on a neuronal level}
\end{itemize}

% COMPARE TO MOUSE
Compared to our previous work in the mesoscale mouse connectome \cite{Fulcher:2016ck}, the present findings in a near-fully mapped neuronal chemical connectome show similarities and differences.
Despite the incredible difference in spatial scale (from each node containing $\tilde 10^5$ neurons in mouse, to just a single neuron in worm), we found a similar basic qualitative signal of increased coexpression in connected pairs of neurons, and amongst connected pairs, the most similar expression signatures in pairs of connected hub neurons.
These overall findings were specific to connections involving interneurons, i.e., were not found in motor or sensory neurons, being driven by the relatively expression similarity of high-degree interneuron hubs, were not driven by similarity in lineage, neurotransmitter type, or modular membership, and were robust to a range of data processing choices (including connectome type, and coexpression measure).
These findings come despite sparsely annotated, binary gene expression data, with less than 5\% coverage of the full genome, with preliminary enrichment analysis indicating a role for glutamate in both connectivity and hub-connectivity.
Our results indicate that even in a highly specialized neuronal nervous system, costly hub-hub connections display a distinctive transcriptional signature that may reflect their unique functional role in the network.
Further work across different species and scales may shed light on deeper principles of how connectomes organize to facilitate efficient biological functioning.


differ significantly (being a neuronal connectome), and with so few neurons, each highly specialized, it may be hard to pick up broad expression patterns that are network topology-specific.
Quite different to the case in mouse, where regions were spread across the brain, and so effects of regional specialization of function, cytoarchitecture, gene expression were averaged, and the remaining hub-related signal could be isolated.


% LINEAGE
[[TODO: add details on lineage]] While the relationship between cell lineage and gene expression was studied in other species \cite{Cui2007, Kluger2004}.

LIMITATIONS:
\begin{itemize}
    \item Did the best we had with sparsely-annotated binary gene expression data
    \item No way of discriminating between missing data and the absence of expression
    \item only around 5 percent of genes in the genome available
    \item annotation problems: different qualifiers, loosing sensitivity/specificity if including too much or too little - need to balance
    \item Future work may have actual high-throughput RNAseq gene data for all neurons, could do this analysis properly!
    \item Would surely allow more subtle understanding of how gene expression varies with connectivity roles
\end{itemize}

% Binary expression data
Compared to continuous expression information provided from techniques like RNA sequencing, \emph{in situ} hybridization, or microarray with whole-brain coverage using a systematic experimental procedure \cite{Lein:2007jn, Shen:2012ua, Tasic:2016jp}, the data analyzed here presents numerous challenges.
With access to only a relatively limited coverage of the genome [[TODO: how many worm genes in total? Add proportion of worm genes represented]], the enrichment results provided here are severely constrained in scope.
Further, using the current data, we are unable to distinguish expression levels between neurons that express a given gene, and we are unable to distinguish missing data, from negative data (both are represented as `0' in our data).
Future work systematically mapping gene expression in a unified experimental paradigm would allow a more subtle and comprehensive analysis of gene expression and function.


\section*{Conclusion}

Conclusion text.

\section*{Acknowledgments}
Thanks to Alex Fornito, for being a big deal.
We thank WormBase.

\nolinenumbers

% Either type in your references using
% \begin{thebibliography}{}
% \bibitem{}
% Text
% \end{thebibliography}
%
% or
%
% Compile your BiBTeX database using our plos2015.bst
% style file and paste the contents of your .bbl file
% here. See http://journals.plos.org/plosone/s/latex for
% step-by-step instructions.
%


\bibliography{library_ben,library}
% \begin{thebibliography}{10}
%\paragraph*{S1 File.}
%\label{S1_File}
%{\bf Lorem ipsum.}  Maecenas convallis mauris sit amet sem ultrices gravida. Etiam eget sapien nibh. Sed ac ipsum eget enim egestas ullamcorper nec euismod ligula. Curabitur fringilla pulvinar lectus consectetur pellentesque.

%\paragraph*{S1 Video.}
%\label{S1_Video}
%{\bf Lorem ipsum.}  Maecenas convallis mauris sit amet sem ultrices gravida. Etiam eget sapien nibh. Sed ac ipsum eget enim egestas ullamcorper nec euismod ligula. Curabitur fringilla pulvinar lectus consectetur pellentesque.

%\paragraph*{S1 Appendix.}
%\label{S1_Appendix}
%{\bf Lorem ipsum.} Maecenas convallis mauris sit amet sem ultrices gravida. Etiam eget sapien nibh. Sed ac ipsum eget enim egestas ullamcorper nec euismod ligula. Curabitur fringilla pulvinar lectus consectetur pellentesque.




% \end{thebibliography}

\newpage
\section*{Supporting information}
\setcounter{figure}{0} \renewcommand{\thefigure}{S\arabic{figure}}
\renewcommand{\thefigure}{S\arabic{figure}}

\subsection*{Expression similarity measures}

In addition to the Pearson correlation, $r_\phi$, used throughout this work, we also compared a range of alternative similarity measures for binary strings, including the
Jaccard index, $n_{11}/(n_{10}+n_{01}+n_{11})$,
Yule's Q coefficient, $(n_{00}n_{11} - n_{01}n_{10})/(n_{00}n_{11} + n_{01}n_{10})$,
and the $\chi^2$ index, $N (n_{00}n_{11} - n_{01}n_{10})/(n_{1\bullet}n_{0\bullet}n_{\bullet 0}n_{\bullet 1})$ \cite{Kaufman2006}.
% and Simple matching coefficients [[TODO: define and give reference]],

All of these standard measures are symmetric in 1/0, however, in the gene expression data used here, a minority of genes are positively expressed in each neuron (a maximum of $15\%$ of genes expressed in any single neuron); we also wanted to distinguish biologically relevant cases where genes are expressed together, $n_{11}$, from the dominant case where a gene is not expressed, $n_{00}$.
To this end, we developed a new index based around that probability, $P(m)$, that two binary strings will contain $m$ `positive matches' (i.e., matching 1 with 1),
\begin{equation}
    P(m) = \binom{n_2}{m} \binom{N-n_2}{n_1-m} / \binom{N}{n_1},
\end{equation}
for two binary expression vectors, $x_i$, $y_i$, of length $N$, containing $n_1$ and $n_2$ 1s, respectively ($n_2 \leq n_1$), with $m$ matches ($n_{11}$).
Our index, $r_\xi$, computes the probability of at least this many matches as $r_\xi = 1 - \sum_{x=0}^{m-1} P(x)$.
This showed ([[biases?]]) which gave similar results [[TODO: check that behavior is indeed similar]].

Given the sparsity of expression annotation in the data analyzed here, we performed numerical tests to ensure that the coexpression measure used here was not highly sensitive to the proportion of genes expressed in each neuron (i.e., that there was no systematic bias in coexpression with the number of expressed genes in each neuron).
To evaluate each measure, generated random binary vectors of length 948 containing different proportions of ones seen in our data, ranging from the minimum, 1, to a maximum, 150, [[Is this actually the max in our data?]].
For all pairwise combinations of proportions, we computed the coexpression measure, taking an average across 1\,000 permutations, and then recording the resulting mean correlation value, plotted in Fig.~\ref{fig:S_propOnes}.
Because all vectors are independent random binary strings, the mean coexpression value should be zero in the absence of bias.
We see that the Pearson correlation (Fig.~\ref{fig:S_propOnes}A) shows no systematic bias to the proportion of ones in the vector (varying within $\approx 10^{-3}$), whereas Yule's Q coefficient shows a negative bias for small annotation proportions (Fig.~\ref{fig:S_propOnes}B), while the Jaccard index shows a strong positive bias across the full range (Fig.~\ref{fig:S_propOnes}C).
Based on these numerical experiments, we selected Pearson's mean square contingency coefficient, $r_\phi$, here, to ensure that changes in coexpression were due to matching structure and not simply the number of gene annotations.
% In contrast, other measures such as Jaccard and Yule's index are highly dependent on this ratio (see S1 Fig.).% done with compareProportions script
% and Simple matching coefficients
% For example, Yule's Q, Yule's Y coefficients on average tend to reach extreme values when the ratio between the number of ones and zeros in a vector is large (e.g., 1:20).
% Simple matching coefficient, on the other hand, shows a steady increase with increasing ratio.
% We therefore selected $r_\phi$ to analyze in this work, which was insensitive to the proportion of genes expressed in each neuron.

%% enrichment table for connected links
% results generated using ermineJ, all categories tested separately. significante threshold pcorrected<0.05. Results arranged using 
\paragraph*{S1 Table}

{\bf Enrichment results for connected \textit{vs} unconnected neurons.} 
\begin{table}[]
\centering
\caption{
[[TODO-AA: Give a brief description of what was done to get this enriched list.
Give brief pointer to method, explain each column (e.g., is num of genes the total, or just how many were in our analysis), and say that analysis was done in ermineJ, with FDR-corrected p-values, etc.]]
[[TODO-AA: Sort each group by increasing $p$-values]]
Biological process, molecular function and cellular component GO categories of genes showing significantly increased coexpression in connected pairs of neurons over unconnected pairs.
}
\label{tab:connected}
\begin{tabular}{llcc}
\hline
\multicolumn{1}{|l|}{\textbf{GO category}} & \multicolumn{1}{l|}{\textbf{Description}}          & \multicolumn{1}{c|}{\textbf{\begin{tabular}[c]{@{}c@{}}No. \\ of genes\end{tabular}}} & \multicolumn{1}{c|}{\textbf{$P$-value (corrected)}} \\ \hline
\multicolumn{4}{l}{\textit{Biological processes}}                                                                                                                                                                                  \\
GO:0035235                                 & ionotropic glutamate receptor signaling pathway    & 7                                                                                        & $6.82^{-10}$                              \\
GO:00072                                 & glutamate receptor signaling pathway               & 9                                                                                        & 0.0034                               \\
\multicolumn{4}{l}{\textit{Molecular function}}                                                                                                                                                                                    \\
GO:0005                                 & ionotropic glutamate receptor activity             & 7                                                                                        & $1.73^{-10}$                              \\
GO:0008                                & glutamate receptor activity                        & 9                                                                                        & 0.0017                              \\
GO:00052                                & extracellular-glutamate-gated ion channel activity & 9                                                                                        & 0.0132                           \\
\multicolumn{4}{l}{\textit{Cellular components}}                                                                                                                                                                                    \\
GO:00300                                & cell junction                                      & 55                                                                                       & 0.0235                              \\
GO:00987                                 & postsynapse                                        & 19                                                                                       & 0.0161                            
\end{tabular}
\end{table}

% rich feeder vs peripheral
\paragraph*{S2 Table.}
{\bf Enrichment results for rich and feeder \textit{vs} peripheral links.}
\begin{table}[]
\centering
\caption{Biological process, molecular function and cellular component GO categories of genes showing significantly increased coexpression in rich and feeder over peripheral links
[[TODO-AA: Sort each group by increasing $p$-values]]
[[TODO-AA: Give a brief description of what was done to get this enriched list.
Give brief pointer to method, explain each column (e.g., is num of genes the total, or just how many were in our analysis), and say that analysis was done in ermineJ, with FDR-corrected p-values, etc. -- can refer the reader to the above table (e.g., Method and table are as described in Table~X except ...)]]}
\label{tab:hub_enrichment}
\begin{tabular}{llcc}
\hline
\multicolumn{1}{|l|}{\textbf{GO category}} & \multicolumn{1}{l|}{\textbf{Description}}                  & \multicolumn{1}{l|}{\textbf{\begin{tabular}[c]{@{}l@{}}No.\\  of genes\end{tabular}}} & \multicolumn{1}{l|}{\textbf{P-value}} \\ \hline
\multicolumn{4}{l}{\textit{Biological processes}}                                                                                                                                                                                          \\
GO:0035235                                 & ionotropic glutamate receptor signaling pathway            & 7                                                                                        & 0.0174                                \\
GO:0007215                                 & glutamate receptor signaling pathway                       & 9                                                                                        & 0.0406                                \\
\multicolumn{4}{l}{\textit{Molecular function}}                                                                                                                                                                                            \\
GO:0022824                                 & transmitter-gated ion channel activity                     & 13                                                                                       & 0.0014                                \\
GO:0022835                                 & transmitter-gated channel activity                         & 13                                                                                       & 0.0014                                \\
GO:0004970                                 & ionotropic glutamate receptor activity                     & 7                                                                                        & 0.0021                                \\
GO:0005230                                 & extracellular ligand-gated ion channel activity            & 19                                                                                       & 0.0019                                \\
GO:0008066                                 & glutamate receptor activity                                & 9                                                                                        & 0.0057                                \\
GO:0005231                                 & excitatory extracellular ligand-gated ion channel activity & 14                                                                                       & 0.0054                                \\
GO:0099094                                 & ligand-gated cation channel activity                       & 11                                                                                       & 0.0163                                \\
GO:0030594                                 & neurotransmitter receptor activity                         & 23                                                                                       & 0.0176                                \\
GO:0015075                                 & ion transmembrane transporter activity                     & 53                                                                                       & 0.0163                                \\
GO:0005234                                 & extracellular-glutamate-gated ion channel activity         & 8                                                                                        & 0.0151                                \\
GO:0038023                                 & signaling receptor activity                                & 63                                                                                       & 0.0140                                \\
GO:0022891                                 & substrate-specific transmembrane transporter activity      & 55                                                                                       & 0.0226                                \\
GO:0022892                                 & substrate-specific transporter activity                    & 55                                                                                       & 0.0207                                \\
GO:0008324                                 & cation transmembrane transporter activity                  & 41                                                                                       & 0.0249                                \\
GO:0004888                                 & transmembrane signaling receptor activity                  & 60                                                                                       & 0.0292                                \\
GO:0015276                                 & ligand-gated ion channel activity                          & 27                                                                                       & 0.0327                                \\
GO:0022834                                 & ligand-gated channel activity                              & 27                                                                                       & 0.0306                                \\
GO:0022857                                 & transmembrane transporter activity                         & 65                                                                                       & 0.0314                                \\
GO:0005215                                 & transporter activity                                       & 66                                                                                       & 0.0407                                \\
GO:0004872                                 & receptor activity                                          & 74                                                                                       & 0.0433                                \\
GO:0060089                                 & molecular transducer activity                              & 74                                                                                       & 0.0433                                \\
GO:0005216                                 & ion channel activity                                       & 48                                                                                       & 0.0466                                \\
GO:0022838                                 & substrate-specific channel activity                        & 50                                                                                       & 0.0473                                \\
GO:0022890                                 & inorganic cation transmembrane transporter activity        & 39                                                                                       & 0.0485                                \\
\multicolumn{4}{l}{\textit{Cellular components}}                                                                                                                                                                                           \\
GO:0098794                                 & postsynapse                                                & 14                                                                                       & 1.04e-10                              \\
GO:0045202                                 & synapse                                                    & 43                                                                                       & 0.0005                                \\
GO:0044456                                 & synapse part                                               & 31                                                                                       & 0.0007                                \\
GO:0045211                                 & postsynaptic membrane                                      & 11                                                                                       & 0.0034                                \\
GO:0030054                                 & cell junction                                              & 39                                                                                       & 0.0108                                \\
GO:0097060                                 & synaptic membrane                                          & 15                                                                                       & 0.0166                                \\
GO:0098802                                 & plasma membrane receptor complex                           & 8                                                                                        & 0.0250                                \\
GO:0043235                                 & receptor complex                                           & 9                                                                                        & 0.0392                               
\end{tabular}
\end{table}
\clearpage

% ------------------------------------------------------------------------------
% Include only the SI item label in the paragraph heading. Use the \nameref{label} command to cite SI items in the text.
% ------------------------------------------------------------------------------

% ------------------------------------------------------------------------------
% compare proportions for simulations to validate
% ------------------------------------------------------------------------------
\paragraph*{S1 Fig.}
{\bf Comparing different coexpression measures.}
\begin{figure}[h]
\centering
    \includegraphics[width=1\textwidth]{CoexpressionMeasures.pdf}
\caption{
\label{fig:S_propOnes}
\textbf{Dependence of binary coexpression metrics to the proportion of positive annotations}
Given the sparse binary annotation of genes to neurons, we investigated bias in each measure across 1\,000 pairs of random binary vectors, each with a fixed proportion of ones.
Here we plot the mean correlation between the two vectors, averaged over 1\,000 runs, for 948-long vectors with between 0--0.16 proportion of ones [[TODO: how many voxels?matrix is 100x100]].
Bias is indicated by a systematic trend in correlation values, not due to actual matching, but driven simply by the proportion of positive annotations for a pair of vectors.
\textbf{A} Pearson, $r_\phi$, the measure used throughout this work, shows no systematic bias (note the color axis scale, 10$^{-3}$)
\textbf{B} [[Yule's Q? YES]], and \textbf{C} Jaccard correlation measures both exhibit a strong systematic bias.
[[TODO-AA: Add A, B, C labels, add our new positive match measure, provide better titles, use appropriate color bars/scales]]
}
\end{figure}


% \paragraph*{S2 Fig.}
% {\bf Coexpression as a function of distance}

% ------------------------------------------------------------------------------
% <<probabilityVSdistance>>
% ------------------------------------------------------------------------------
\begin{figure}[h]
\centering
    \includegraphics[width=1\textwidth]{ConnectionProbability.pdf}
\caption{
\label{fig:S_connProbability}
\textbf{Connection probability decreases with the separation distance between neurons.}
\textbf{A}
We show the probability of a connection between a pair of neurons as a function of their separation distance, estimated in 10 equiprobable distance bins, demonstrating a decrease for connections within the head (`HeadHead', aqua), connections from head $\rightarrow$ tail (`HeadTail', brown) and from tail $\rightarrow$ head (`TailHead', stone blue), connections within the tail (`TailTail', red), and connections within the body (`BodyBody', dark purple).
Connection probability in each distance bins is shown as a circle (bin centers) and a horizontal line (bin extent).
Exponential fits are shown dotted.
\textbf{B}
Plots as in A, but for connections showing minimal distance dependence: from body $\rightarrow$ head (`BodyHead', forest green), from body $\rightarrow$ tail (`BodyTail', dirt green), connections from head $\rightarrow$ body (`HeadBody', purple), and from tail $\rightarrow$ body (`TailBody', dark brown).
% A decreasing relationship is evident up to a separation distance of approximately 0.2\,mm (driven by tail-tail and head-head connections), with a longer-range decrease up to $\approx$0.5\,mm due to body-body connections.
% The increase at large distances ($\approx 1$\,mm) is due to head-tail connections, which also show a decay with distance, from $\approx 1$\,mm through to $\approx$1.2\,mm.
% connections between body and head and body and tail showed no clear distance dependence.}
\textbf{C}
Distribution of hub-hub connections (`rich', red), and all other connections (`non-rich', i.e., feeder and peripheral, gray) across anatomical divisions (described above), defining hubs as those with degree, $k > 41$.
The increased separation distance between hubs is driven by a relative increase in connection density between the head and tail.
}
\end{figure}
% ------------------------------------------------------------------------------

% ------------------------------------------------------------------------------
% <<plot_coexpDistance.m>>
% [[coexpressionDistance(C,G),CoExpressionDistanceDetail(C,G)]]
% ------------------------------------------------------------------------------
\begin{figure}[h]
\centering
    \includegraphics[width=1\textwidth]{DistanceCoexpression.pdf}
\caption{
\label{fig:S_distCoexp}
\textbf{Gene coexpression decreases with separation distance within the head and tail.}
Gene coexpression, $r_\phi$ (in all cases excluding left-right homologous pairs of neurons), is shown as a function of the pairwise separation distance between pairs of neurons (shown as the mean (solid) $\pm$ standard deviation (dotted) in seven equiprobable distance bins, with extent shown as horizontal bars), for \textbf{A} all pairs of neurons in the head, \textbf{B} all pairs of neurons in the tail, and \textbf{C} all other pairs (labeled).
Scatters for all neuron pairs are added in \textbf{A} and \textbf{B}.
An exponential relationship, $f(x) = A\exp(-Bx)+C$, is fitted in \textbf{A} and \textbf{B}, revealing a slight decreasing trend in $r_\phi$ with distance, as per macroscopic mammalian brains \cite{Fulcher:2016ck, Krienen:2016eq}.
Looking within the head, we find different spatial relationships as a function of neuron type, shown for \textbf{C} pairs of interneurons, \textbf{D} pairs of motor neurons, \textbf{E} motor-interneuron pairs, \textbf{F} pairs of sensory neurons, \textbf{G} interneuron-sensory neuron pairs, and \textbf{H} motor-sensory pairs.
[[TODO: add labels for subplots C--H, label all axes?]]
}
\end{figure}
% ------------------------------------------------------------------------------


% ------------------------------------------------------------------------------
% <<FeedInOutNodes.m>>
% ------------------------------------------------------------------------------
\begin{figure}[h]
\centering
    \includegraphics[width=1\textwidth]{FeedInOutNodes.pdf}
\caption{
\textbf{Neuron properties vary in terms of their connectivity relationship to hub nodes.}
Using a degree threshold for hubs, $k > 41$, we class all individual neurons into five mutually exclusive categories, shown schematically in the figures (for hubs, red, and nonhubs, light blue), as:
(i) `hub' ($k>41$),
and then classify nonhubs ($k\leq41$) as:
(ii) `feedIn' (projects to a hub, but does not receive a projection from a hub),
(iii) `feedOut' (receives a projection from a hub, but does project to a hub),
(iv) `feedBoth' (receives a projection from, and projects to a hub),
(v) `peripheral' (neither receives a projection from nor projects to a hub).
We show the number of neurons in each of these fives classes that:
\textbf{A} is an interneuron (orange), motor neuron (green), sensory neuron (blue), or multimodal (yellow),
\textbf{B} is in the head (gray), body (purple), or tail (red),
\textbf{C} uses one of nine neurotransmitter systems (labeled).
\label{fig:S_feedInOutNodes}
[[TODO: make this look decent -- could stretch and then put all bars into a single plot, rather than A--C. Come up with new names for nodes that don't confuse with edge names. Labels not visible for B]]
}
\end{figure}
% ------------------------------------------------------------------------------


% ------------------------------------------------------------------------------
\begin{figure}[h]
\centering
    \includegraphics[width=1\textwidth]{SpatialPlot.png}
\caption{
\textbf{Spatial distribution of rich links, and neuron types across the C. elegans nervous system}.
Neurons are colored by type: interneurons (orange), sensory neurons (blue), motor neurons (green), and multimodal (yellow).
Hub-hub links are shown, and zoom-in plots of head and tail are shown as dotted rectangles in the lower plot.
All pairs of horizontal and vertical axes are in scale with each other.
\label{fig:S_neuronsSpace}
}
\end{figure}
% ------------------------------------------------------------------------------

% ------------------------------------------------------------------------------
% <<coexpRecUnidirUncon.m>>
% ------------------------------------------------------------------------------
\paragraph*{S3 Fig.}
{\bf Coexpression for rich, feeder and peripheral links as a function of degree.}
\begin{figure}[!h]
\label{S3_Fig}
\centering
    \includegraphics[width=1\textwidth]{Distributions.pdf}
    \caption{
\textbf{Left}: Distribution of transcriptional similarity between reciprocally connected (219 pairs), unidirectionally connected (1721) and  and unconnected (36\,749 pairs) pairs of neurons as a violin plot, with the median of each distribution shown as a horizontal line.
Connections between symmetrical (left/right) pairs of neurons are excluded from the analysis. Coexpression values for 
Coexpression, $r_\phi$ is increased in connected (reciprocally and unidirectionally) pairs of neurons ($P < 10^{-50}$, Wilcoxon rank sum test)
\textbf{Right}: Distribution of transcriptional similarity for rich, feed-in, feed-out and peripheral links, where hubs are neurons with degree $k>41$.
[[TODO-AA:color inside of the distributions, change y axis to Gene coexpression, $r_\phi$, change $k>42$ to $k>41$.
Label subplots as A and B.
Add $p$ values on the plot (sure, where significant)
]]
}
\end{figure}

% <<networkRC.m>>
\paragraph*{S5 Fig.}
{\bf Weighted and mixed rich club}
\begin{figure}[!h]
\label{S5_Fig}
\centering
    \includegraphics[width=1\textwidth]{WeightedMixedRC.pdf}
    \caption{\textbf{A} Degree distribution of the binary chemical connectome, where neurons are labeled according to four categories:
(i) interneuron (85 neurons, orange),
(ii) sensory (68 neurons, blue),
(iii) motor (108 neurons, green), or
(iv) multimodal (18 neurons, yellow).
An extended tail of high-degree neurons can be seen, which are mostly interneurons.
\textbf{B}
Normalized weighted (topology fixed and weights randomized in the null model - orange) $\Phi_\mathrm{normWeighted}$ and mixed (both topology and weights mixed in the null model - green) $\Phi_\mathrm{normMixed}$ rich club coefficient as a function of the degree, $k$, at which hubs are defined (as neurons with degree $>k$).
Circles indicate values of $\Phi_\mathrm{norm}$ that are significantly higher than an ensemble of 1\,000 degree-matched null networks ($P < 0.05$).
(Welch's $t$-test, $P < 0.05$).
}
\end{figure}

% <<RichClub.m>>
\paragraph*{S6 Fig.}
{\bf Mean birth time difference as a function of degree for different link types}
\begin{figure}[!h]
\label{S6_Fig}
\centering
    \includegraphics[width=1\textwidth]{MeanBirthTimes.pdf}
    \caption{\emph{Top}: Degree distribution.
\emph{Middle}: proportion of connections that are `rich' (hub$\rightarrow$hub, red), `feed-in' (nonhub$\rightarrow$hub, yellow), `feed-out' (hub$\rightarrow$nonhub, orange), and `peripheral' (nonhub$\rightarrow$nonhub, blue) as a function of the degree threshold, $k$, used to define hubs.
Note that at high $k$, most neurons are labeled as nonhubs, and hence the vast majority of connections are `peripheral'.
\emph{Lower}: Mean birth time difference for each connection type as a function of $k$.
The mean birth time difference across all network links shown as a dotted black line; the topological rich-club regime (determined from the network topology, cf. Fig.~\ref{fig:topology_rich}) is shaded gray.
Circles indicate a statistically significant increase or decrease in birth time difference in a given link type relative to the rest of the network (two-sided Welch’s t test; $P < 0.05$).
}
\end{figure}

% propTypesDegree(C) script
%\paragraph*{S6 Fig.}
%{\bf Neuron type and neurotransmitter as a function of degree}
%\begin{figure}[!h]
%\label{S7_Fig}
%\centering
%    \includegraphics[width=0.7\textwidth]{TypeTransmitterDegree}
%\end{figure}

% RichClub script
\paragraph*{S7 Fig.}
{\bf Average coexpression for rich, feeder and peripheral links as a function of degree.}
\begin{figure}[!h]
\label{S7_Fig}
\centering
    \includegraphics[width=1\textwidth]{coXmeanCoexpression.pdf}
\caption{\emph{Top}: Degree distribution.
\emph{Middle}: proportion of connections that are `rich' (hub$\rightarrow$hub, red), `feed-in' (nonhub$\rightarrow$hub, yellow), `feed-out' (hub$\rightarrow$nonhub, orange), and `peripheral' (nonhub$\rightarrow$nonhub, blue) as a function of the degree threshold, $k$, used to define hubs.
Note that at high $k$, most neurons are labeled as nonhubs, and hence the vast majority of connections are `peripheral'.
\emph{Lower}: Mean gene coexpression calculated using similarity index from only positive matches, $r_\xi$, for each connection type as a function of $k$.
The mean coexpression across all network links shown as a dotted black line; the topological rich-club regime (determined from the network topology, cf. Fig.~\ref{fig:topology_rich}) is shaded gray.
Circles indicate a statistically significant increase in gene coexpression in a given link type relative to the rest of the network (one-sided Welch’s t test; $P < 0.05$).}
\end{figure}




\end{document}
