% Template for PLoS
% Version 3.4 January 2017
%
% % % % % % % % % % % % % % % % % % % % % %
%
% -- IMPORTANT NOTE
%
% This template contains coments intended
% to minimize problems and delays during our production
% process. Please follow the template instructions
% whenever possible.
%
% % % % % % % % % % % % % % % % % % % % % % %
%
% Once your paper is accepted for publication,
% PLEASE REMOVE ALL TRACKED CHANGES in this file
% and leave only the final text of your manuscript.
% PLOS recommends the use of latexdiff to track changes during review, as this will help to maintain a clean tex file.
% Visit https://www.ctan.org/pkg/latexdiff?lang=en for info or contact us at latex@plos.org.
%
%
% There are no restrictions on package use within the LaTeX files except that
% no packages listed in the template may be deleted.
%
% Please do not include colors or graphics in the text.
%
% The manuscript LaTeX source should be contained within a single file (do not use \input, \externaldocument, or similar commands).
%
% % % % % % % % % % % % % % % % % % % % % % %
%
% -- FIGURES AND TABLES
%
% Please include tables/figure captions directly after the paragraph where they are first cited in the text.
%
% DO NOT INCLUDE GRAPHICS IN YOUR MANUSCRIPT
% - Figures should be uploaded separately from your manuscript file.
% - Figures generated using LaTeX should be extracted and removed from the PDF before submission.
% - Figures containing multiple panels/subfigures must be combined into one image file before submission.
% For figure citations, please use "Fig" instead of "Figure".
% See http://journals.plos.org/plosone/s/figures for PLOS figure guidelines.
%
% Tables should be cell-based and may not contain:
% - spacing/line breaks within cells to alter layout or alignment
% - do not nest tabular environments (no tabular environments within tabular environments)
% - no graphics or colored text (cell background color/shading OK)
% See http://journals.plos.org/plosone/s/tables for table guidelines.
%
% For tables that exceed the width of the text column, use the adjustwidth environment as illustrated in the example table in text below.
%
% % % % % % % % % % % % % % % % % % % % % % % %
%
% -- EQUATIONS, MATH SYMBOLS, SUBSCRIPTS, AND SUPERSCRIPTS
%
% IMPORTANT
% Below are a few tips to help format your equations and other special characters according to our specifications. For more tips to help reduce the possibility of formatting errors during conversion, please see our LaTeX guidelines at http://journals.plos.org/plosone/s/latex
%
% For inline equations, please be sure to include all portions of an equation in the math environment.  For example, x$^2$ is incorrect; this should be formatted as $x^2$ (or $\mathrm{x}^2$ if the romanized font is desired).
%
% Do not include text that is not math in the math environment. For example, CO2 should be written as CO\textsubscript{2} instead of CO$_2$.
%
% Please add line breaks to long display equations when possible in order to fit size of the column.
%
% For inline equations, please do not include punctuation (commas, etc) within the math environment unless this is part of the equation.
%
% When adding superscript or subscripts outside of brackets/braces, please group using {}.  For example, change "[U(D,E,\gamma)]^2" to "{[U(D,E,\gamma)]}^2".
%
% Do not use \cal for caligraphic font.  Instead, use \mathcal{}
%
% % % % % % % % % % % % % % % % % % % % % % % %
%
% Please contact latex@plos.org with any questions.
%
% % % % % % % % % % % % % % % % % % % % % % % %

\documentclass[10pt,letterpaper]{article}
\usepackage[top=0.85in,left=2.75in,footskip=0.75in]{geometry}

% amsmath and amssymb packages, useful for mathematical formulas and symbols
\usepackage{amsmath,amssymb}

% Use adjustwidth environment to exceed column width (see example table in text)
\usepackage{changepage}

% Use Unicode characters when possible
\usepackage[utf8x]{inputenc}

% textcomp package and marvosym package for additional characters
\usepackage{textcomp,marvosym}

% cite package, to clean up citations in the main text. Do not remove.
\usepackage{cite}

% % Added natbib for citet commands:
% \usepackage[numbers]{natbib}

% Added url for urls:
\usepackage{url}

% Use nameref to cite supporting information files (see Supporting Information section for more info)
\usepackage{nameref,hyperref}

% line numbers
\usepackage[right]{lineno}

% ligatures disabled
\usepackage{microtype}
\DisableLigatures[f]{encoding = *, family = * }

% color can be used to apply background shading to table cells only
\usepackage[table]{xcolor}

% array package and thick rules for tables
\usepackage{array}

% create "+" rule type for thick vertical lines
\newcolumntype{+}{!{\vrule width 2pt}}

% create \thickcline for thick horizontal lines of variable length
\newlength\savedwidth
\newcommand\thickcline[1]{%
  \noalign{\global\savedwidth\arrayrulewidth\global\arrayrulewidth 2pt}%
  \cline{#1}%
  \noalign{\vskip\arrayrulewidth}%
  \noalign{\global\arrayrulewidth\savedwidth}%
}

% \thickhline command for thick horizontal lines that span the table
\newcommand\thickhline{\noalign{\global\savedwidth\arrayrulewidth\global\arrayrulewidth 2pt}%
\hline
\noalign{\global\arrayrulewidth\savedwidth}}


% Remove comment for double spacing
\usepackage{setspace}
\doublespacing

% Text layout
\raggedright
\setlength{\parindent}{0.5cm}
\textwidth 5.25in
\textheight 8.75in

% Bold the 'Figure #' in the caption and separate it from the title/caption with a period
% Captions will be left justified
\usepackage[aboveskip=1pt,labelfont=bf,labelsep=period,justification=raggedright,singlelinecheck=off]{caption}
\renewcommand{\figurename}{Fig}

% Use the PLoS provided BiBTeX style
\bibliographystyle{plos2015}
% \bibliographystyle{plainnat}

% Remove brackets from numbering in List of References
\makeatletter
\renewcommand{\@biblabel}[1]{\quad#1.}
\makeatother

% Leave date blank
\date{}

% Header and Footer with logo
\usepackage{lastpage,fancyhdr,graphicx}
\usepackage{epstopdf}
\usepackage{hyperref}% http://ctan.org/pkg/hyperref
\pagestyle{myheadings}
\pagestyle{fancy}
\fancyhf{}
\setlength{\headheight}{27.023pt}
\lhead{\includegraphics[width=2.0in]{PLOS-submission.eps}}
\rfoot{\thepage/\pageref{LastPage}}
\renewcommand{\footrule}{\hrule height 2pt \vspace{2mm}}
\fancyheadoffset[L]{2.25in}
\fancyfootoffset[L]{2.25in}
\lfoot{\sf PLOS}


% Make it so figures are more likely to share a page with text:
\renewcommand{\topfraction}{0.85} % 85% rather than default 70% of the top of a text page can contain figures.
\renewcommand{\textfraction}{0.1} % as little as 10% of a page can be text when shared with a figure.
\renewcommand{\floatpagefraction}{0.80} % only makes a figure-only page if figure takes up more than 75% of a page

%% Include all macros below

% \newcommand{\lorem}{{\bf LOREM}}
% \newcommand{\ipsum}{{\bf IPSUM}}

%% END MACROS SECTION
%\usepackage{setspace}
%\doublespacing
\graphicspath{{./figs/}}
%\linespread{1.6}
\begin{document}
\vspace*{0.2in}

% Title must be 250 characters or less.
\begin{flushleft}
{\Large
\textbf\newline{Hub connectivity, neuronal diversity, and gene expression in the \emph{Caenorhabditis elegans} connectome} % Please use "sentence case" for title and headings (capitalize only the first word in a title (or heading), the first word in a subtitle (or subheading), and any proper nouns).
}
\newline
% Insert author names, affiliations and corresponding author email (do not include titles, positions, or degrees).
\\
Aurina Arnatkevi\u{c}i\={u}t\.{e}\textsuperscript{1\Yinyang}*,
Ben D. Fulcher\textsuperscript{1\Yinyang},
Roger Pocock\textsuperscript{2},
Alex Fornito\textsuperscript{1}\\
\bigskip
\textbf{1} Brain and Mental Health Laboratory, Monash Institute of Cognitive and Clinical Neurosciences, School of Psychological Sciences, Monash University, 770 Blackburn Rd, Clayton, 3168, VIC, Australia.\\
\textbf{2} Development and Stem Cells Program, Monash Biomedicine Discovery Institute and Department of Anatomy and Developmental Biology, Monash University, 15 Innovation Walk, Clayton 3800, VIC, Australia.
\\
\bigskip
%
% % Insert additional author notes using the symbols described below. Insert symbol callouts after author names as necessary.
% %
% % Remove or comment out the author notes below if they aren't used.
% %
% % Primary Equal Contribution Note
\Yinyang These authors contributed equally to this work.\\
%
% % Additional Equal Contribution Note
% % Also use this double-dagger symbol for special authorship notes, such as senior authorship.
% % \ddag These authors also contributed equally to this work.
%
% % Current address notes
% % \textcurrency Current Address: Dept/Program/Center, Institution Name, City, State, Country % change symbol to "\textcurrency a" if more than one current address note
% % \textcurrency b Insert second current address
% % \textcurrency c Insert third current address
%
% % Deceased author note
% % \dag Deceased
%
% % Group/Consortium Author Note
% % \textpilcrow Membership list can be found in the Acknowledgments section.
%
% % Use the asterisk to denote corresponding authorship and provide email address in note below.
* aurina.arnatkeviciute@monash.edu

\end{flushleft}
% Please keep the abstract below 300 words
\section*{Abstract}
Studies of nervous system connectivity, in a wide variety of species and at different scales of resolution, have identified several highly conserved motifs of network organization.
One such motif is a heterogeneous distribution of connectivity across neural elements, such that some elements act as highly connected and functionally important network hubs.
These brain network hubs are also densely interconnected, forming a so-called rich club.
Recent work in mouse has identified a distinctive transcriptional signature of neural hubs, characterized by tightly coupled expression of oxidative metabolism genes, with similar genes characterizing macroscale inter-modular hub regions of the human cortex.
Here, we sought to determine whether hubs of the neuronal \textit{C. elegans} connectome also show tightly coupled gene expression.
Using open data on the chemical and electrical connectivity of 279 \textit{C. elegans} neurons, and binary gene expression data for each neuron across 948 genes, we computed a correlated gene expression score for each pair of neurons, providing a measure of their gene expression similarity.
We demonstrate that connections between hub neurons are the most similar in their gene expression while connections between nonhubs are the least similar.
Genes with the greatest contribution to this effect are involved in glutamatergic and cholinergic signaling, and other communication processes.
We further show that coupled expression between hub neurons cannot be explained by their neuronal subtype (i.e., sensory, motor, or interneuron), separation distance, chemically secreted neurotransmitter, birth time, pairwise lineage distance, or their topological module affiliation.
Instead, this coupling is intrinsically linked to the identity of most hubs as command interneurons, a specific class of interneurons that regulates locomotion.
Our results suggest that neural hubs may possess a distinctive transcriptional signature, preserved across scales and species, that is related to the involvement of hubs in regulating the higher-order behaviors of a given organism.

% Please keep the Author Summary between 150 and 200 words
% Use first person. PLOS ONE authors please skip this step.
% Author Summary not valid for PLOS ONE submissions.
\section*{Author summary}

Some elements of neural systems possess many more connections than others, marking them as network hubs.
These hubs are often densely interconnected with each other, forming a so-called rich-club that is thought to support integrated function.
Recent work in the mouse suggests that connected pairs of hubs show higher levels of transcriptional coupling than other pairs of brain regions.
Here, we show that hub neurons of the nematode \textit{C. elegans} also show tightly coupled gene expression and that this effect cannot be explained by the spatial proximity or anatomical location of hub neurons, their chemical composition, birth time, neuronal lineage or topological module affiliation.
Instead, we find that elevated coexpression is driven by the identity of most hubs of the \textit{C. elegans} connectome as command interneurons, a specific functional class of neurons that regulate locomotion.
These findings suggest that coupled gene expression is a highly conserved genomic signature of neural hubs that may be related to the specific functional role that hubs play in broader network function.

\linenumbers

\section*{Introduction}

Neuronal connectivity provides the substrate for integrated brain function.
Recent years have seen several systematic and large-scale attempts to generate comprehensive wiring diagrams, or connectomes, of nervous systems \cite{vandenHeuvel:2016eg} in species as diverse as the nematode \emph{Caenorhabditis elegans} \cite{White:1986tx, Varshney2011}, \emph{Drosophila} \cite{Chiang:2011, Shih2015}, zebrafish \cite{Wanner:2016ea, Hildebrand:2017iu}, mouse \cite{Oh2014, Zingg:2014el}, rat \cite{bota2015}, cat \cite{scannell1995}, macaque \cite{Markov:2012wu, stephan2001}, and human \cite{Hagmann:2008gda, VanEssen2013}.
One of the most striking findings to emerge from analyses of these diverse data, acquired using different measurement techniques and at resolution scales ranging from nm to mm, is of a strong conservation of certain topological properties of network organization (reviewed in \cite{Bullmore:2009iv, Bullmore:2012vl, Sporns2011, vandenHeuvel:2016eg, Schroter:2017eo}, see also \cite{fornito2016book}).
These properties include a modular organization, such that subsets of functionally related (and usually spatially adjacent) elements are densely interconnected with each other;
a hierarchy of modules, such that modules contain nested sub-modules and so on over multiple scales \cite{Meunier:2010hq, Bassett2010};
economical connectivity, such that wiring costs (typically measured in terms of wiring length) are near-minimal given the topological complexity of the system \cite{Betzel:2016jt, Bassett2010};
a heterogeneous distribution of connections across network nodes, such that most nodes possess relatively few connections and a small proportion of nodes have a very high degree of connectivity \cite{vandenHeuvel:2013ge, Varshney2011};
and stronger interconnectivity between hub nodes than expected by chance, leading to a so-called rich-club organization (i.e., the hubs are rich because they are highly connected and form a club because they are densely interconnected) \cite{vandenHeuvel:2011he, ZamoraLopez:2010hy, deReus:2013cy,Towlson2013, Shih2015}.

The rich-club organization of hub connectivity is thought to play a central role in integrating functionally diverse and anatomically disparate neuronal systems \cite{Fornito2015, vandenHeuvel:2013ij, ZamoraLopez:2010hy, Crossley2014, Crossley:2013kl}.
Consistent with this view, experimental data and computational modeling indicates that hub nodes, and the connections between them, are topologically positioned to mediate a high volume of signal traffic \cite{vandenHeuvel:2012kh, Harriger2012, Misic:2014it, Misic:2015jw}.
This integrative role comes at a cost, with connections between hubs extending over longer distances, on average, than other types of connections, a finding reported in the human \cite{vandenHeuvel:2012kh}, macaque \cite{Harriger2012}, rat \cite{VandenHeuvel2016b}, mouse \cite{Fulcher:2016ck}, and nematode \cite{Towlson2013}.
Human positron emission tomography also suggests that hub nodes consume greater metabolic resources and have higher levels of blood flow than other areas \cite{Tomasi:2013kl, Collin:2014kq, Liang2013a}.
This high metabolic cost may underlie the involvement of hub regions in a broad array of human diseases \cite{Fornito2015, Bullmore:2012vl, Crossley2014}.

Recent studies in mice and humans suggest that the high cost of hub connectivity is associated with a distinct
molecular signature, as inferred from brain-wide transcriptomic data. 
This work has focused on how patterns of expression across many thousands of genes covary between pairs of brain regions.
Such patterns of covariation are variously referred to as transcriptional coupling \cite{Fulcher:2016ck}, gene coexpression \cite{Krienen:2016eq} or correlated gene expression \cite{Richiardi2015, Mills2017, Goel2014}.
The goal of this work is to understand how pair-wise coupling of gene expression corresponds to the pairwise connection topology of the brain, thus drawing a link between molecular function and large-scale network organization.
For example, Fulcher and Fornito \cite{Fulcher:2016ck} combined viral tract tracing data on connectivity between 213 regions of the right hemisphere of the mouse brain \cite{Oh2014} with \emph{in situ} hybridization measures of the expression of 17\,642 genes in each of those regions.
They found that transcriptional coupling, across all genes, is stronger for connected compared to unconnected brain regions and that, in general, this coupling decays with increasing separation distance between brain regions.
Countering this general trend, however, connected pairs of hubs (i.e., the ``rich club'' of the brain) show the highest levels of transcriptional coupling (compared to hub-nonhub and nonhub-nonhub pairs), despite being separated by larger distances, on average, and being distributed across diverse anatomical brain divisions.
Moreover, this coupling is driven predominantly by genes regulating the oxidative synthesis and metabolism of adenosine triphosphate (ATP), the primary energetic currency of neuronal signaling \cite{Lennie:2003ia, Laughlin:2003vu}.
V\'ertes et al. \cite{Vertes2016a} later combined gene expression data of 20\,737 genes through 285 cortical areas of the human brain and found evidence that inter-modular hubs in resting state fMRI connectivity networks also have local transcriptional profiles enriched in oxidative metabolism and mitochondria.

Together, the analyses of Fulcher and Fornito \cite{Fulcher:2016ck} and V\'ertes et al. \cite{Vertes2016a}, which were performed using measures of mesoscale ($\mu$m to mm) and macroscale (mm to cm) connectivity, respectively, suggest that the molecular signature of hub connectivity, characterized by elevated coupling of genes regulating oxidative metabolism, may be conserved across species and resolution scales.
Here, we sought to test this hypothesis by examining the link between gene expression and microscale connectivity in the nematode \emph{C. elegans}.
\emph{C. elegans} is the only species to have its connectome mapped almost completely at the level of individual neurons and synapses using electron microscopy \cite{White:1986tx, Varshney2011}.
It comprises 302 neurons and around 5600 chemical and electrical synapses \cite{White:1986tx}.
We combined these data on neuronal connectivity with curated information on the binary expression patterns of 948 genes across neurons to examine the relationship between gene expression and the large-scale topological organization of the nematode nervous system.
We also used detailed information on neuron spatial positions, birth times, neuronal lineage as well as the functional and chemical composition of each neuron to understand the mechanisms through which gene expression might influence network topology.
Paralleling findings in humans and mouse, we find that hub neurons of \emph{C. elegans} are genomically distinct, with connected hub neurons showing the most similar patterns of gene expression.
Genes that contribute most to this effect are involved in regulating glutamate and acetylcholine function, and neuronal communication.
We demonstrate that this effect cannot be explained by factors such as neuronal birth time, lineage, neurotransmitter system or spatial position, but may instead be related to the functional specialization of hub neurons in mediating higher order behaviours of the organism.

% METHODS SECTION
\section*{Materials and methods}

We first describe the neural connectivity data used to construct a connectome for \textit{C. elegans} and the methods used to quantify network connectivity and other properties of individual neurons, including their neurochemical composition, birth times, and lineage relationships.
We then describe the gene expression data, how we measure expression similarity between pairs of neurons, and our method for scoring the contribution of individual genes to patterns of correlated gene expression.
Note that all data used for analysis in this work were obtained from publicly available sources, and can be downloaded from an accompanying figshare repository (\url{figshare.com/s/797199619fbabdab8c86}).
Code to process this data and reproduce all figures and analyses presented here is on github (\url{github.com/BMHLab/CElegansConnectomeGeneExpression}).

% METHODS: connectivity data
\subsection*{Neuronal connectivity data}
The nervous system of \emph{C. elegans} comprises 302 neurons, divided into the pharyngeal nervous system (20 neurons) and the somatic nervous system (282 neurons).
While research detailing the genetic underpinnings that guide the formation of \textit{C. elegans} nervous system is still ongoing, the spatial positions of neurons, and their interconnections, are known to be stereotypical across organisms \cite{Riddle1997}.
Neuronal connectivity data for the adult hermaphrodite \emph{C. elegans} was first mapped by White et al.~\cite{White:1986tx} through detailed electron microscopy, and then revised by Chen et al. \cite{Chen:2006ie} and Varshney et al.~\cite{Varshney2011}.
Here we analyze the larger somatic nervous system using data from Varshney et al.~\cite{Varshney2011}, who mapped connectivity between 279 neurons (282 somatic neurons, i.e., excluding CANL/R, and VC6, which do not form synapses with other neurons), which we obtained from WormAtlas \cite{WormAtlas} (\url{www.wormatlas.org/neuronalwiring.html#NeuronalconnectivityII}).

Connectivity data are available for both electrical gap junctions and chemical synapses.
The chemical synapse network is both directed (i.e., the pre-synaptic and post-synaptic neurons are identified) and weighted (as the number of synapses from one neuron to another), while gap junctions are conventionally represented as weighted (as the number of electrical synapses connecting two neurons), undirected connections.
Previous investigations of \emph{C. elegans} neuronal connectivity have used differently processed versions of these data, including:
(i) only chemical synapses \cite{Kashtan:2004ev};
(ii) a combination of chemical and electrical synapses as a directed network (electrical synapses represented as reciprocal connections) \cite{Azulay:2016cg, Kim:2016gl};
(iii) a combination of chemical and electrical synapses as an undirected network (representing unidirectional and reciprocal chemical connections equivalently) \cite{Towlson2013, Kim:2014bu, Pavlovic:2014gx, van2017guiding};
or (iv) comparing multiple connectome representations \cite{Pan:2010jt}.
Our analysis here focuses on the combined directed, binary network, treating gap junctions as bidirectional connections.
We chose to focus on a binarized network to follow previous studies on \textit{C. elegans} connectome data \cite{Kaufman2006, Towlson2013, Varier2011, Varadan2006, Pavlovic:2014gx} and to enable a more direct comparison to our previous analysis of the relationship between (binary) connectivity and gene expression in mouse \cite{Fulcher:2016ck}.
The resulting connectome contains 279 neurons, with 2\,990 unique connections linking 2\,287 pairs of neurons.

Note that the qualitative results presented here are not highly sensitive to the types of connections included in the connectome.
For example, neuron degree is highly correlated between networks generated using: (i) combined chemical and electrical synapses and (ii) chemical synapses only (Spearman's $\rho= 0.9$, $p = 3 \times 10^{-107}$).
We also obtained qualitatively similar results for rich-club organization and trends in CGE when excluding gap junctions from our analysis (see \nameref{S1_Fig}).

In addition, we assembled a range of data characterizing other properties of \emph{C. elegans} neurons.
To examine the effect of physical distance between pairs of neurons, we obtained two dimensional spatial coordinates for each neuron as \texttt{celegans277.mat} from \url{www.biological-networks.org/?page_id=25} \cite{choe2004network}.
Coordinates for three neurons (AIBL, AIYL, SMDVL) were missing in this dataset, and were reconstructed by assigning identical coordinates to the corresponding contralateral neurons (AIBR, AIYR, SMDVR) \cite{Varier2011}.
To examine the influence of anatomical location, each neuron was labeled by its anatomical location, as:
(i) `head', (ii) `tail', or (iii) `body', using data from release WS256 of WormBase \cite{Harris:2009kd}, (\url{ftp://ftp.wormbase.org/pub/wormbase/releases/WS256/ONTOLOGY/anatomy_association.WS256.wb}).
These annotations were assigned to individual neurons using the anatomical hierarchy defined in WormBase, which we retrieved using the WormBase API (WormMine: \url{intermine.wormbase.org}) \cite{Harris:2009kd}, propagating each term down the hierarchy to individual neurons.
We manually corrected the assignment of twelve head neurons (ALA, AVFL, AVFR, AVG, RIFL, RIFR, RIGL, RIGR, SABD, SABVL, SABVR, SMDVL), which were assigned as `head' in WormAtlas \cite{WormAtlas} but not on WormBase.
To examine the influence of neuronal subtype, all neurons were labeled to one (or multiple) of the following three categories:
(i) `sensory' (have clear sensory specializations),
(ii) `motor' (make synaptic contacts onto muscle cells), or
(iii) `interneuron' (receive synapses from and send synapses onto other neurons) \cite{White:1986tx}.
A total of 79 neurons are annotated as sensory, 121 annotated as motor, and 97 annotated as interneurons (including eighteen neurons assigned to two categories: five are both `interneuron' and `sensory', seven are `interneuron' and `motor', and six are both `sensory' and `motor' neurons).
To examine the neurotransmitter systems used by each neuron, neurons were labeled by matching to Table 2 of Pereira et al.~\cite{Pereira:2015er}.
In order to determine the influence of neuron birth time, we obtained neuronal birth time information in minutes from the Dynamic Connectome Lab website (\url{https://www.dynamic-connectome.org/?page_id=25}), \cite{Varier2011}.
To assess the influence of lineage similarity, we obtained a measure of lineage distance for all pairs of neurons from previously published embryonic and post-embryonic lineage trees \cite{Sulston1977, Sulston1983}, using data downloaded from WormAtlas (\url{http://www.wormatlas.org/neuronalwiring.html#Lineageanalysis}) \cite{WormAtlas}.
In this dataset, the closest common ancestor neuron was identified for each pair of neurons, and then the lineage distance was calculated as the number of cell divisions from the closest common progenitor neuron.

% METHODS: rich club analysis
\subsection*{Network analysis}
In this section we describe the network analysis methods used to characterize the \emph{C. elegans} connectome.

\paragraph{Degree.}
Neuronal connectivity is most simply quantified by counting the number of neurons that a given neuron projects to, known as its out-degree, $k_\mathrm{out}$, or by counting the number of neurons that a given neuron receives projections from, known as its in-degree, $k_\mathrm{in}$.
These quantities can be summed to give the total number of connections involving a given neuron, $k_\mathrm{tot} = k_\mathrm{in} + k_\mathrm{out}$, which we refer to as simply the degree, $k$, throughout this work.
At a given degree threshold, $k$, each neuron was classified as either a `hub' (degree $>k$) or a `nonhub' (degree $\leq k$).
All connections were subsequently classified in terms of their source and target neurons as either
`rich' (hub $\rightarrow$ hub, or hub $\leftrightarrow$ hub),
`feed-in' (nonhub $\rightarrow$ hub),
`feed-out' (hub $\rightarrow$ nonhub),
or `peripheral' (nonhub $\rightarrow$ nonhub, or nonhub $\leftrightarrow$ nonhub).

\paragraph{Rich-club organization.}
We used the rich-club coefficient, $\phi(k)$, to quantify the interconnectivity of hub neurons:
\begin{equation}
    \label{eqn:rich_club}
    \phi(k) = \frac{2E_{>k}}{N_{>k}(N_{>k}-1)},
\end{equation}
where $N_{>k}$ is the number of nodes with degree $>k$, and $E_{>k}$ is the number of edges between them \cite{Colizza:2006kz}.
Thus, $\phi(k)$ measures the link density in the subgraph containing nodes with degree $>k$.
Because $\phi(k)$ invariably increases with $k$ (as nodes with higher degree make more connections, yielding a higher expected link density in the subgraph containing nodes with degree $>k$), we compared $\phi(k)$ measured from the \emph{C. elegans} connectome to the mean value of an ensemble of randomized null networks, $\phi_\mathrm{rand}(k)$ \cite{Colizza:2006kz}.
An ensemble of 1000 null networks was generated by shuffling the links in the empirical network while retaining the same degree sequence \cite{Maslov:2002hi} (rewiring each edge an average of 50 times per null network) using the \texttt{randmio\_dir} function from the Brain Connectivity Toolbox \cite{Rubinov:2010jd}.
The normalized rich-club coefficient, $\Phi_\mathrm{norm}(k)$, was computed as the ratio of the rich-club coefficient of the empirical network to the mean rich-club coefficient of the ensemble of randomized networks:
\begin{equation}
    \label{eqn:rich_club_norm}
    \Phi_\mathrm{norm}(k) = \frac{\phi(k)}{\langle \phi_\mathrm{rand}(k) \rangle}.
\end{equation}
Values of $\Phi_\mathrm{norm} > 1$ indicate rich-club organization of the network, with statistically significant deviations assessed by computing a $p$-value directly from the empirical null distribution, $\phi_\mathrm{rand}(k)$, as a permutation test \cite{vandenHeuvel:2011he}.

\paragraph{Modularity.}
Another important topological property of neural systems is modularity, whereby network nodes coalesce into tightly connected subsystems that are thought to serve a common function \cite{Sporns2016}.
To investigate whether our results were influenced by the well-known modular organization of the \emph{C. elegans} connectome \cite{Kim:2014bu, Pan:2010jt, Bassett2010, Achacoso:1992ay, Pavlovic:2014gx}, we decomposed the network into modules using two methods.
The first method involved applying the Louvain community detection algorithm \cite{Blondel:2008do} using the \texttt{community\_louvain} function in the Brain Connectivity Toolbox \cite{Rubinov:2010jd}.
To identify the optimal modular assignment, neurons were assigned to modules using consensus clustering across 1000 repeats of the Louvain algorithm \cite{Lancichinetti2012}, weighting each partition by its modularity, $Q$, using the \texttt{agreement\_weighted} and \texttt{consensus\_und} functions of the BCT\cite{Rubinov:2010jd}.
The second modular decomposition was taken from a previously-reported nine-module partition derived from an Erd\"os-R\'enyi Mixture Model (ERMM) \cite{Pavlovic:2014gx}.

% METHODS: gene expression
\subsection*{Gene expression}
Gene expression is represented as a binary indicator of which genes are expressed in a given neuron using data from many individual experiments compiled into a unified data resource on WormBase \cite{Harris:2009kd}.
We use release WS256 of this dataset (\url{ftp://ftp.wormbase.org/pub/wormbase/releases/WS256/ONTOLOGY/anatomy_association.WS256.wb}) and analyze annotations made `directly' to individual neurons, excluding `uncertain' annotations (see \nameref{text:expressionAnnotations}).
We denote the expression of a gene in a neuron as a `1', and other cases as a `0'.
Note that a value of `0' indicates either:
(i) `gene is not expressed' or
(ii) `there is no information on whether the gene is expressed'.
Expression data are sparse, in part due to incomplete annotations---an average of 30 genes are expressed in each neuron (range: 3 to 138 genes), and each gene is expressed in an average of 9 neurons (range: 1 to 148 neurons).
A total of 948 genes are expressed in at least one neuron, allowing us to represent the full expression dataset as a binary 279 (neuron) $\times$ 948 (genes) matrix, shown in Fig.~\ref{fig:Fig1}C.


\subsection*{Correlated gene expression}
Our primary aim in this work is to understand how pairwise patterns of neuronal connectivity (shown in Fig.~\ref{fig:Fig1}A) relate to coupled expression across 948 genes between pairs of neurons (i.e., pairs of rows of Fig.~\ref{fig:Fig1}C).
To estimate the coupling between neuronal gene expression profiles, we required a similarity measure for pairs of neurons that captures their similarity of gene expression profiles, or correlated gene expression (CGE).
We used a binary analogue of the linear Pearson correlation coefficient, the mean square contingency coefficient \cite{Warrens2008}:
\begin{equation} \label{eq:rphi}
    r_\phi = \frac{n_{11}n_{00} - n_{10}n_{01}}{\sqrt{n_{1\bullet}n_{0\bullet}n_{\bullet 0}n_{\bullet 1}}},
\end{equation}
for two vectors, $x$ and $y$, of length $L (=948)$, where $n_{xy}$ counts the number of observations of each of the four outcomes (e.g., $n_{10} = \sum_i \delta_{x_i,1}\delta_{y_i,0}$ counts the number of times $x=1$ and $y=0$), and the symbol $\bullet$ sums across a given variable (e.g., $n_{\bullet 0} = \sum_i \delta_{y_i,0}$ counts the number of times $y = 0$).
The coefficient assumes a maximum value $r_\phi = 1$ when $x$ and $y$ are identical (such that $n_{11} + n_{00} = L$), and a minimum value $r_\phi = -1$ when $x$ and $y$ are always mismatched (such that $n_{10} + n_{01} = L$).
Note that we use the notation $r_\phi$ to denote the phi coefficient, Eq.~\eqref{eq:rphi}; this notation should not be confused with the rich-club coefficient, $\phi(k)$.

One concern about applying this measure to sparsely annotated data is that it may be biased by differences in the number of expressed genes in a neuron, which ranged from 3 (0.3\% of 948 genes analyzed here) to 138 (14.6\%).
To explore this further, we compared $r_\phi$ with several other commonly used metrics of association between binary vectors including Yule's $Q$ and the Jaccard index.
While these other binary similarity metrics exhibited strong bias with the proportion of gene expression annotations made to a given neuron, $r_\phi$ was not biased (see \nameref{S2_Fig} and \nameref{text:coexprSensitivity}).

The 92 bilateral pairs of neurons (e.g., AVAL/AVAR, CEPVL/CEPVR, etc.) exhibit highly correlated gene expression patterns: all bilateral pairs of neurons have $r_\phi > 0.8$, and 96\% of bilateral pairs have $r_\phi > 0.95$.
Although including bilateral pairs of neurons do not change the main results of this paper, to ensure that our results are not driven by the high CGE between bilateral pairs of neurons, we excluded CGE values of bilateral pairs of neurons from all analyses involving CGE.

\subsection*{Gene scoring and enrichment}
Our CGE measure, $r_\phi$, quantifies the similarity between the expression profiles of two neurons across all 948 genes.
To further investigate the role of individual genes in producing different CGE patterns, we developed a method for scoring the contribution of each individual gene to the overall correlation coefficient, following prior work using continuous gene expression data \cite{Fulcher:2016ck}.
Performing similar analyses with \emph{C. elegans} data poses additional challenges due to:
(i) binary expression data, making robust quantification difficult;
(ii) sparse and incomplete data, posing statistical problems for quantifying a signal in genes with limited expression;
and (iii) low genome coverage (less than 5\% of the protein coding genes in \emph{C. elegans}), constraining our ability to perform a comprehensive enrichment analysis, e.g., using the Gene Ontology (GO) \cite{Ashburner2000}.

Given that $r_\phi$ treats mutual gene expression (i.e., cases in which a gene is expressed in pairs of neurons, $n_{11}$) the same as mutual absence of gene expression ($n_{00}$), we started by developing a new analytic measure of the probability of mutual gene expression, given its clearer biological interpretation (see \nameref{text:matchingIndex}).
This measure was not biased by differences in the relative number of expressed genes (\nameref{S2_Fig}D) and yields qualitatively similar outputs to $r_\phi$ on our data.
Thus, while we use $r_\phi$ throughout this work for its ease of interpretation (as an analogue of the conventional correlation coefficient), our new probability-based CGE measure allowed us to motivate a method for quantifying the contribution of individual genes (and functional groups of genes) to patterns of CGE that addresses some of the above-mentioned challenges.
Note that our main findings, obtained using $r_\phi$, are replicated using our new CGE measure (cf.~\nameref{S3_Fig}).

As a starting point, we quantified the contribution of individual genes to differences in CGE for different categories of neuron pairs, specifically for (i) increased CGE in connected compared to unconnected pairs of neurons, and (ii) increased CGE in rich and feeder compared to peripheral connections.
Note that our method scores genes on their contribution to differences in CGE between categories of pairwise connections.
We first scored each gene for whether it is more likely to be expressed in a given class of neuron pair over another as the probability of obtaining at least as many matches (defined as expression in both neurons of a pair) as observed under a random CGE null model using the binomial distribution:

\begin{eqnarray}
	\label{eq:CBinomialProbability}
     p^{(a)} = 1 - \sum_{i=0}^{m-1}\binom{n}{i} p_\mathrm{class}^{i}(1-p_\mathrm{class}^{n-i}),
\end{eqnarray}
where $m$ is the number of matches (a match indicates that a given gene was expressed in both neurons) on the class of neuron pairs of interest, $n$ is the total number of matches across all neuron pairs considered in the analysis, $p_\mathrm{class} = n_\mathrm{class}/M$ is the probability of the given class of inter-region pairs, as the total number of neuron pairs of that class, $n_\mathrm{class}$, divided by the maximum number of possible neuron pairs, $M$, for a given gene, indexed with $a$.
This score, $p^{(a)}$, can be interpreted as a $p$-value under the null hypothesis that the number of expression matches of gene $a$ is consistent with a purely random pattern of matches/mismatches across edges, giving lower values to genes with more matches in the edge class of interest (compared to an alternative set of edges) than expected by chance.
For reasons described earlier, bilateral pairs of neurons were excluded from all scoring procedures and, to ensure that each gene contributes a meaningful score, we imposed a data quality threshold on the number of possible matches, $n \geq 10$.

Our first analysis compares two mutually exclusive types of neuron pairs:
(i) all pairs of neurons that are connected by at least one chemical or electrical synapse, and
(ii) all pairs of neurons that are unconnected.
For this analysis, $p_\mathrm{class} = 0.059$ is the proportion of neuron pairs that are connected, $n$ is the total number of neuron pairs that both exhibit expression of gene $a$, and $m$ is the number of neuron pairs that are structurally connected for which both neurons express gene $a$.
A total of 414 (/948) genes had $n \geq 10$ for this analysis.
Our second analysis compares pairs of connected neurons for which at least one is a hub (i.e., rich, feed-in, or feed-out connections), to pairs in which both neurons are nonhubs (i.e., peripheral connections).
In this case, $p_\mathrm{class} = 0.35$ is the proportion of connected pairs of neurons that involve hubs, $n$ is the number of connected neuron pairs for which gene $a$ is expressed in both, and $m$ counts the number of connected neuron pairs involving hubs for which gene $a$ is expressed.
A total of 168 (/948) genes had $n \geq 10$ for this analysis.

As well as interpreting the list of individual genes that were significant after correcting for multiple hypothesis comparison, we performed an over-representation analysis (ORA) using the genes that contribute most to a given connectivity pattern to assess whether any gene sets (GO categories) were statistically over-represented in this list.
To obtain the gene list, we used the false discovery rate correction of Benjamini and Hochberg \cite{Benjamini:1995cd} on $p$-values computed using Eq.~\eqref{eq:CBinomialProbability}, which were thresholded at a stringent level of $p = 10^{-4}$ (corresponding to approximately the top 20\% of genes in each analysis).
Over-representation for each biological process GO category with 5 to 100 genes available was quantified as an FDR-corrected $p$-value (across around 700 GO categories) using version 3.0.2 of \emph{ErmineJ} software \cite{Gillis2010}.
Biological process GO annotations \cite{Ashburner2000} were obtained from GEMMA \cite{Zoubarev2012} (using \texttt{Generic\_worm\_noParents.an.txt.gz} downloaded on March 31, 2017).
Gene Ontology terms and definitions were obtained in RDF XML file format downloaded from \url{archive.geneontology.org/latest-termdb/go_daily-termdb.rdf-xml.gz} on March 31 2017.

\section*{Results}

A schematic overview of our data is in Fig.~\ref{fig:Fig1}, including
the directed binary connectome (Fig.~\ref{fig:Fig1}A),
additional anatomical data gathered for each neuron (Fig.~\ref{fig:Fig1}B),
and binary gene expression across 948 genes (Fig.~\ref{fig:Fig1}C).
Our analysis is presented in five parts:
(i) given past evidence for a major effect of physical distance on connection probability and CGE \cite{Fulcher:2016ck}, we first characterize the spatial dependency of connection probability and CGE;
(ii) we confirm the rich-club organization of the \emph{C. elegans} connectome;
(iii) we show that CGE is increased in connected pairs of neurons relative to unconnected pairs, in electrical synapses relative to chemical synapses, and in connected hub neurons relative to other types of connected neuron pairs (mirroring previous results in the mesoscale mouse connectome \cite{Fulcher:2016ck});
(iv) we demonstrate that high CGE between connected hub neurons is not driven by factors like stereotypical interneuron expression, birth time, lineage similarity, neuromodulator types or expression similarity within modules, but may be driven by the high CGE of command interneurons;
(v) we characterize the contribution of specific genes, and broader gene ontology categories, to the observed patterns.

\begin{figure}[h]
  \centering
 \caption{\textbf{Schematic representation of the data used in this study.}
All plots show neurons (rows) ordered by anterior-posterior along the longitudinal axis, from the top of the head (upper, left) to the bottom of the tail (lower, right).
(A) Connectivity matrix summarized 2990 directed chemical and electrical connections between 279 neurons from neuron $i$ (row) to neuron $j$ (column).
Connections are colored according to how they connect hubs ($k > 44$) and nonhubs ($k \leq 44$), as `rich' (hub $\rightarrow$ hub), `feed-in' (nonhub $\rightarrow$ hub), `feed-out' (hub $\rightarrow$ nonhub), and `peripheral' (nonhub $\rightarrow$ nonhub).
(B) Neurochemistry (types as labeled), anatomical location (as labeled), birth time (from early born neurons, black, to late-born neurons, white), hub assignment (hubs labeled red), and functional type (as labeled).
(C) Binary gene expression indicated as a green dot when a gene (column) is expressed in a neuron (row).}
\label{fig:Fig1}
\end{figure}

\subsection*{Spatial dependency}
Previous work has demonstrated the importance of spatial effects in driving patterns of gene expression, with more proximal brain areas \cite{Horvat:2016ia, Wang:2016gg, Markov:2013jo, Henderson:2014fg, Fulcher:2016ck, Noori:2017ce, Levy:2012dy, Azulay:2016cg} having both a increased connection probability and more similar gene expression profiles \cite{Krienen:2016eq, Fulcher:2016ck, Pantazatos:2016ir, Richiardi:2017hb} than more distance brain areas.
Unlike network analyses of mammalian brains, where all neurons are confined to a spatially contiguous organ, neurons of the \emph{C. elegans} nervous system are distributed throughout the entire organism, forming a dense cluster of 147 neurons in the head (all within 130\,$\mu$m), 105 sparser neurons in the body (spanning 1.02\,mm), which are predominantly motor neurons (75\%), and another dense cluster of 27 neurons in the tail (all within 90\,$\mu$m of each other), as plotted in Fig.~\ref{fig:Fig2}.
In order to examine the relationship between connectivity and CGE, we need to understand the spatial dependence of both connectivity and CGE to characterize the extent to which previously reported spatial dependencies of these measurements apply to the microscale nervous system of \emph{C. elegans}.

\begin{figure}[h]
\centering
\caption{
\textbf{Hub neurons are contained within the head and tail of \emph{C. elegans}}.
Neurons are positioned along the anterior--posterior (horizontal), and dorsal--ventral (vertical) axes, and are colored by type:
(i) interneuron (85 neurons, orange),
(ii) sensory (68 neurons, blue),
(iii) motor (108 neurons, green), or
(iv) multiple assignments (18 neurons, yellow).
Hub neurons (i.e., neurons with $k > 44$, see Fig.~\ref{fig:Fig5}) are shown as larger circles and outlined in black.
`Rich-club' connections between hub neurons are shown (red), and all other connections are also shown in the upper plots (gray).
Axes of each subplot are to scale with each other, and the upper zoomed-in plots of the head and tail are shown as dotted rectangles in the lower plot.
\label{fig:Fig2}
}
\end{figure}

We first characterize the probability that two neurons will be connected given their source and target types, labeling each neuron as being in either the `head', `body', or `tail' of \emph{C. elegans}.
Connection probability is plotted as a function of Euclidean separation distance in Fig.~\ref{fig:Fig3} for each combination of source and target neuron labels, across 10 equiprobable distance bins (with exponential fits added for visualization).
Distinguishing connections by source and target neuron types uncovers clear spatial relationships (that are obscured when all connections are grouped, as in \cite{Azulay:2016cg}), that differ across connection classes.
From the very short distance scale of $\lessapprox 100\,\mu$m of head$\rightarrow$head and tail$\rightarrow$tail connections to the very longest-range head$\rightarrow$tail and tail$\rightarrow$head connections ($\gtrapprox 1\,$mm), connection probability decreases with separation distance (Fig.~\ref{fig:Fig3}A).
For connections between pairs of neurons located in the body, ranging up to $\approx 1$\,mm, a near-exponential trend is exhibited, mirroring results in other species and across longer length scales \cite{Wang:2016gg}, including mouse \cite{Goulas:2016hr, Fulcher:2016ck}, and in rodents and primates \cite{Horvat:2016ia}.
Other connections do not exhibit strong spatial connectivity relationships, i.e., connections between the body and head or between the body and tail, shown in Fig.~\ref{fig:Fig3}B.

\begin{figure}[h]
  \centering
  \caption{
\textbf{Connection probability decreases with separation distance within and between the head and tail, and within the body.}
The connection probability for a pair of neurons as a function of their Euclidean distance is estimated in 10 equiprobable distance bins, shown as a circle (bin centers) and a horizontal line (bin extent).
There is a decreasing relationship for connections: within the head (aqua), from head$\rightarrow$tail (brown) and from tail$\rightarrow$head (stone blue), within the tail (red), and within the body (dark purple).
Exponential fits of the form $f(x) = A\exp(-\lambda x) + B$, some of which appear approximately linear across the range of the data, are shown as dotted lines.
(B)
Plots as in (A), but for connection classes between the body and head/tail: from body$\rightarrow$head (forest green), from body$\rightarrow$tail (dirt green), connections from head $\rightarrow$ body (purple), and from tail$\rightarrow$body (dark brown).
Apart from a small effect at short range for tail$\rightarrow$body connections, these connection classes show minimal distance dependence.
  }
  \label{fig:Fig3}
\end{figure}


We next investigate the dependence of CGE, $r_\phi$, on the separation distance between neuron pairs, shown in Fig.~\ref{fig:Fig4}.
CGE decreases slightly with separation distance for the spatially close neurons within the head (Fig.~\ref{fig:Fig4}A) and within the tail (Fig.~\ref{fig:Fig4}B), but not for pairs of neurons involving the body (Fig.~\ref{fig:Fig4}C).
The decreasing trend in CGE with distance within the head and tail is primarily driven by a subset of nearby neurons with high $r_\phi$.
It may therefore represent a relationship specific to particular functionally related neurons, rather than a general, bulk spatial relationship seen in macroscopic mammalian brains \cite{Fulcher:2016ck}.
Accordingly, attempting to correct for a bulk, non-specific trend by taking residuals from an exponential fitted to the relationship produced artifactual reductions in the CGE of many neuron pairs (shown in \nameref{S4_Fig}).
Thus, we found no evidence for bulk spatial relationships of $r_\phi$ in the neuronal connectome of \emph{C. elegans}.

\begin{figure}[h]
  \centering
  \caption{
  \textbf{Dependence of correlated gene expression, $r_\phi$, on spatial separation between pairs of neurons.}
  Correlated gene expression, $r_\phi$ (excluding bilateral homologous pairs of neurons), is shown as a function of the pairwise separation distance between pairs of neurons (shown as the mean (solid) $\pm$ standard deviation (dotted) in seven equiprobable distance bins, with extent shown as horizontal bars), for (A) all pairs of neurons in the head, (B) all pairs of neurons in the tail, and (C) all other pairs (labeled).
  Scatters for all neuron pairs are added in (A) and (B).
  An exponential relationship, $f(x) = A\exp(-\lambda x) + B$, is fitted in (A) and (B).
  The weak decreasing trend in $r_\phi$ with distance, is primarily driven by a small subset of nearby neurons with high $r_\phi$, and may therefore represent a more specific relationship between particular neurons, rather than a general, bulk spatial relationship observed in macroscopic mammalian brains \cite{Fulcher:2016ck, Krienen:2016eq}.
\label{fig:Fig4}
  }
\end{figure}

\subsection*{Hub connectivity}

Next we analyze the topological properties of the \emph{C. elegans} connectome, represented as a directed, binary connectivity matrix between 279 neurons, combining directed chemical synapses and undirected electrical gap junctions (Fig.~\ref{fig:Fig1}).
The degree distribution is shown in Fig.~\ref{fig:Fig5}A, where neurons are labeled as sensory neurons, interneurons, motor neurons, or neurons with multiple functional assignments.
Consistent with the results of Towlson et al., who used an undirected version of the connectome (by ignoring the directionality of chemical synapses), we found a positively-skewed degree distribution containing an extended tail of high-degree hub interneurons.
Hub interneurons of \emph{C. elegans} are mostly command interneurons and mediate behaviors like coordinated locomotion and foraging \cite{tsalik2003}.

\begin{figure}[h]
   \centering
 \caption{\textbf{Rich-club organization of the \emph{C. elegans} connectome}.
(A) Degree distribution of neurons, labeled to four categories:
(i) interneuron (85 neurons, orange),
(ii) motor (108 neurons, green),
(iii) sensory (68 neurons, blue), or
(iv) multiple assignments (18 neurons, yellow).
The distribution features an extended tail of high-degree interneurons.
(B)
Normalized rich club coefficient, $\Phi_\mathrm{norm}$ (red), as a function of the degree, $k$, at which hubs are defined (as neurons with degree $>k$).
Also shown is the mean Euclidean separation distance, $d$ (purple) between connected hub regions (across degree thresholds, $k$).
$\Phi_\mathrm{norm} > 1$ indicates that hubs are more densely interconnected among each other than expected by chance, with red circles indicate values of $\Phi_\mathrm{norm}$ that are significantly higher than an ensemble of 1\,000 degree-matched null networks ($p < 0.05$).
Purple circles indicate where the Euclidean distance between connected pairs of hubs is significantly greater than the Euclidean distance for all other pairs of connected regions (right-tailed Welch's $t$-test, $p < 0.05$).
}
 \label{fig:Fig5}
 \end{figure}
 
Using the normalized rich-club coefficient, $\Phi_\mathrm{norm}$, to quantify the extent to which hubs are densely interconnected, we confirmed the results of Towlson et al. \cite{Towlson2013}, finding rich-club organization in the connectome, as shown in Fig.~\ref{fig:Fig5}B.
The figure plots the variation of $\Phi_\mathrm{norm}$ across degree thresholds, $k$, at which hubs are defined (as neurons with degree $>k$), with red circles indicating a significant increase in link density among hubs relative to 1000 degree-preserving nulls (permutation test, $p < 0.05$).
The plot reveals rich-club organization ($\Phi_\mathrm{norm} > 1$) at the upper tail of the degree distribution, particularly across the range $44 < k < 63$, shaded gray in Fig.~\ref{fig:Fig5}B.
Similar results were obtained using weighted representations of the connectome (i.e., using information about the number of synapses in the connectivity network) for two different definitions of the weighted rich-club coefficient \cite{Opsahl2008}, shown in \nameref{S5_Fig}.
Throughout this work, we define a set of hubs as the sixteen neurons with $k > 44$, which corresponds to the lowest degree threshold at which the network displays a contiguous region of significant rich-club organization at high $k$.
Our list of hubs includes all of the 11 hub neurons of Towlson et al.~\cite{Towlson2013} at $3 \sigma$ (see \nameref{tab:HubList}), with five additional hubs identified in our analysis of the directed connectome.

The rich-club connectivity of the \emph{C. elegans} connectome is accompanied by an increase in mean hub-hub connection distance \cite{Towlson2013}, with a significant increase through the topological rich-club regime (right-tailed Welch's $t$-test, $p < 0.05$), shown in Fig.~\ref{fig:Fig5}B.
This can be attributed to a relative increase in long-distance hub-hub connections between the head and tail, shown in Fig.~\ref{fig:Fig2} (cf. \nameref{S6_Fig}).
The high connection density and long mean anatomical distance between pairs of hub neurons counters the general trend in the \emph{C. elegans} connectome, where the probability of connectivity between two neurons decays with their separation distance (Fig.~\ref{fig:Fig3}).
These results are consistent with previous findings across diverse neural systems and suggest that the rich club may provide a central yet costly backbone for neuronal communication in \emph{C. elegans} \cite{vandenHeuvel:2012kh, Towlson2013}.

%%
\subsection*{Correlated gene expression and connectivity}

We next investigate how the network connectivity properties of \emph{C. elegans} relate to patterns of CGE, using the mean square contingency coefficient, $r_\phi$.
To test whether CGE varies as a function of connectivity, we compared the distribution of $r_\phi$ between
(i) all connected pairs of neurons, and
(ii) all unconnected pairs of neurons.
Connected pairs of neurons have more similar expression profiles than unconnected pairs (Wilcoxon rank-sum test, $p = 1.8 \times 10^{-78}$).
Figure~\ref{fig:Fig6}A (left) shows distributions of $r_\phi$ for:
(i) all pairs of neurons that are connected via an electrical gap junctions (474 pairs, after excluding bilateral pairs),
(ii) all pairs of neurons that are connected via reciprocal (291 pairs) and,
(iii) unidirectional chemical synapses (1721 pairs) as well as
(iv) all pairs of neurons that have neither connection (36\,450 pairs).
Note that 175 pairs of neurons are connected by both a chemical synapse and a gap junction, and are thus included in both chemical and electrical categories.
Amongst connected pairs of neurons, those connected via gap junctions exhibit more similar gene expression profiles than those connected via chemical synapses (Wilcoxon rank-sum test, $p = 5.4 \times 10^{-22}$).
We found no difference in CGE between pairs of neurons connected reciprocally by chemical synapses ($N_1 \leftrightarrow N_2$ for two neurons $N_1$ and $N_2$) versus those connected unidirectionally ($N_1 \rightarrow N_2$) (Wilcoxon rank-sum test, $p = 0.99$).

 \begin{figure}[h]
 \centering
\caption{{\bf Correlated gene expression varies as a function of connectedness and connection type.}
(A) \textit{Left}: distribution of CGE for (i) pairs of neurons connected by gap junctions, (ii) pairs of neurons connected by reciprocal chemical synapses, (iii) pairs of neurons connected by unidirectional chemical synapses, (iv) pairs of neurons that are unconnected, shown as a violin plot, with the median of each distribution represented by a horizontal line.
CGE is increased in connected (electrical or chemical; reciprocally or unidirectionally) pairs of neurons relative to unconnected pairs ($p = 1.8 \times 10^{-78}$, Wilcoxon rank sum test).
Among connected pairs of neurons neurons connected via gap junctions have more similar CGE than connected via chemical synapses (Wilcoxon rank-sum test, $p = 5.4 \times 10^{-22}$).
\textit{Right}: GCE for pairs of neurons labeled as peripheral, feed-in, feed-out, and rich, where hubs are neurons with degree $k>44$. The median of each distribution shown as a horizontal line.
CGE is significantly higher between hubs (rich links) compared to feeder ($p = 5 \times 10^{-22}$, Wilcoxon rank sum test) and peripheral ($p = 3.9 \times 10^{-19}$, Wilcoxon rank sum test) links.
Feed-out links show significantly higher CGE than both feed-in ($p = 1.9 \times 10^{-6}$, Wilcoxon rank sum test) and peripheral links ($p = 4.5 \times 10^{-12}$, Wilcoxon rank sum test).
(B)
\emph{Top}: Degree distribution, $k$, of the \emph{C. elegans} connectome.
\emph{Middle}: proportion of connections that are:
`rich' (hub$\rightarrow$hub, red),
`feed-in' (nonhub$\rightarrow$hub, yellow),
`feed-out' (hub$\rightarrow$nonhub, orange), or
`peripheral' (nonhub$\rightarrow$nonhub, blue) as a function of the degree threshold, $k$, used to define hubs.
Note that at high $k$ most neurons are labeled as nonhubs and hence the vast majority of connections are labeled `peripheral'.
 \emph{Bottom}: Median CGE, $\tilde{r}_\phi$, for each connection type as a function of $k$.
The median CGE across all network links is shown as a dotted black line; the topological rich-club regime (determined from the network topology, cf. Fig.~\ref{fig:Fig5}) is shaded gray.
Circles indicate a statistically significant increase in CGE in a given link type relative to the rest of the network (one-sided Wilcoxon rank-sum test, $p < 0.05$).
}
 \label{fig:Fig6}
\end{figure}

% ---HUB CGE---
We next investigated whether CGE varies across different types of connections defined in terms of their hub connectivity.
For a given hub threshold, $k$, we first labeled each neuron as either a `hub' (nodes with degree $> k$) or a `nonhub' (degree $\leq k$), and then labeled each connection as either `rich' (hub $\rightarrow$ hub), `feed-in' (nonhub $\rightarrow$ hub), `feed-out' (hub $\rightarrow$ nonhub), or `peripheral' (nonhub $\rightarrow$ nonhub).
The median CGE, $\tilde{r}_\phi$, of each of these four connection types is plotted in Fig.~\ref{fig:Fig6}B, with circles indicating statistically significant increases of a given connection type relative to all other connections (one-sided Wilcoxon rank-sum test, $p < 0.05$).
Correlated gene expression in rich connections increases with degree, $k$, particularly in the topological rich-club regime where hubs are densely interconnected (shaded gray in Fig.~\ref{fig:Fig6}B).
In this topological rich-club regime, both feed-in and feed-out connections exhibit increased CGE relative to peripheral connections, which show the lowest levels of CGE.
Full distributions of $r_\phi$ for each edge type at a hub threshold of $k > 44$ are in Fig.~\ref{fig:Fig6}A (right).
This plot shows that, compared to all different types of pairs of neurons, connected pairs of hubs showed the highest CGE.

In summary, our results reveal:
(i) increased CGE in connected pairs of neurons;
(ii) the highest CGE in rich connections; and
(iii) lowest CGE in peripheral connections.
These results, obtained using incomplete binary annotations of gene expression across 948 genes in a microscale neuronal connectome, are consistent with a prior analysis of the expression of over 17\,000 genes across 213 regions of the mesoscale mouse connectome \cite{Fulcher:2016ck}.

\subsection*{Potential drivers of elevated correlated gene expression between hubs}
The sixteen hub neurons in \emph{C. elegans} ($k > 44$):
are all interneurons,
are all located in either the head or tail,
are mostly contained within a single topological module of the network,
are mostly cholinergic (13/16),
and are all born prior to hatching.
We therefore investigated whether the similarity of gene expression profiles between hubs is specific to their high levels of connectivity, or whether it could instead be driven by these other characteristics.

\paragraph{Interneurons.}
The sixteen hubs in \textit{C. elegans} are all interneurons.
To determine whether the increase in CGE in rich connections was specific to interneurons, we plotted the median CGE for hub-hub connections, $\tilde{r}_\phi$, as a function of the degree threshold, $k$, separately for connections involving interneurons, sensory neurons, and motor neurons, as shown in Fig.~\ref{fig:Fig7}B.
For the curve labeled `sensory', for example, each point is the median $\tilde{r}_\phi$ across connections involving sensory neurons (i.e., at least one neuron of a connected pair is a sensory neuron), for which both neurons have degree $>k$.
The increase in median hub-hub CGE is strongest for connections involving interneurons.
Motor neurons show a smaller increase with $k$, although the absence of motor and sensory neurons with high {$k$ makes it difficult to draw firm conclusions.
However, we do find that CGE is higher for hub-hub pairs of interneurons compared to connections between all pairs of nonhub interneurons (Wilcoxon rank sum test, $p = 5 \times 10^{-21}$), indicating that the high CGE of rich pairs cannot simply be related to the fact that all hub neurons are interneurons (Fig.~\ref{fig:Fig7}C).
We next investigated whether greater CGE of hub-hub pairs of neurons could be driven by specific anatomical properties of hub interneurons.
Specifically, we selected a subset of nonhub interneurons that most closely resemble the anatomical properties of hub interneurons in terms of their position and projection pattern; that is, the cells are in similar locations in the head and their axons project to similar targets in the tail. These neurons were AVFL, AVFR, AVHL, AVHR, AVKR, AVJL, and AVJR.
Pairs of hub interneurons show higher median CGE than pairs of anatomically-matched nonhub interneurons, with the difference being at the threshold of statistical significance (Wilcoxon rank sum test, $p = 0.051$), suggesting that the increase in CGE amongst hub interneurons is not a consequence of them being interneurons.

\begin{figure}[h]
\centering
 \caption{
\textbf{Correlated gene expression is highest for hub interneurons.}
(A) The number of connected neuron pairs involving interneurons (orange), sensory neurons (blue), and motor neurons (green) across degree threshold, $k$, represented as $\log_{10}$(number of links).
(B) Median CGE as a function of degree for connections involving different types of neurons.
Circles indicate a statistically significant increase in CGE in a given link type relative to the rest of the network (one-sided Wilcoxon rank-sum test, $p < 0.05$).
(C) CGE distributions for connected pairs of hub interneurons (red) and connected pairs of non-hub interneurons (dark yellow) (Wilcoxon rank sum test, $p = 5 \times 10^{-21}$). * represents statistically significant difference.
}
 \label{fig:Fig7}
\end{figure}

\paragraph{Modular organization.}

Prior work in humans has shown that functional networks in the brain have elevated transcriptional coupling \cite{Richiardi2015}.
The \emph{C. elegans} connectome has a modular organization, with prior work decomposing it into:
(i) modules of neurons with dense intra-module connectivity (and relatively sparse connectivity between modules) \cite{Kim:2014bu, Pan:2010jt, Bassett2010}, or
(ii) groups of neurons with more similar connectivity patterns within groups than between groups \cite{Achacoso:1992ay, Pavlovic:2014gx}.
We therefore examined the association between topological modularity of the \emph{C. elegans} connectome and CGE.
We used the Louvain community detection algorithm \cite{Blondel:2008do} to extract modules from the \emph{C. elegans} connectome using consensus clustering (see \textit{Methods}).
Four modules were extracted, with eleven hubs in module one (which contains 111 neurons), four hubs in module two (96 neurons), one hub in module three (40 neurons), and no hubs in module four (32 neurons).
We also compared the results of this modular partition of neurons to a previously reported nine-module partition derived from an Erd\"os-R\'enyi Mixture Model (ERMM) \cite{Pavlovic:2014gx}.
For the Louvain consensus modules, CGE, $r_\phi$, was significantly increased for connected neurons in the same module (1552 pairs) relative to connected pairs in different modules (687 pairs) (Wilcoxon rank sum test, $p = 6.6 \times 10^{-4}$), but there was no significant difference between intra-modular connected neurons and inter-modular connected neurons for the nine-module ERMM partition (Wilcoxon rank sum test, $p = 0.46$).
The resolution and type of modular decomposition thus affects the relationship between CGE, connectivity, and modular network structure in \emph{C. elegans}.

We then tested whether connected hubs exhibit more similar CGE within and between modules (for both the consensus Louvain and ERMM modular decompositions).
For pairs of connected neurons within the same module, $r_\phi$ is higher for pairs of hubs than pairs of nonhubs (Wilcoxon rank sum test, $p = 6.9\times 10^{-17}$ for consensus Louvain modules, shown in Fig.~\ref{fig:Fig8}A; $9.3 \times 10^{-7}$ for ERMM partition).
We found a similar result for connected neurons in different modules: pairs of connected hubs exhibit increased CGE than other types of connected pairs of neurons (Wilcoxon rank sum test, $p = 1.6 \times 10^{-5}$ for consensus Louvain modules, shown in Fig.~\ref{fig:Fig8}B; $p = 1.6 \times 10^{-16}$ for ERMM).
Thus, for both types of modular decompositions considered, intra-modular and inter-modular connections involving hub neurons exhibit more correlated gene expression patterns than other intra-modular and inter-modular connections.

\begin{figure}[!h]
\centering
 \caption{
 \textbf{Increased CGE of hub neurons is not driven by modularity, neuronal birth time, or cell lineage distance.}
(A) Distributions of CGE, $r_\phi$, for intra-modular rich (red) non-rich (blue) connections, shown as violin plots with the median shown as a horizontal bar (Wilcoxon rank sum test, $p = 6.9 \times 10^{-17}$).
(B) Distributions of CGE, $r_\phi$, for inter-modular rich (red) and non-rich (blue) connections, shown as violin plots with the median shown as a horizontal bar (Wilcoxon rank sum test, $p = 1.6 \times 10^{-5}$).
(C) Distributions of lineage distance between rich links (red) and non-rich links (blue), plotted as histograms due to a discrete nature of this measure (Wilcoxon rank sum test, $p = 0.079$).
(D) Distributions of CGE, $r_\phi$, between early born hubs (rich links, red) and nonhubs (non-rich links, blue) shown as violin plots with the median shown as a horizontal bar (Wilcoxon rank sum test, $p = 3.9 \times 10^{-22}$).
(E) Distributions of CGE between hub command interneurons (red) and hub non-command interneurons (blue) shown as violin plots with the median shown as a horizontal bar (Wilcoxon rank sum test, $p = 3.3 \times 10^{-8}$).
* represents statistically significant differences.
}
 \label{fig:Fig8}
\end{figure}

\paragraph{Lineage distance.}
The lineage distance between a pair of neurons is defined as the sum of total divisions that have taken place since the most recent common ancestor cell \cite{Pavlovic:2014gx, Sulston1977, Sulston1983}.
In the mammalian brain, neuronal lineage has been associated with both functional properties \cite{Ciceri2013, Li2012} as well as connectivity \cite{Yu2012}.
Moreover, tissue distance (resembling lineage distance on a cellular scale) correlates with gene expression divergence, meaning that tissues from the same branch on the fate map share more similar gene expression patterns in both human and mouse mesoderm as well as ectoderm tissues \cite{Cui2007}.
Given that the ectoderm eventually differentiates to form the nervous system, this finding suggests a possible relationship between lineage distance and CGE in a microscale neuronal system such as that of \textit{C. elegans}.
However, we find no significant correlation between lineage distance and CGE in \textit{C. elegans} (Spearman's $\rho = -0.027$, $p = 0.2$).
As shown in Fig.~\ref{fig:Fig8}C, there was only a weak tendency for the meadian lineage distance lineage distance to be increased in non-rich pairs (Wilcoxon rank sum test, $p = 0.079$).
Thus, we can not attribute the transcriptional similarity of connected hub neurons to their neuronal lineage.


\paragraph{Birth time.}
The genesis of neurons in \emph{C. elegans} is separated into two distinct time periods: before hatching (birth time $<550$ min -- `early-born') and after hatching (birth time $>1200$ min -- `late-born'), with no neurons formed during intermediate times \cite{Varier2011}.
As a broad group, connected pairs of early-born neurons do not exhibit significantly different CGE compared to connected pairs of late-born neurons (Wilcoxon rank sum test, $p = 0.64$), but connected pairs of early-born neurons do exhibit significantly higher CGE relative to pairs of connected neurons for which one neuron is born prior to hatching and the other is born after hatching (Wilcoxon rank sum test, $p = 4.2 \times 10^{-3}$).
Since all \emph{C. elegans} hub neurons are born prior to hatching \cite{Towlson2013} and neurons born at similar times may share similar connectivity properties \cite{Varier2011, Towlson2013}, we investigated whether the increase in CGE between hub neurons of \emph{C. elegans} may be driven by their similar birth times.
Focusing on the 201 neurons born prior to hatching (16 hub and 185 nonhub neurons), connected pairs of hub neurons exhibit significantly increased CGE, $r_\phi$, relative to other pairs of connected early-born neurons (Wilcoxon rank sum test, $p < 10^{-22}$), as shown in Fig.~\ref{fig:Fig8}D.

\paragraph{Neurochemistry.}
Hub neurons ($k > 44$) consist of thirteen cholinergic neurons, two glutamatergic neurons, and one neuron of unknown neurotransmitter type \cite{Pereira:2015er}.
We find that neuron pairs show different CGE relationships as a function of their neurotransmitter type, e.g., pairs of GABAergic neurons have high median CGE, $\tilde{r}_\phi = 0.59$, while pairs of glutamatergic neurons exhibit a relatively low median CGE, $\tilde{r}_\phi = 0.08$.
To determine whether the similarity in CGE between pairs of hub neurons is associated with their neurotransmitter types, we constructed $10^8$ random sets of sixteen neurons of the same neurotransmitter types as hubs (e.g., thirteen random cholinergic neurons, two random glutamatergic neurons, and one random neuron of an unknown neurotransmitter type) and compared the distribution of median CGE of each group to the median CGE of hub neurons as a permutation test.
Neurochemical identity is not associated with the elevated CGE of hub neurons, with hubs displaying a significant increase in median CGE relative to random sets of neurons with the same neurotransmitter types as hubs (permutation test, $p = 3\times10^{-4}$).

\paragraph{Anatomical location.}
Of the sixteen hub neurons, thirteen are in the head and three are in the tail; none are located in the body.
CGE varies as a function of anatomical location (i.e., `head', `body', and `tail'), with pairs of neurons in the same anatomical class exhibiting the highest median CGE (e.g., pairs of neurons within the body have a median $\tilde{r}_\phi = 0.14$, pairs of neurons within the tail have $\tilde{r}_\phi = 0.12$, and pairs of neurons within the head have $\tilde{r}_\phi = 0.07$), followed by lower median CGE in mixed classes (e.g., head-body pairs of neurons have $\tilde{r}_\phi = 0.01$).
Given this variation, we tested whether the increased CGE between hub neurons could be explained by their anatomical distribution using the permutation testing procedure described above for neurochemistry.
That is, we compared the distribution of CGE between hubs to a null distribution formed from $10^8$ random permutations of thirteen head neurons and three tail neurons.
The median CGE, $\tilde{r}_\phi$, between hub neurons is significantly increased relative to random sets of thirteen head neurons and three tail neurons (permutation test, $p = 8\times10^{-8}$).
Furthermore, CGE is significantly increased amongst hub neurons relative to other pairs of head neurons (Wilcoxon rank sum test, $p = 4.1 \times 10^{-11}$ for 46 hub-hub pairs and 1\,186 others),
and also amongst head/tail pairs of neurons (Wilcoxon rank sum test, $p = 1.6 \times 10^{-14}$ for 23 hub-hub pairs, 174 others),
but not for the three hub neurons in the tail (Wilcoxon rank sum test, $p = 0.15$ for three hub-hub pairs, 53 others).
Thus, anatomical location does not explain the high CGE of hub neurons.

\paragraph{Functional class.}
\emph{C. elegans} neurons can be divided into distinct groups that each perform a specialized behavioral function \cite{Hobert2003}.
One of the best-characterised functional classes that is particularly relevant for our analysis is the set of `command interneurons' which are a functional group of ten neurons that govern forward (AVBL, AVBR, PVCL, PVCR) and backward (AVAL, AVAR, AVDL, AVDR, AVEL, AVER) locomotion \cite{Rakowski2013}.
All of these neurons are hubs.
Given the overlap between hub neurons and command interneurons, we investigated whether command interneurons exhibit more similar expression than other hub interneurons, and may therefore drive the increase in CGE amongst hubs as a whole.
We compared CGE, $r_\phi$, between all pairs of command interneurons (ten neurons), and between all pairs of hubs that are not command interneurons (six neurons), shown in Fig.~\ref{fig:Fig8}E.
Correlated gene expression between command interneurons is significantly greater than between other hub neurons (Wilcoxon rank sum test, $p = 3.3 \times 10^{-8}$), indicating that command interneurons play a major role in driving the increased CGE amongst hub neurons.
Moreover, there is no difference in CGE between pairs of hubs that are not command interneurons (DVA, RIBL, RIBR, AIBR, RIGL, AVKL) and a set of seven anatomically matched nonhub head interneurons (AVFL, AVFR, AVHL, AVHR, AVJL, AVJR, AVKR) (Wilcoxon rank sum test, $p = 0.13$), indicating that the status of many hub neurons as command interneurons makes a significant contribution to the elevated CGE of hubs.

\subsection*{Genes driving correlated gene expression patterns}

Having characterized a robust relationship between CGE and (i) connectivity, and (ii) hub connectivity, we next investigated which specific genes contribute most to this relationship.
Despite challenges with the incomplete binary expression measurements in a small proportion of the genome, we developed a method to score genes according to their contribution to a given CGE pattern (see \emph{Methods}).
We characterized individual high-scoring genes, with $p_\mathrm{corr} < 10^{-4}$ (approximately 20\% of genes with the highest scores in each analysis), and attempted to summarize functional groups of genes as biological process categories of the gene ontology (GO) that were enriched in high scoring genes using overrepresentation analysis (ORA) \cite{Ashburner2000, Gillis2010}.

We first investigated which genes drive increased CGE in connected pairs of neurons relative to unconnected pairs.
Previous studies in mouse have indicated that genes driving an increase in CGE between connected pairs of brain regions are enriched in GO categories related to neuronal connectivity and communication \cite{Fulcher:2016ck, Ji:2014jw, Fakhry:2015kl, French2011}.
First, we manually investigated individual high-scoring genes (i.e., those with $p_\mathrm{corr} < 10^{-4}$, see Supplementary Data File (\nameref{file:geneList})).
Given that glutamate is a prevalent neurotransmitter in \textit{C. elegans} (26\% of neurons with known neurotransmitter type are glutamatergic \cite{Pereira:2015er}), it is appropriate that many high scoring genes are related to glutamate receptors (including \emph{glr-1}, \emph{glr-2}, \emph{glr-4}, \emph{glr-5}, \emph{nmr-1}, and \emph{nmr-2}).
Consistent with the importance of innexins in forming electrical synapses \cite{Starich2001}, our list contained the following innexin genes: \emph{unc-9}, \emph{unc-7}, \emph{inx-7}, \emph{inx-19}, \emph{inx-13}.
Genes encoding cell adhesion molecules related to axon outgrowth and guidance, cell migration and locomotion (\emph{sax-3}, \emph{cam-1}, \emph{unc-6}, \emph{rig-1}, \emph{unc-5}), learning (\emph{casy-1}) \cite{Zallen1999, Garriga1999, Leung-Hagesteijn1992, Harris:2009kd, Ikeda2008},
as well as genes involved in determining cell polarity (\emph{vang-1}, \emph{prkl-1}) \cite{Wu2006, Hoffmann2010} were also amongst the top scoring genes for connectivity.
These genes have been implicated in neuronal connectivity in both flies and humans \cite{Paemka2013, Ehaideb2016, Sowers2013}, with our results predicting that they may play a similar role in \textit{C. elegans}.
In addition, transcription factors regulating neuronal development,
fate specification (\emph{lin-11}, \emph{unc-3}, \emph{unc-42}, \emph{ceh-14}, \emph{ast-1}, \emph{cfi-1}) \cite{Sarafi-Reinach2001, Prasad2008, Baran1999, Cassata2000, Schmid2006, Shaham2002a},
and locomotion (\emph{unc-3}) \cite{Prasad2008}) were also implicated in driving increased CGE amongst connected pairs of neurons.
Both adhesion molecules and transcription factors are candidates for facilitating signal transduction and communication.

In order to summarize the above mentioned results and determine if any particular functional groups of genes are dominating in driving this effect, we performed an enrichment analysis.
Top scoring biological process GO categories from ORA analysis (of 85 genes relative to the 414 genes with sufficient data for this analysis) are listed in \nameref{tab:enrichmentCON}.
Although no GO categories are significant at a false discovery rate of 0.05, the top categories are consistent with a connectivity profile, including `glutamate receptor signaling', `cell surface receptor signaling', and `ion transport', with other categories involved in regulation of growth rate and several related to catabolic processes.
Thus, despite incomplete gene expression data that do not provide sufficient coverage to detect statistically significant effects, these results indicate that our data-driven gene scoring method is able to yield sensible, biologically relevant insights into the genetic basis of neuronal connectivity in \textit{C. elegans} connectome.

Having characterized genes that contribute to the increase in CGE between connected pairs of neurons, we next investigated whether particular functional groups of genes drive differences in CGE between connections involving hub neurons (i.e., in rich, feed-in, and feed-out connections) relative to connections between pairs of nonhub neurons (i.e., peripheral connections).
In order to investigate which specific genes contribute most to the increase in CGE for connections involving hubs, we first investigated the highest-scoring genes, with $p_\mathrm{corr} < 10^{-4}$ (corresponding to approximately the top 20\% of genes in the analysis).
In addition to glutamate receptor genes (\emph{glr-5}, \emph{nmr-1}, \emph{nmr-2}, \emph{glr-1}, \emph{glr-2}, \emph{grld-1})
and acetylcholine related genes (\emph{ace-2}, \emph{cho-1}, \emph{unc-17}, \emph{deg-3}),
we again find a high number of genes regulating cell adhesion (\emph{cam-1}, \emph{rig-1}, \emph{rig-6}, \emph{unc-6}, \emph{grld-1}, \emph{dbl-1}, \emph{ncam-1})
and relevant transcription factors (\emph{unc-3}, \emph{unc-42}, \emph{ast-1}).
The implication of glutamate and acetylcholine may be attributable to the importance of glutamate in the regulation of locomotion in command interneurons \cite{Choi2015, Zheng1999}, with acetylcholine being the dominant neurotransmitter in hubs (13 out of 16 hubs are cholinergic).
We also find a high overlap between adhesion molecule and transcription factor genes found in the previous analysis and the implication of human orthologs (\emph{rig-1}, \emph{ncam-1}, \emph{grld-1}, corresponding to human genes ROBO4, NCAM2, and RBM15 respectively  \cite{Harris:2009kd}) for genes regulating cell migration, differentiation and neuron cell adhesion.


While previous work implicated genes regulating oxidative metabolism for connections involving hubs in mouse \cite{Fulcher:2016ck}, and for hub regions in human \cite{Vertes2016a}, the gene expression dataset used here was not sufficiently comprehensive to investigate these processes.
For example, only one of the 948 genes annotated to the GO categories related to hub connectivity in mouse is present in our gene expression dataset (\emph{unc-32} is annotated to the GO category: `ATP hydrolysis coupled proton transport').
Thus, although a direct test of the metabolic hypothesis for neural hubs is not possible from current data, we investigated whether other biological process GO categories were overrepresented in pairs of connected hubs using ORA (of 30 genes relative to the 168 genes with sufficient data for this analysis), with results listed in \nameref{tab:enrichmentRICH}.
Even though no categories are statistically significant at a false discovery rate of 0.05, the list of top categories includes both `glutamate receptor signaling pathway' as well as more general `cell surface receptor signaling pathway' in addition to several ion transport related gene groups ('ion transport', `ion transmembrane transport', `transmembrane transport').
Other top-ranked GO categories include regulation of locomotion, and various metabolism and biosynthesis related processes.
Our gene scoring method again yields interpretable insights into the types of genes that contribute to differences in CGE between different classes of neuronal connections in \emph{C. elegans}.
While current data are limited, more comprehensive expression annotations in the future would allow more systematic and statistically powered inferences across GO categories.

\section*{Discussion}
Highly connected hubs of neural systems play an important role in brain function, with their dense rich-club interconnectivity integrating disparate neural networks \cite{vandenHeuvel:2013ge, Fornito2015, deReus:2013cy, vandenHeuvel:2013ij}.
Here, our analysis linking hub connectivity of the microscale connectome of \emph{C. elegans} to patterns of neuron-specific gene expression has identified a transcriptional signature that appears to be highly conserved, given recent findings reported in a mesoscale investigation of the mouse \cite{Fulcher:2016ck} and a macroscale study of humans \cite{Vertes2016a}.
Specifically, we show that:
(i) CGE is higher for connected pairs of neurons compared to unconnnected pairs;
(ii) the neuron connection probability decays as a function of spatial separation, and;
(iii) connected pairs of hub neurons, which are generally separated by longer anatomical distances, show the highest levels of CGE.
This association between CGE and hub connectivity followed a gradient, such that CGE was lowest for connected nonhubs, intermediate for hub-nonhub pairs, and highest for connected hubs, consistent with results reported in the mouse brain \cite{Fulcher:2016ck}.
Amongst the genes considered here, many of those with the greatest contribution to connectivity are biologically plausible genes related to receptors, neurotransmitters, and cell adhesion, and those with the greatest contribution to hub connectivity are related to glutamate receptors, acetylcholine signaling, and other neuronal communication related genes.
The methods we develop here for quantifying CGE, and for scoring the contribution of individuals genes to overall CGE, yield biologically interpretable results from incomplete binary gene expression data.
With improvements in gene annotation quality and specificity, and increases in genome coverage, similar methods could be used in future work to characterize the biological basis of a range of neuronal connectivity patterns.\\

The availability of spatial maps of gene expression with genome-wide coverage has allowed the relationship between gene expression and connectivity to be investigated in species ranging from \textit{C. elegans} through to mouse and human.
For example, Krienen et al. \cite{Krienen:2016eq} showed that the topography of transcriptional expression of a small number of human supragranular enriched genes mirrors the large-scale brain network organization of rs-fMRI in the healthy human brain, and Romme et al. \cite{Romme2016} showed that schizophrenia-related structural disconnectivity is significantly correlated to the expression profiles of schizophrenia risk genes.
Recent work has shown a relationship between spatial gene expression maps and cortical hierarchy using structural MRI imaging in macaque and human \cite{Burt2017}.
Spatial maps of gene transcription will continue to play a key role in uncovering species-conserved mechanisms underlying brain connectivity.

Computational methods to extract relationships between network organization and gene expression can help understand the molecular processes underlying neuronal connectivity.
Previous research has related gene expression data in \textit{C. elegans} to axonal connectivity patterns, focusing on pairwise relationships of genes that might underpin axonal connectivity.
Both Kaufman et al. \cite{Kaufman2006} and Baruch et al. \cite{Baruch2008} developed statistical models to predict the postsynaptic partners of individual neurons in \textit{C. elegans} (using $k$-nearest neighbors and boosted decision tree models, respectively).
This research found that the targets of some neurons are easier to predict than others \cite{Kaufman2006}, and that the prediction can be done with good accuracy using only a small subset of genes \cite{Baruch2008}.
Our results demonstrate differences in CGE across different topological classes of connections, and highlight genes that make the biggest contribution to these differences.
More detailed investigations of these relationships (e.g., across \textit{C. elegans} development) may shed light on the molecular logic underlying the establishment and maintenance of neuronal connectivity.

It is reasonable to expect that the principles of neural organization may differ from the scale of individual neurons to the scale of macroscopic brain regions (in which each brain region contains millions of neurons).
However, many of our results in \emph{C. elegans} suggest a striking conservation of many fundamental spatial trends in neural connectivity and CGE across scales and species.
For example, connection probability decreases with spatial separation between brain areas in rodents and primates \cite{Horvat:2016ia, Wang:2016gg} (including in macaque \cite{Markov:2013jo}, human \cite{Henderson:2014fg}, mouse \cite{Fulcher:2016ck}, and rat \cite{Noori:2017ce}),
for individual neurons in mouse primary auditory cortex \cite{Levy:2012dy},
and between neurons in \emph{C. elegans} (cf. Fig.~S1 of \cite{Azulay:2016cg}).
Unlike mammalian brains, where all neurons are confined to a spatially contiguous organ, neurons are distributed across nearly the entire length of \emph{C. elegans}, including a dense cluster of neurons in the head and in the tail.
Despite these distinct morphologies, we report a qualitatively similar spatial dependence of connection probability with separation distance for many classes of connections in \emph{C. elegans}, including those within the head, body, and tail, indicating that this distance-dependence may be a generic property of evolved neuronal systems that must balance the energetic cost of long-range connections with their functional benefit \cite{Bullmore:2012vl, vandenHeuvel:2012kh, Kim:2014bu, Betzel:2016jt}.
Less frequently characterized is the spatial dependence of CGE, with available evidence indicating that more proximal brain areas exhibit more similar gene expression patterns than more distant brain areas in the mouse brain \cite{Fulcher:2016ck} and human cortex \cite{Krienen:2016eq, Pantazatos:2016ir, Richiardi:2017hb}.
Some of the spatial trends in CGE found in the 948 genes analyzed here mirror these trends of bulk regions of macroscopic mammalian brains.
It is therefore possible that these spatial dependences of connectivity and CGE may not be simply due to bulk spatial trends in macroscopic brains containing millions of neurons, but may reflect conserved organizational principles that hold across species and spatial scales.
Our results highlight the importance of treating nervous systems as spatially embedded objects, as many seemingly non-trivial properties of brain organization may be well approximated by simple, isotropic spatial rules \cite{Henderson:2014fg, Roberts2016, Horvat:2016ia, Bassett2010, Chen:2006ie} (see also \cite{Bullmore:2012vl, Betzel:2016jt}).

Our analysis indicates that CGE patterns in \emph{C. elegans} show many surprising similarities to previous work in the mesoscale mouse connectome \cite{Fulcher:2016ck}, despite:
(i) involving different gene expression annotation data (comprehensive \emph{in situ} hybridization expression data across $\sim20\,000$ genes in mouse versus literature-curated annotations across $\sim 1000$ genes in \emph{C. elegans}),
(ii) being a different type of neural system (from the spatially continuous macroscopic brain of mouse, to the spatially separated nervous system of \emph{C. elegans});
(iii) orders of magnitude differences in spatial scale.
The findings were also robust to a range of data processing choices, including different representations of the connectome (e.g., directed/undirected, or excluding electrical synapses), and across alternative metrics for quantifying transcriptional similarity.

What could drive this highly conserved association between CGE and hub connectivity?
Here, we took advantage of the rich and diverse information available for each neuron of the \emph{C. elegans} connectome to begin to address this question.
We show that CGE between hub neurons is not determined by their neuronal type (i.e., the fact that all hubs are interneurons rather than sensory or motor neurons), since CGE between hub neurons is higher than between other pairs of interneurons.
The effect cannot be attributed to the modular organization of the network either, since CGE between hubs in the same module is higher than between other pairs of neurons in the same topological module, with a similar increase in CGE for pairs of hubs in different modules.
We also show that the effect is not driven by similarities in the birth time nor lineage distance of hub neurons, which exhibit higher CGE than other early-born neurons (prior to hatching) and are not closer in their lineage.
Moreover, the abundance of cholinergic signaling of hub neurons cannot explain the effect.
Rather, the CGE between pairs of hub neurons in \textit{C. elegans} may be related to the specific functional role of these cells.
Namely, 60\% of them are command interneurons, which play a vital role in coordinating forward and backward locomotion in \textit{C. elegans} \cite{Kim:2016gl}.
The overlap between command interneurons and hub neurons has interesting parallels with the human cortex, where polymodal association areas tend to be the most highly connected network elements \cite{vandenHeuvel:2016eg}.
Association areas sit at atop the cortical hierarchy and support complex behaviors by integrating information from diverse neural systems \cite{Mesulam1998}.
Locomotion is arguably one of the most complex behaviors expressed by \emph{C. elegans}.
Thus, the association between hub status and command interneurons may reflect the specialization of these neurons for supporting higher-order functions in the behavioral repertoire of \emph{C. elegans}.

It is as yet unclear whether CGE between network hubs, regardless of species and scale, is simply a byproduct of tightly coupled hub activity, or some shared morphological or development characteristic between hubs that we have not captured in the present analysis.
More comprehensive transcriptomic data (e.g., obtained through systematic single-neuron RNA sequencing), measured through development and coupled with measures of neuronal activity, would allow us to address these questions.
Additionally, we cannot rule out the possibility that gene annotations have been influenced by the nature of the curated data that we have used here.
Given their functional similarity, command interneurons might have been tested as a group in a set of experiments for the expression of particular genes and consequently assigned similar expression signatures.
More precise and systematic measurement of neuron-specific gene expression patterns would be required to address this question.
Studies of gene expression often assume that expression levels correspond to protein abundance, but this assumption does not always hold \cite{Futcher1999, Greenbaum2003, Gygi1999}.
Thus, analyses of transcriptomic data can be viewed as a relatively efficient approach for investigating potential links between molecular function and nervous system organization, that can be more strongly verified using subsequent proteomic analysis.

In this work we developed methods to relate correlations in binary gene expression data to pairwise connectivity and subsequently score and evaluate the contribution of individual genes to these patterns.
Compared to continuous \emph{in situ} hybridization measurements of the expression of $>17\,000$ genes in the mouse brain \cite{Lein:2007jn}, or microarray measurements of $>20\,000$ genes in the human brain \cite{Hawrylycz:2012ky, Shen:2012ua}, which permit more detailed analysis \cite{Fulcher:2016ck, Ji:2014jw, Fakhry:2015kl, French2011, Vertes2016a, Parkes:2017dn}, working with \emph{C. elegans} gene expression data is challenging due to its low coverage ($<5$\% coverage of the worm genome), binary indications of expression, and incompleteness (an inability to distinguish missing data from lack of expression).
Moreover, the data have different qualifiers related to the certainty of gene expression annotations (see \nameref{text:expressionAnnotations}), requiring choices to be made to appropriately balance sensitivity and specificity.
Although gene enrichment analyses did not have enough power to detect significant effects here, top GO categories point us towards biologically relevant categories related to neuronal connectivity, neurotransmitters, and metabolism.
We note, however, that the incomplete coverage of the genome in our annotated dataset may mask many true GO associations.
Our single gene analysis identified specific genes contributing to increases in CGE for connected pairs of neurons and for connections involving hub neurons.
In line with our expectations, genes regulating both chemical and electrical signaling, namely glutamate receptor and innexin genes, were implicated in general connectivity.
In addition, we also find multiple cell adhesion molecule genes and transcription factors that regulate neuronal development and fate specification -- both groups are important for forming neuronal connections.
High overlap between adhesion molecule genes and transcription factors implicated in regulating both general and hub connectivity highlights that related mechanisms might be used in both cases.
While we were not able to test GO categories related to neurotransmitter signaling comprehensively, due to insufficient coverage of gene expression annotations, single gene analysis revealed the importance of acetylcholine genes, which may be related to the fact that acetylcholine is the dominant neurotransmitter in hub neurons.

\section*{Acknowledgments}
We thank WormBase and particularly Daniela Raciti for providing valuable information regarding the curation of \emph{C. elegans} gene expression data and Taci Ali for preliminary data processing. 

\nolinenumbers

% Either type in your references using
% \begin{thebibliography}{}
% \bibitem{}
% Text
% \end{thebibliography}
%
% or
%
% Compile your BiBTeX database using our plos2015.bst
% style file and paste the contents of your .bbl file
% here. See http://journals.plos.org/plosone/s/latex for
% step-by-step instructions.
%
\begin{thebibliography}{100}

\bibitem{vandenHeuvel:2016eg}
van~den Heuvel MP, Bullmore ET, Sporns O.
\newblock {Comparative Connectomics}.
\newblock Trends Cogn Sci. 2016;20(5):345--361.
\newblock doi:{10.1016/j.tics.2016.03.001}.


\bibitem{White:1986tx}
White JG, Southgate E, Thomson JN.
\newblock {The structure of the nervous system of the nematode Caenorhabditis
  elegans}.
\newblock Philos Trans R Soc Lond B Biol Sci. 1986;314(1165):1--340.

\bibitem{Varshney2011}
Varshney LR, Chen BL, Paniagua E, Hall DH, Chklovskii DB.
\newblock {Structural properties of the Caenorhabditis elegans neuronal
  network}.
\newblock PLoS Comp Biol. 2011;7(2):e1001066.
\newblock doi:{10.1371/journal.pcbi.1001066}.

\bibitem{Chiang:2011}
Chiang AS, Lin CY, Chuang CC, Chang HM, Hsieh CH, Yeh CW, et~al.
\newblock {Three-dimensional reconstruction of brain-wide wiring networks in
  {Drosophila} at single-cell resolution}.
\newblock Curr Biol. 2011;21(1):1--11.

\bibitem{Shih2015}
Shih CT, Sporns O, Yuan SL, Su TS, Lin YJ, Chuang CC, et~al.
\newblock {Connectomics-Based Analysis of Information Flow in the Drosophila
  Brain}.
\newblock Curr Biol. 2015;25(10):1249--1258.
\newblock doi:{10.1016/j.cub.2015.03.021}.

\bibitem{Wanner:2016ea}
Wanner AA, Genoud C, Masudi T, Siksou L, Friedrich RW.
\newblock {Dense EM-based reconstruction of the interglomerular projectome in
  the zebrafish olfactory bulb}.
\newblock Nat Neurosci. 2016;19(6):816--825.

\bibitem{Hildebrand:2017iu}
Hildebrand DGC, Cicconet M, Torres RM, Choi W, Quan TM, Moon J, et~al.
\newblock {Whole-brain serial-section electron microscopy in larval zebrafish}.
\newblock Nature. 2017;545(7654):345--349.

\bibitem{Oh2014}
Oh SW, Harris JA, Ng L, Winslow B, Cain N, Mihalas S, et~al.
\newblock {A mesoscale connectome of the mouse brain}.
\newblock Nature. 2014;508(7495):207--214.
\newblock doi:{10.1038/nature13186}.

\bibitem{Zingg:2014el}
Zingg B, Hintiryan H, Gou L, Song MY, Bay M, Bienkowski MS, et~al.
\newblock {Neural Networks of the Mouse Neocortex}.
\newblock Cell. 2014;156(5):1096--1111.

\bibitem{bota2015}
Bota M, Sporns O, Swanson LW.
\newblock {Architecture of the cerebral cortical association connectome
  underlying cognition}.
\newblock Proc Natl Acad Sci USA. 2015;112(16):E2093--E2101.

\bibitem{scannell1995}
Scannell JW, Blakemore C, Young MP.
\newblock {Analysis of connectivity in the cat cerebral cortex}.
\newblock J Neurosci. 1995;15(2):1463--1483.

\bibitem{Markov:2012wu}
Markov NT, Ercsey-Ravasz MM, Gomes ARR, Lamy C, Magrou L, Vezoli J, et~al.
\newblock {A Weighted and Directed Interareal Connectivity Matrix for Macaque
  Cerebral Cortex}.
\newblock Cereb Cortex. 2012;24(1):17--36.

\bibitem{stephan2001}
Stephan KE, Kamper L, Bozkurt A, Burns GA, Young MP, K{\"o}tter R.
\newblock {Advanced database methodology for the Collation of Connectivity data
  on the Macaque brain (CoCoMac)}.
\newblock Philos Trans Roy Soc B: Biol Sci. 2001;356(1412):1159--1186.

\bibitem{Hagmann:2008gda}
Hagmann P, Cammoun L, Gigandet X, Meuli R, Honey CJ, Wedeen VJ, et~al.
\newblock {Mapping the Structural Core of Human Cerebral Cortex}.
\newblock PLoS Biol. 2008;6(7):e159.

\bibitem{VanEssen2013}
{Van Essen} DC, Smith SM, Barch DM, Behrens TEJ, Yacoub E, Ugurbil K.
\newblock {The WU-Minn Human Connectome Project: An overview}.
\newblock NeuroImage. 2013;80:62--79.
\newblock doi:{10.1016/j.neuroimage.2013.05.041}.

\bibitem{Bullmore:2009iv}
Bullmore ET, Sporns O.
\newblock {Complex brain networks: graph theoretical analysis of structural and
  functional systems}.
\newblock Nat Rev Neurosci. 2009;10(3):186--198.

\bibitem{Bullmore:2012vl}
Bullmore ET, Sporns O.
\newblock {The economy of brain network organization}.
\newblock Nat Rev Neurosci. 2012;13(5):337--349.

\bibitem{Sporns2011}
Sporns O.
\newblock {The non-random brain: efficiency, economy, and complex dynamics.}
\newblock Front Comput Neurosci. 2011;5:5.
\newblock doi:{10.3389/fncom.2011.00005}.

\bibitem{Schroter:2017eo}
Schr{\"o}ter M, Paulsen O, Bullmore E.
\newblock {Micro-connectomics: probing the organization of neuronal networks at
  the cellular scale}.
\newblock Nat Rev Neurosci. 2017;18(3):131--146.

\bibitem{fornito2016book}
Fornito A, Zalesky A, Bullmore E.
\newblock {Fundamentals of brain network analysis}.
\newblock Academic Press; 2016.

\bibitem{Meunier:2010hq}
Meunier D, Lambiotte R, Bullmore ET.
\newblock {Modular and Hierarchically Modular Organization of Brain Networks}.
\newblock Front Psychiat. 2010;4:200.

\bibitem{Bassett2010}
Bassett DS, Greenfield DL, Meyer-Lindenberg A, Weinberger DR, Moore SW,
  Bullmore ET.
\newblock {Efficient Physical Embedding of Topologically Complex Information
  Processing Networks in Brains and Computer Circuits}.
\newblock PLoS Comp Biol. 2010;6(4):e1000748.
\newblock doi:{10.1371/journal.pcbi.1000748}.

\bibitem{Betzel:2016jt}
Betzel RF, Avena-Koenigsberger A, Go{\~n}i J, He Y, de~Reus MA, Griffa A,
  et~al.
\newblock {Generative models of the human connectome}.
\newblock NeuroImage. 2016;124(Pt A):1054--1064.

\bibitem{vandenHeuvel:2013ge}
van~den Heuvel MP, Sporns O.
\newblock {Network hubs in the human brain}.
\newblock Trends Cogn Sci. 2013;17(12):683--696.

\bibitem{vandenHeuvel:2011he}
van~den Heuvel MP, Sporns O.
\newblock {Rich-Club Organization of the Human Connectome}.
\newblock J Neurosci. 2011;31(44):15775--15786.

\bibitem{ZamoraLopez:2010hy}
Zamora-L{\'o}pez G, Zhou C, Kurths J.
\newblock {Cortical Hubs Form a Module for Multisensory Integration on Top of
  the Hierarchy of Cortical Networks}.
\newblock Front Neuroinf. 2010;4:1.

\bibitem{deReus:2013cy}
de~Reus MA, van~den Heuvel MP.
\newblock {Rich Club Organization and Intermodule Communication in the Cat
  Connectome}.
\newblock J Neurosci. 2013;33(32):12929--12939.

\bibitem{Towlson2013}
Towlson EK, V{\'{e}}rtes PE, Ahnert SE, Schafer WR, Bullmore ET.
\newblock {The rich club of the C. elegans neuronal connectome}.
\newblock J Neurosci. 2013;33(15):6380--6387.
\newblock doi:{10.1523/JNEUROSCI.3784-12.2013}.

\bibitem{Fornito2015}
Fornito A, Zalesky A, Breakspear M.
\newblock {The connectomics of brain disorders}.
\newblock Nat Rev Neurosci. 2015;16(3):159--172.
\newblock doi:{10.1038/nrn3901}.

\bibitem{vandenHeuvel:2013ij}
van~den Heuvel MP, Sporns O.
\newblock {An Anatomical Substrate for Integration among Functional Networks in
  Human Cortex}.
\newblock J Neurosci. 2013;33(36):14489--14500.

\bibitem{Crossley2014}
Crossley NA, Mechelli A, Scott J, Carletti F, Fox PT, McGuire P, et~al.
\newblock {The hubs of the human connectome are generally implicated in the
  anatomy of brain disorders}.
\newblock Brain. 2014;137(Pt 8):2382--2395.
\newblock doi:{10.1093/brain/awu132}.

\bibitem{Crossley:2013kl}
Crossley NA, Mechelli A, V{\'e}rtes PE, Winton-Brown TT, Patel AX, Ginestet CE,
  et~al.
\newblock {Cognitive relevance of the community structure of the human brain
  functional coactivation network}.
\newblock Proc Natl Acad Sci USA. 2013;110(28):11583--11588.

\bibitem{vandenHeuvel:2012kh}
van~den Heuvel MP, Kahn RS, Go{\~n}i J, Sporns O.
\newblock {High-cost, high-capacity backbone for global brain communication}.
\newblock Proc Natl Acad Sci USA. 2012;109(28):11372--11377.

\bibitem{Harriger2012}
Harriger L, van~den Heuvel MP, Sporns O.
\newblock {Rich Club Organization of Macaque Cerebral Cortex and Its Role in
  Network Communication}.
\newblock PLoS ONE. 2012;7(9):e46497.
\newblock doi:{10.1371/journal.pone.0046497}.

\bibitem{Misic:2014it}
Mi{\v s}i{\'c} B, Sporns O, McIntosh AR.
\newblock {Communication Efficiency and Congestion of Signal Traffic in
  Large-Scale Brain Networks}.
\newblock PLoS Comp Biol. 2014;10(1):e1003427.

\bibitem{Misic:2015jw}
Mi{\v s}i{\'c} B, Betzel RF, Nematzadeh A, Go{\~n}i J, Griffa A, Hagmann P,
  et~al.
\newblock {Cooperative and Competitive Spreading Dynamics on the Human
  Connectome}.
\newblock Neuron. 2015;86(6):1518--1529.

\bibitem{VandenHeuvel2016b}
van~den Heuvel MP, Scholtens LH, de~Reus MA.
\newblock {Topological organization of connectivity strength in the rat
  connectome}.
\newblock Brain Struct Funct. 2016;221(3):1719--1736.
\newblock doi:{10.1007/s00429-015-0999-6}.

\bibitem{Fulcher:2016ck}
Fulcher BD, Fornito A.
\newblock {A transcriptional signature of hub connectivity in the mouse
  connectome}.
\newblock Proc Natl Acad Sci USA. 2016;113(5):1435--1440.

\bibitem{Tomasi:2013kl}
Tomasi D, Wang GJ, Volkow ND.
\newblock {Energetic cost of brain functional connectivity}.
\newblock Proc Natl Acad Sci USA. 2013;110(33):13642--13647.

\bibitem{Collin:2014kq}
Collin G, Sporns O, Mandl RCW, van~den Heuvel MP.
\newblock {Structural and Functional Aspects Relating to Cost and Benefit of
  Rich Club Organization in the Human Cerebral Cortex}.
\newblock Cereb Cortex. 2014;24(9):2258--2267.

\bibitem{Liang2013a}
Liang X, Zou Q, He Y, Yang Y.
\newblock {Coupling of functional connectivity and regional cerebral blood flow
  reveals a physiological basis for network hubs of the human brain}.
\newblock Proc Natl Acad Sci USA. 2013;110(5):1929--1934.
\newblock doi:{10.1073/pnas.1214900110}.

\bibitem{Lennie:2003ia}
Lennie P.
\newblock {The Cost of Cortical Computation}.
\newblock Curr Biol. 2003;13(6):493--497.

\bibitem{Laughlin:2003vu}
Laughlin SB, Sejnowski TJ.
\newblock {Communication in neuronal networks}.
\newblock Science. 2003;301(5641):1870-–1874.

\bibitem{Vertes2016a}
V{\'{e}}rtes PE, Rittman T, Whitaker KJ, Romero-Garcia R, V{\'{a}}{\v{s}}a F,
  Kitzbichler MG, et~al.
\newblock {Gene transcription profiles associated with inter-modular hubs and
  connection distance in human functional magnetic resonance imaging networks}.
\newblock Phil Trans Roy Soc B. 2016;371(1705):735--769.
\newblock doi:{10.1098/rstb.2015.0362}.

\bibitem{Riddle1997}
Riddle DL, Blumenthal T, Meyer BJ, et al., editors.
\newblock {C. elegans II. 2nd edition}.
\newblock Cold Spring Harbor (NY): Cold Spring Harbor Laboratory Press; 1997.
\newblock Available from: \url{http://www.ncbi.nlm.nih.gov/books/NBK19997/}.


\bibitem{Chen:2006ie}
Chen BL, Hall DH, Chklovskii DB.
\newblock {Wiring optimization can relate neuronal structure and function}.
\newblock Proc Natl Acad Sci USA. 2006;103(12):4723--4728.

\bibitem{WormAtlas}
Altun ZF, Herndon LA, Wolkow CA, Crocker C, Lints R, Hall DH. WormAtlas;
2002--2017.
\newblock Available from: \url{http://www.wormatlas.org}.

\bibitem{Kashtan:2004ev}
Kashtan N, Itzkovitz S, Milo R, Alon U.
\newblock {Topological generalizations of network motifs}.
\newblock Phys Rev E. 2004;70(3 Pt 1):031909.

\bibitem{Azulay:2016cg}
Azulay A, Itskovits E, Zaslaver A.
\newblock {The C. elegans Connectome Consists of Homogenous Circuits with
  Defined Functional Roles}.
\newblock PLoS Comp Biol. 2016;12(9):e1005021.

\bibitem{Kim:2016gl}
Kim S, Kim H, Kralik JD, Jeong J.
\newblock {Vulnerability-Based Critical Neurons, Synapses, and Pathways in the
  \emph{Caenorhabditis elegans} Connectome}.
\newblock PLoS Comp Biol. 2016;12(8):e1005084.

\bibitem{Kim:2014bu}
Kim JS, Kaiser M.
\newblock {From Caenorhabditis elegans to the human connectome: a specific
  modular organization increases metabolic, functional and developmental
  efficiency}.
\newblock Phil Trans Roy Soc B. 2014;369(1653):20130529.

\bibitem{Pavlovic:2014gx}
Pavlovic DM, V{\'e}rtes PE, Bullmore ET, Schafer WR, Nichols TE.
\newblock {Stochastic Blockmodeling of the Modules and Core of the
  {Caenorhabditis} elegans Connectome}.
\newblock PLoS ONE. 2014;9(7):e97584.

\bibitem{van2017guiding}
Van De~Ville D, Demesmaeker R, Preti MG.
\newblock {Guiding Network Analysis using Graph Slepians: {An} Illustration for
  the {\emph{C. Elegans}} Connectome}.
\newblock arXiv. 2017; p. 1708.04657.

\bibitem{Pan:2010jt}
Pan RK, Chatterjee N, Sinha S.
\newblock {Mesoscopic Organization Reveals the Constraints Governing
  {Caenorhabditis} elegans Nervous System}.
\newblock PLoS ONE. 2010;5(2):e9240.

\bibitem{choe2004network}
Choe Y, McCormick B, Koh W.
\newblock {Network connectivity analysis on the temporally augmented C. elegans
  web: A pilot study}.
\newblock In: Soc. Neurosci. Abstr. vol.~30; 2004. p. 921.9.

\bibitem{Varier2011}
Varier S, Kaiser M, Koh W, Meuli R, Honey C.
\newblock {Neural Development Features: Spatio-Temporal Development of the
  Caenorhabditis elegans Neuronal Network}.
\newblock PLoS Comp Biol. 2011;7(1):e1001044.
\newblock doi:{10.1371/journal.pcbi.1001044}.

\bibitem{Harris:2009kd}
Harris TW, Antoshechkin I, Bieri T, Blasiar D, Chan J, Chen WJ, et~al.
\newblock {WormBase: a comprehensive resource for nematode research}.
\newblock Nucleic Acids Res. 2009;(Database issue):D463--D467.

\bibitem{Pereira:2015er}
Pereira L, Kratsios P, Serrano-Saiz E, Sheftel H, Mayo AE, Hall DH, et~al.
\newblock {A cellular and regulatory map of the cholinergic nervous system of
  {C. elegans}}.
\newblock eLife. 2015;4:e12432.

\bibitem{Sulston1977}
Sulston JE, Horvitz HR.
\newblock {Post-embryonic cell lineages of the nematode, Caenorhabditis
  elegans.}
\newblock Dev Biol. 1977;56(1):110--156.

\bibitem{Sulston1983}
Sulston JE, Schierenberg E, White JG, Thomson JN.
\newblock {The embryonic cell lineage of the nematode Caenorhabditis elegans.}
\newblock Dev Biol. 1983;100(1):64--119.

\bibitem{Colizza:2006kz}
Colizza V, Flammini A, Serrano M, Vespignani A.
\newblock {Detecting rich-club ordering in complex networks}.
\newblock Nat Phys. 2006;2(2):110--115.

\bibitem{Maslov:2002hi}
Maslov S, Sneppen K.
\newblock {Specificity and stability in topology of protein networks}.
\newblock Science. 2002;296:910--913.

\bibitem{Rubinov:2010jd}
Rubinov M, Sporns O.
\newblock {Complex network measures of brain connectivity: Uses and
  interpretations}.
\newblock NeuroImage. 2010;52(3):1059--1069.


\bibitem{Achacoso:1992ay}
Achacoso T, Yamamoto W.
\newblock {AY's Neuroanatomy of C. elegans for Computation}.
\newblock CRC Press, Boca Raton, FL; 1992.


\bibitem{Blondel:2008do}
Blondel VD, Guillaume JL, Lambiotte R, Lefebvre E.
\newblock {Fast unfolding of communities in large networks}.
\newblock J Stat Mech. 2008;2008(10):P10008.

\bibitem{Lancichinetti2012}
Lancichinetti A, Fortunato S, Thomson JN, Brenner S, Moon S.
\newblock {Consensus clustering in complex networks}.
\newblock Sci. Rep. 2012;2:336.
\newblock doi:{10.1038/srep00336}.

\bibitem{Warrens2008}
Warrens MJ.
\newblock {On Association Coefficients for 2x2 Tables and Properties That Do
  Not Depend on the Marginal Distributions}.
\newblock Psychometrika. 2008;73(4):777--789.
\newblock doi:{10.1007/s11336-008-9070-3}.

\bibitem{Ashburner2000}
Ashburner M, Ball CA, Blake JA, Botstein D, Butler H, Cherry JM, et~al.
\newblock {Gene Ontology: Tool for The Unification of Biology}.
\newblock Nat. Gen. 2000;25(1):25--29.
\newblock doi:{10.1038/75556}.


\bibitem{Benjamini:1995cd}
Benjamini Y, Hochberg Y.
\newblock {Controlling the False Discovery Rate: A Practical and Powerful
  Approach to Multiple Testing}.
\newblock J Roy Stat Soc B. 1995;57(1):289--300.

\bibitem{Gillis2010}
Gillis J, Mistry M, Pavlidis P.
\newblock {Gene function analysis in complex data sets using ErmineJ}.
\newblock Nat. Protoc. 2010;5(6):1148--1159.
\newblock doi:{10.1038/nprot.2010.78}.

\bibitem{Zoubarev2012}
Zoubarev A, Hamer KM, Keshav KD, {Luke Mccarthy} E, Santos JRC, Van~rossum T,
  et~al.
\newblock {Gemma: A resource for the reuse, sharing and meta-analysis of
  expression profiling data}.
\newblock Bioinformatics. 2012;28(17):2272--2273.
\newblock doi:{10.1093/bioinformatics/bts430}.

\bibitem{Krienen:2016eq}
Krienen FM, Yeo BTT, Ge T, Buckner RL, Sherwood CC.
\newblock {Transcriptional profiles of supragranular-enriched genes associate
  with corticocortical network architecture in the human brain}.
\newblock Proc Natl Acad Sci USA. 2016;113(4):E469--78.

\bibitem{Pantazatos:2016ir}
Pantazatos SP, Li X.
\newblock {Commentary: {BRAIN NETWORKS}. Correlated gene expression supports
  synchronous activity in brain networks. Science 348, 1241-4}.
\newblock Front Neurosci. 2017;11:412.
\newblock doi:{10.3389/fnins.2017.00412}.


\bibitem{Richiardi:2017hb}
Richiardi J, Altmann A, Greicius M.
\newblock {Distance Is Not Everything In Imaging Genomics Of Functional Networks: Reply To A Commentary On Correlated Gene Expression Supports
  Synchronous Activity In Brain Networks}.
\newblock bioRxiv. 2017; p. 132746.

\bibitem{Horvat:2016ia}
Horv{\'a}t S, G{\u{a}}m{\u{a}}nut R, Ercsey-Ravasz M, Magrou L,
  G{\u{a}}m{\u{a}}nut B, Van~Essen DC, et~al.
\newblock {Spatial Embedding and Wiring Cost Constrain the Functional Layout of
  the Cortical Network of Rodents and Primates}.
\newblock PLoS Biol. 2016;14(7):e1002512.

\bibitem{Wang:2016gg}
Wang XJ, Kennedy H.
\newblock {Brain structure and dynamics across scales: in search of rules}.
\newblock Curr Opin Neurobiol. 2016;37:92--98.

\bibitem{Markov:2013jo}
Markov NT, Ercsey-Ravasz M, Van~Essen DC, Knoblauch K, Toroczkai Z, Kennedy H.
\newblock {Cortical High-Density Counterstream Architectures}.
\newblock Science. 2013;342(6158):1238406--1238406.
\newblock doi:{10.1126/science.1238406}.

\bibitem{Henderson:2014fg}
Henderson JA, Robinson PA.
\newblock {Relations Between the Geometry of Cortical Gyrification and
  White-Matter Network Architecture}.
\newblock Brain Conn. 2014;4(2):112--130.

\bibitem{Noori:2017ce}
Noori HR, Sch{\"o}ttler J, Ercsey-Ravasz M, Cosa-Linan A, Varga M, Toroczkai Z,
  et~al.
\newblock {A multiscale cerebral neurochemical connectome of the rat brain}.
\newblock PLoS Biol. 2017;15(7):e2002612.
\newblock doi:{10.1371/journal.pbio.2002612}.

\bibitem{Levy:2012dy}
Levy RB, Reyes AD.
\newblock {Spatial Profile of Excitatory and Inhibitory Synaptic Connectivity in
  Mouse Primary Auditory Cortex}.
\newblock J Neurosci. 2012;32(16):5609--5619.

\bibitem{Goulas:2016hr}
Goulas A, Uylings HBM, Hilgetag CC.
\newblock {Principles of ipsilateral and contralateral cortico-cortical
  connectivity in the mouse}.
\newblock Brain Struct Funct. 2017;222(3):1281--1295.
\newblock doi:{10.1007/s00429-016-1277-y}.

\bibitem{tsalik2003}
Tsalik EL, Hobert O.
\newblock {Functional mapping of neurons that control locomotory behavior in
  Caenorhabditis elegans}.
\newblock J Neurobiol. 2003;56(2):178--197.

\bibitem{Opsahl2008}
Opsahl T, Colizza V, Panzarasa P, Ramasco JJ.
\newblock {Prominence and control: the weighted rich-club effect}.
\newblock Phys Rev Lett. 2008;101(16):168702.
\newblock doi:{10.1103/PhysRevLett.101.168702}.

\bibitem{Sporns2016}
Sporns O, Betzel RF.
\newblock {Modular Brain Networks}.
\newblock Annu Rev Psychol. 2016;67(1):613--640.
\newblock doi:{10.1146/annurev-psych-122414-033634}.

\bibitem{Richiardi2015}
Richiardi J, Altmann A, Milazzo AC, Chang C, Chakravarty MM, Banaschewski T,
  et~al.
\newblock {Correlated gene expression supports synchronous activity in brain networks}.
\newblock Science. 2015;348(6240):1241--1244.
\newblock doi:{10.1126/science.1255905}.

\bibitem{Ciceri2013}
Ciceri G, Dehorter N, Sols I, Huang ZJ, Maravall M, Mar{\'{i}}n O.
\newblock {Lineage-specific laminar organization of cortical GABAergic
  interneurons}.
\newblock Nat Neurosci. 2013;16(9):1199--1210.
\newblock doi:{10.1038/nn.3485}.

\bibitem{Li2012}
Li Y, Lu H, Cheng Pl, Ge S, Xu H, Shi SH, et~al.
\newblock {Clonally related visual cortical neurons show similar stimulus feature selectivity}.
\newblock Nature. 2012;486(7401):118--121.
\newblock doi:{10.1038/nature11110}.

\bibitem{Yu2012}
Yu YC, He S, Chen S, Fu Y, Brown KN, Yao XH, et~al.
\newblock {Preferential electrical coupling regulates neocortical lineage-dependent microcircuit assembly}.
\newblock Nature. 2012;486(7401):113--117.
\newblock doi:{10.1038/nature10958}.

\bibitem{Cui2007}
Cui Q, Yu Z, Purisima EO, Wang E, Li X, Schwarz P, et~al.
\newblock {MicroRNA regulation and interspecific variation of gene expression}.
\newblock Trends Genet. 2007;23(8):372--375.
\newblock doi:{10.1016/j.tig.2007.04.003}.

\bibitem{Hobert2003}
Hobert O.
\newblock {Behavioral Plasticity in C. elegans: Paradigms, Circuits, Genes}.
\newblock J Neurobiol. 2003;54(1):203--223.
\newblock doi:{10.1002/neu.10168}.

\bibitem{Rakowski2013}
Rakowski F, Srinivasan J, Sternberg PW, Karbowski J.
\newblock {Synaptic polarity of the interneuron circuit controlling C. elegans
  locomotion.}
\newblock Front Comput Neurosci. 2013;7:128.
\newblock doi:{10.3389/fncom.2013.00128}.

\bibitem{Ji:2014jw}
Ji S, Fakhry A, Deng H.
\newblock {Integrative analysis of the connectivity and gene expression atlases
  in the mouse brain}.
\newblock NeuroImage. 2014;84:245--253.

\bibitem{Fakhry:2015kl}
Fakhry A, Ji S.
\newblock {High-resolution prediction of mouse brain connectivity using gene expression patterns}.
\newblock Methods. 2015;73:71--78.

\bibitem{French2011}
French L, Pavlidis P.
\newblock {Relationships between gene expression and brain wiring in the adult rodent brain}.
\newblock PLoS Comp Biol. 2011;7(1):e1001049.
\newblock doi:{10.1371/journal.pcbi.1001049}.

\bibitem{Starich2001}
Starich T, Sheehan M, Jadrich J, Shaw J.
\newblock {Innexins in C. elegans}.
\newblock Cell Commun. Adhes.. 2001;8(4-6):311--314.
\newblock doi:{10.3109/15419060109080744}.

\bibitem{Zallen1999}
Zallen JA, Kirch SA, Bargmann CI.
\newblock {Genes required for axon pathfinding and extension in the C. elegans nerve ring}.
\newblock Development. 1999;126(16):3679--3692.

\bibitem{Garriga1999}
Garriga G, Forrester WC, Dell M, Perens E.
\newblock {A C. elegans Ror receptor tyrosine kinase regulates cell motility and asymmetric cell division}.
\newblock Nature. 1999;400(6747):881--885.
\newblock doi:{10.1038/23722}.

\bibitem{Leung-Hagesteijn1992}
Leung-Hagesteijn C, Spence AM, Stern BD, Zhou Y, Su MW, Hedgecock EM, et~al.
\newblock {UNC-5, a transmembrane protein with immunoglobulin and
  thrombospondin type 1 domains, guides cell and pioneer axon migrations in C.
  elegans}.
\newblock Cell. 1992;71(2):289--299.
\newblock doi:{10.1016/0092-8674(92)90357-I}.

\bibitem{Ikeda2008}
Ikeda DD, Duan Y, Matsuki M, Kunitomo H, Hutter H, Hedgecock EM, et~al.
\newblock {CASY-1, an ortholog of calsyntenins/alcadeins, is essential for
  learning in Caenorhabditis elegans}.
\newblock Proc Natl Acad Sci USA. 2008;105(13):5260--5265.
\newblock doi:{10.1073/pnas.0711894105}.

\bibitem{Wu2006}
Wu M, Herman MA.
\newblock {A novel noncanonical Wnt pathway is involved in the regulation of
  the asymmetric B cell division in C. elegans}.
\newblock Dev Biol. 2006;293(2):316--329.
\newblock doi:{10.1016/j.ydbio.2005.12.024}.

\bibitem{Hoffmann2010}
Hoffmann M, Segbert C, Helbig G, Bossinger O.
\newblock {Intestinal tube formation in Caenorhabditis elegans requires vang-1
  and egl-15 signaling}.
\newblock Dev Biol. 2010;339(2):268--279.
\newblock doi:{10.1016/j.ydbio.2009.12.002}.

\bibitem{Paemka2013}
Paemka L, Mahajan VB, Skeie JM, Sowers LP, Ehaideb SN, Gonzalez-Alegre P,
  et~al.
\newblock {PRICKLE1 Interaction with SYNAPSIN I Reveals a Role in Autism
  Spectrum Disorders}.
\newblock PLoS ONE. 2013;8(12):e80737.
\newblock doi:{10.1371/journal.pone.0080737}.

\bibitem{Ehaideb2016}
Ehaideb SN, Wignall EA, Kasuya J, Evans WH, Iyengar A, Koerselman HL, et~al.
\newblock {Mutation of orthologous prickle genes causes a similar epilepsy
  syndrome in flies and humans}.
\newblock Ann Clin Transl Neurol. 2016;3(9):695--707.
\newblock doi:{10.1002/acn3.334}.

\bibitem{Sowers2013}
Sowers LP, Loo L, Wu Y, Campbell E, Ulrich JD, Wu S, et~al.
\newblock {Disruption of the non-canonical Wnt gene PRICKLE2 leads to
  autism-like behaviors with evidence for hippocampal synaptic dysfunction}.
\newblock Mol. Psychiatry. 2013;18(10):1077--1089.
\newblock doi:{10.1038/mp.2013.71}.

\bibitem{Sarafi-Reinach2001}
Sarafi-Reinach TR, Melkman T, Hobert O, Sengupta P.
\newblock {The lin-11 LIM homeobox gene specifies olfactory and chemosensory
  neuron fates in C. elegans.}
\newblock Development. 2001;128(17):3269--3281.

\bibitem{Prasad2008}
Prasad B, Karakuzu O, Reed RR, Cameron S.
\newblock {unc-3-dependent repression of specific motor neuron fates in
  Caenorhabditis elegans}.
\newblock Dev Biol. 2008;323(2):207--215.
\newblock doi:{10.1016/j.ydbio.2008.08.029}.

\bibitem{Baran1999}
Baran R, Aronoff R, Garriga G.
\newblock {The C. elegans homeodomain gene unc-42 regulates chemosensory and
  glutamate receptor expression}.
\newblock Development. 1999;126(10):2241--2251.

\bibitem{Cassata2000}
Cassata G, Kagoshima H, Andachi Y, Kohara Y, D{\"{u}}rrenberger MB, Hall DH,
  et~al.
\newblock {The LIM homeobox gene ceh-14 confers thermosensory function to the
  AFD neurons in Caenorhabditis elegans}.
\newblock Neuron. 2000;25(3):587--597.

\bibitem{Schmid2006}
Schmid C, Schwarz V, Hutter H.
\newblock {AST-1, a novel ETS-box transcription factor, controls axon guidance
  and pharynx development in C. elegans}.
\newblock Dev Biol. 2006;293(2):403--413.
\newblock doi:{10.1016/j.ydbio.2006.02.042}.

\bibitem{Shaham2002a}
Shaham S, Bargmann CI.
\newblock {Control of neuronal subtype identity by the C. elegans ARID protein CFI-1}.
\newblock Genes Dev. 2002;16(8):972--983.
\newblock doi:{10.1101/gad.976002}.

\bibitem{Choi2015}
Choi S, Taylor KP, Chatzigeorgiou M, Hu Z, Schafer WR, Kaplan JM.
\newblock {Sensory Neurons Arouse C. elegans Locomotion via Both Glutamate and
  Neuropeptide Release}.
\newblock PLoS Genet. 2015;11(7):e1005359.
\newblock doi:{10.1371/journal.pgen.1005359}.

\bibitem{Zheng1999}
Zheng Y, Brockie PJ, Mellem JE, Madsen DM, Maricq AV.
\newblock {Neuronal Control of Locomotion in C. elegans Is Modified by a
  Dominant Mutation in the GLR-1 Ionotropic Glutamate Receptor}.
\newblock Neuron. 1999;24(2):347--361.
\newblock doi:{10.1016/S0896-6273(00)80849-1}.

\bibitem{Roberts2016}
Roberts JA, Perry A, Lord AR, Roberts G, Mitchell PB, Smith RE, et~al.
\newblock {The contribution of geometry to the human connectome}.
\newblock NeuroImage. 2016;124(Pt A):379--993.
\newblock doi:{10.1016/j.neuroimage.2015.09.009}.


\bibitem{Mesulam1998}
Mesulam MM.
\newblock {From sensation to cognition}.
\newblock Brain. 1998;121(Pt 6):1013--1052.

\bibitem{Lein:2007jn}
Lein E, Hawrylycz MJ, Ao N, Ayres M, Bensinger A, Bernard A, et~al.
\newblock {Genome-wide atlas of gene expression in the adult mouse brain}.
\newblock Nature. 2007;445(7124):168--176.

\bibitem{Hawrylycz:2012ky}
Hawrylycz MJ, Lein E, Guillozet-Bongaarts AL, Shen EH, Ng L, Miller JA, et~al.
\newblock {An anatomically comprehensive atlas of the adult human brain
  transcriptome}.
\newblock Nature. 2012;489(7416):391--399.

\bibitem{Shen:2012ua}
Shen EH, Overly CC, Jones AR.
\newblock {The Allen Human Brain Atlas: comprehensive gene expression mapping
  of the human brain}.
\newblock Trends Neurosci. 2012;32(12):711--714.

\bibitem{Parkes:2017dn}
Parkes L, Fulcher BD, Y{\"u}cel M, Fornito A.
\newblock {Transcriptional signatures of connectomic subregions of the human
  striatum}.
\newblock Genes, Brain Behav. 2017;16(7):647--663.

\bibitem{Kaufman2006}
Kaufman A, Dror G, Meilijson I, Ruppin E.
\newblock {Gene expression of Caenorhabditis elegans neurons carries
  information on their synaptic connectivity}.
\newblock PLoS Comp Biol. 2006;2(12):e167.
\newblock doi:{10.1371/journal.pcbi.0020167}.

\bibitem{Alstott2014}
Alstott J, Panzarasa P, Rubinov M, Bullmore ET, V{\'{e}}rtes PE.
\newblock {A unifying framework for measuring weighted rich clubs}.
\newblock Sci. Rep. 2014;4:7258.
\newblock doi:{10.1038/srep07258}.

\bibitem{Goel2014}
Goel P, Kuceyeski A, LoCastro, E, Raj A.
\newblock {Spatial patterns of genome-wide expression profiles reflect anatomic and fiber connectivity architecture of healthy human brain}.
\newblock Hum Brain Mapp. 2014;35(8): 4204-4218.
\newblock doi:{10.1002/hbm.22471}.

\bibitem{Mills2017}
Mills BD, Grayson D, Shunmugavel A, Miranda-Dominguez O, Feczko E, Earl E, Neve K, Fair D.
\newblock {Correlated gene expression and anatomical communication support synchronized brain activity in the mouse functional connectome}.
\newblock bioRxiv.
\newblock doi:{10.1101/167304}.

\bibitem{Futcher1999}
Futcher B, Latter GI, Monardo P, McLaughlin CS, Garrels JI.
\newblock {A sampling of the yeast proteome}.
\newblock Mol. Cell. Biol. 1999;19(11):7357-7368.

\bibitem{Greenbaum2003}
Greenbaum D, Colangelo Ch, Williams K, Gerstein M.
\newblock {Comparing protein abundance and mRNA expression levels on a genomic scale}.
\newblock Genome Biol. 2003;4(9):117.

\bibitem{Gygi1999}
Gygi SP, Rochon Y, Franza BR, Aebersold R.
\newblock {Correlation between protein and mRNA abundance in yeast}.
\newblock Mol. Cell. Biol. 1999;19(3):1720-1730.

\bibitem{Varadan2006}
Varadan V, Miller DM, Anastassiou D.
\newblock {Computational inference of the molecular logic for synaptic connectivity in C. elegans}.
\newblock Bioinformatics. 2006;22(14):497-506.
\newblock doi:{10.1093/bioinformatics/btl224}.

\bibitem{Baruch2008}
Baruch L, Itzkovitz S, Golan-Mashiach M, Shapiro E, Segal E.
\newblock {Using Expression Profiles of Caenorhabditis elegans Neurons To Identify Genes That Mediate Synaptic Connectivity}.
\newblock PLoS Comp Biol. 2008;4(7):e1000120.
\newblock doi:{10.1371/journal.pcbi.1000120}.

\bibitem{Romme2016}
Romme IAC. , de Reus MA, Ophoff RA, Kahn RS, van den Heuvel MP.
\newblock {Connectome Disconnectivity and Cortical Gene Expression in Patients With Schizophrenia}.
\newblock Biol. Psychiatry 2016;15;81(6):495-502.
\newblock doi:{10.1016/j.biopsych.2016.07.012}.

\bibitem{Burt2017}
Burt JB, Demirtas M, Eckner WJ, Navejar NM, Ji JL, Martin WJ, Bernacchia A, Anticevic A, Murray JD.
\newblock {Hierarchy of transcriptomic specialization across human cortex captured by myelin map topography}.
\newblock bioRxiv 2017;
\newblock doi:{10.1101/199703}.

\end{thebibliography}

% \begin{thebibliography}{10}
%\paragraph*{S1 File.}
%\label{S1_File}
%{\bf Lorem ipsum.}  Maecenas convallis mauris sit amet sem ultrices gravida. Etiam eget sapien nibh. Sed ac ipsum eget enim egestas ullamcorper nec euismod ligula. Curabitur fringilla pulvinar lectus consectetur pellentesque.

%\paragraph*{S1 Video.}
%\label{S1_Video}
%{\bf Lorem ipsum.}  Maecenas convallis mauris sit amet sem ultrices gravida. Etiam eget sapien nibh. Sed ac ipsum eget enim egestas ullamcorper nec euismod ligula. Curabitur fringilla pulvinar lectus consectetur pellentesque.

%\paragraph*{S1 Appendix.}
%\label{S1_Appendix}
%{\bf Lorem ipsum.} Maecenas convallis mauris sit amet sem ultrices gravida. Etiam eget sapien nibh. Sed ac ipsum eget enim egestas ullamcorper nec euismod ligula. Curabitur fringilla pulvinar lectus consectetur pellentesque.


% \end{thebibliography}

\newpage
\section*{Supporting Information}
\setcounter{figure}{0} \renewcommand{\thefigure}{S\arabic{figure}}
\renewcommand{\thefigure}{S\arabic{figure}}
\renewcommand{\thetable}{S\arabic{table}}

\paragraph*{S1 Text}
\label{text:expressionAnnotations}
{\bf Expression annotations from WormBase.} 

\paragraph*{S2 Text}
\label{text:matchingIndex}
{\bf Correlated gene expression matching index.} 

\paragraph*{S3 Text}
\label{text:coexprSensitivity}
{\bf Sensitivity of correlated gene expression measures.} 

\clearpage

\paragraph*{S1 Table}
\label{tab:HubList}
{\bf Hub neurons of the \textit{C. elegans} connectome.} Hubs are defined as neurons with degree $k > 44$.
For each hub, we list (i) the neuron name, (ii) its degree, $k$, (iii) location (`head', `body', or `tail'), and function based on information presented in the wormatlas website \url{http://www.wormatlas.org/}.
Neurons are sorted (descending) by degree.}
\paragraph*{S2 Table}
\label{tab:enrichmentCON}
{\bf Enrichment results for connected \textit{vs} unconnected neurons.} Top 15 biological process GO categories enriched in genes with the highest mean increase in CGE for connected neurons compared to unconnected neurons.
Categories are sorted by $p$-value (ascending).
\paragraph*{S3 Table.}
\label{tab:enrichmentRICH}
{\bf Enrichment results for links involving hubs.} Top 15 biological process GO categories enriched in genes with the highest increase in CGE for connections involving hub neurons (i.e., rich, feed-in and feed-out connections) compared to connections between nonhub neurons (i.e., in peripheral connections).
Categories are sorted by $p$-value (ascending).

\clearpage

% Please add the following required packages to your document preamble:
% \usepackage[normalem]{ulem}
% \useunder{\uline}{\ul}{}

%------------------------------------------------------------------------------
% Include only the SI item label in the paragraph heading. Use the \nameref{label} command to cite SI items in the text.
%------------------------------------------------------------------------------

\paragraph*{S1 Fig.}
\label{S1_Fig}
{\bf Rich club organisation and correlated gene expression in synaptic connectivity matrix.}
(A) Rich-club organization of the synaptic C. elegans connectome.
Top: Degree distribution of neurons, labelled to four categories: (i) interneuron (85 neurons, orange), (ii) motor (108 neurons, green), (iii) sensory (68 neurons, blue), or (iv) multiple assignments (18 neurons, yellow).
The distribution features an extended tail of high-degree neurons. Bottom: Normalized rich club coefficient, $\Phi_\mathrm{norm}$ (red), as a function of the degree, $k$, at which hubs are defined (as neurons with degree $>k$).
Also shown is the mean Euclidean separation distance, d (purple) between connected hub regions (across degree thresholds, k). $\Phi_\mathrm{norm} > 1$ indicates that hubs are more densely interconnected among each other than expected by chance, with red circles indicate values of norm that are significantly higher than an ensemble of 1 000 degree-matched null networks ($p < 0.05$).
Purple circles indicate where the Euclidean distance between connected pairs of hubs is significantly greater than the Euclidean distance for all other pairs of connected regions (right-tailed Welch's t-test, $p < 0.05$).
(B) Top: Degree distribution, $k$, of the synaptic \textit{C. elegans}  connectome.
Middle: proportion of connections that are: `rich' (hub $\rightarrow$ hub, red), `feed-in' (nonhub $\rightarrow$ hub, yellow), `feed-out' (hub $\rightarrow$ nonhub, orange), or `peripheral' (nonhub $\rightarrow$ nonhub, blue) as a function of the degree threshold, $k$, used to define hubs.
Note that at high $k$ most neurons are labeled as nonhubs and hence the vast majority of connections are labeled `peripheral'.
Bottom: Median CGE, $\tilde{r}_\phi$, for each connection type as a function of $k$.
The median CGE across all network links is shown as a dotted black line; the topological rich-club regime (determined from the network topology, cf. A) is shaded gray.
Circles indicate a statistically significant increase in CGE in a given link type relative to the rest of the network (one-sided Wilcoxon rank-sum test, $p < 0.05$).

\paragraph*{S2 Fig.}
\label{S2_Fig}
{\bf Dependence of correlated gene expression measures on the proportion of positive annotations.} We plot the mean value of each metric across 1000 different pairs of random, binary vectors of length 948, which vary only in their proportion of `1's (between 0--0.15; corresponding to a number of `1's ranging from 1 to 150).
This is repeated for:
(A) mean square contingency coefficient, $r_\phi$,
(B) Jaccard index,
(C) Yule's $Q$, and
(D) our developed positive match measure, $p_\mathrm{match}$, (see \nameref{text:matchingIndex}).
Any systematic trend in correlation values indicates a bias driven by the proportion of positive annotations for a pair of vectors, as is seen for the Jaccard index and Yule's $Q$.
By contrast, $r_\phi$, which is used through this work, and our probability-based measure, $p_\mathrm{match}$, used to motivate individual gene scoring for enrichment analysis, show no evidence of systematic bias (note the color axis scales).

\paragraph*{S3 Fig.}
\label{S3_Fig}
{\bf Correlated gene expression measured using the positive matching probability index.} The matching probability index, $p_\mathrm{match}$, as introduced in \nameref{text:matchingIndex}. 
\emph{Top}: Degree distribution.
\emph{Middle}: Proportion of connections that are `rich' (hub$\rightarrow$hub, red), `feed-in' (nonhub$\rightarrow$hub, yellow), `feed-out' (hub$\rightarrow$nonhub, orange), and `peripheral' (nonhub$\rightarrow$nonhub, blue) as a function of the degree threshold, $k$, used to define hubs.
Note that at high $k$, most neurons are labeled as nonhubs, and hence the vast majority of connections are `peripheral'.
\emph{Bottom}: Mean CGE calculated using similarity index from only positive matches, $p_{match}$, for each connection type as a function of $k$.
The mean CGE across all network links shown as a dotted black line; the topological rich-club regime (determined from the network topology, cf. Fig.~\ref{fig:Fig5}) is shaded gray.
Circles indicate a statistically significant increase in CGE in a given link type relative to the rest of the network (one-sided Welch's $t$ test; $p < 0.05$).

\paragraph*{S4 Fig.}
\label{S4_Fig}
{\bf Correcting for spatial effects in CGE data using a bulk exponential trend.}
  Here we consider correlated gene expression in the head, where the strongest spatial relationship exists (cf. Fig.~\ref{fig:Fig4}. (A) CGE values, $r_\phi$, plotted as a function of Euclidean separation distance for all pairs of neurons within the head (gray dots), with a fitted exponential trend shown in black, $f(x) = A\exp(-\lambda x) + B$.
(B) Taking residuals from this trend does not adequately correct the spatial trend.
    Note the artifactual negative correlations indicated with an arrow.
    “ This indicates that the trend is not a bulk, isotropic effect, but may instead be driven primarily by a small number of neuron pairs with high $r_\phi$ at short distances ($\lessapprox 50\mu$m), indicated with a circle in (A).
For example, neuron pairs with $r_\phi > 0.8$, are all between the following classes of head neurons: CEP, IL1, OLQ, RMD, RME, RMF, SAAD, SAAV, SAB, SIA, SIB, SMB, SMD, URA, URY.

\paragraph*{S5 Fig.}
\label{S5_Fig}
{\bf Weighted and unweighted rich-club analyses yield similar results.}
   (A) Degree distribution of the \emph{C. elegans} connectome.
    Neurons are labeled to four types as in the legend.
    (B)
    Normalized weighted rich-club coefficient, $\Phi_\mathrm{norm}^\mathrm{weighted}$ (i.e., topology fixed and weights randomized in the null model, shown orange), and
    normalized mixed rich-club coefficient, $\Phi_\mathrm{norm}^\mathrm{mixed}$ (i.e., both topology and weights mixed in the null model, shown green) are plotted as a function of the degree, $k$, at which hubs are defined (as neurons with degree $>k$) \cite{Alstott2014}.
    Circles indicate values of $\Phi_\mathrm{norm}$ that are significantly higher than an ensemble of 1\,000 degree-matched null networks (Welch's $t$-test, $p < 0.05$).
    Compared to topological rich-club analysis presented in the main text, here the weights of the connections are also accounted for when calculating the rich club coefficient.
    In the case of the weighted rich-club coefficient, the topology for the null models was kept stable and only the weights of the connections randomized. Results presented here show that connections between higher degree nodes are stronger than expected by chance.
    On the other hand, in the mixed rich club coefficient both the topology and weights are randomized, therefore we see the combined effect of both types.
    Distinction between the different null models is discussed in detail in \cite{Alstott2014}.
    These results show that connections between high degree nodes are both denser and stronger than expected by chance.

\paragraph*{S6 Fig.}
\label{S6_Fig}
{\bf Rich and non-rich connections in the \emph{C. elegans} connectome, categorized by the  anatomical location of the source and target neurons.}
   Hub-hub connections (`rich') are shown red, and all other connections (`non-rich', i.e., feeder and peripheral) are shown gray, where hubs are defined as neurons with degree, $k > 44$.
Anatomical locations are labeled as `head', `body', and `tail', and each connection is labeled according to its source and target neurons, labeled on the vertical axis in the form `Source-Target'.
The plot shows that the increased separation distance between connected hubs relative to other types of connected neurons is driven by a relative increase in long-range connections between the head and tail.

\paragraph*{S1 File}
\label{file:geneList}
{\bf A list of genes that contributed towards the increased correlated gene expression for (i) connected \textit{vs} unconnected neurons and (ii) rich and feeder \textit{vs} peripheral connections the most.} 

\end{document}
